--- templates/american-medical-association-template.csl
+++ development/american-medical-association-no-url.csl
@@ -5,11 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style class="in-text" default-locale="en-US" delimiter-precedes-last="always" demote-non-dropping-particle="sort-only" initialize-with="" initialize-with-hyphen="false" name-as-sort-order="all" name-delimiter=", " page-range-format="expanded" sort-separator=" " version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>AMA Manual of Style 11th edition (template)</title>
-    <title-short>American Medical Association (numeric/Vancouver/citation-sequence)</title-short>
-    <id>http://www.zotero.org/styles/american-medical-association</id>
-    <link href="http://www.zotero.org/styles/american-medical-association" rel="self"/>
-    <link href="http://www.zotero.org/styles/american-medical-association-10th-edition" rel="template"/>
+    <title>AMA Manual of Style 11th edition (without URLs)</title>
+    <title-short>American Medical Association (numeric/Vancouver/citation-sequence, no URLs)</title-short>
+    <id>http://www.zotero.org/styles/american-medical-association-no-url</id>
+    <link href="http://www.zotero.org/styles/american-medical-association-no-url" rel="self"/>
+    <link href="http://www.zotero.org/styles/american-medical-association" rel="template"/>
     <link href="https://academic.oup.com/amamanualofstyle" rel="documentation"/>
     <author>
       <name>Julian Onions</name>
@@ -33,7 +33,7 @@
     </contributor>
     <category citation-format="numeric"/>
     <category field="medicine"/>
-    <summary>AMA Manual of Style: A Guide for Authors and Editors (11th ed.)</summary>
+    <summary>AMA Manual of Style: A Guide for Authors and Editors (11th ed.), without URLs.</summary>
     <updated>2026-01-19T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -293,7 +293,6 @@
             </else>
           </choose>
         </group>
-        <text macro="access"/>
       </group>
     </layout>
   </bibliography>
