--- templates/american-medical-association-template.csl
+++ development/american-medical-association-parentheses.csl
@@ -5,11 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style class="in-text" default-locale="en-US" delimiter-precedes-last="always" demote-non-dropping-particle="sort-only" initialize-with="" initialize-with-hyphen="false" name-as-sort-order="all" name-delimiter=", " page-range-format="expanded" sort-separator=" " version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>AMA Manual of Style 11th edition (template)</title>
-    <title-short>American Medical Association (numeric/Vancouver/citation-sequence)</title-short>
-    <id>http://www.zotero.org/styles/american-medical-association</id>
-    <link href="http://www.zotero.org/styles/american-medical-association" rel="self"/>
-    <link href="http://www.zotero.org/styles/american-medical-association-10th-edition" rel="template"/>
+    <title>AMA Manual of Style 11th edition (parentheses)</title>
+    <title-short>American Medical Association (numeric/Vancouver/citation-sequence, parentheses)</title-short>
+    <id>http://www.zotero.org/styles/american-medical-association-parentheses</id>
+    <link href="http://www.zotero.org/styles/american-medical-association-parentheses" rel="self"/>
+    <link href="http://www.zotero.org/styles/american-medical-association-brackets" rel="template"/>
     <link href="https://academic.oup.com/amamanualofstyle" rel="documentation"/>
     <author>
       <name>Julian Onions</name>
@@ -137,9 +137,9 @@
     <sort>
       <key variable="citation-number"/>
     </sort>
-    <layout delimiter="," vertical-align="sup">
+    <layout delimiter="," prefix="(" suffix=")">
       <text variable="citation-number"/>
-      <group prefix="(" suffix=")">
+      <group prefix=", ">
         <label form="short" strip-periods="true" variable="locator"/>
         <text variable="locator"/>
       </group>
