--- templates/american-medical-association-template.csl
+++ development/american-medical-association-no-et-al.csl
@@ -5,11 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style class="in-text" default-locale="en-US" delimiter-precedes-last="always" demote-non-dropping-particle="sort-only" initialize-with="" initialize-with-hyphen="false" name-as-sort-order="all" name-delimiter=", " page-range-format="expanded" sort-separator=" " version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>AMA Manual of Style 11th edition (template)</title>
-    <title-short>American Medical Association (numeric/Vancouver/citation-sequence)</title-short>
-    <id>http://www.zotero.org/styles/american-medical-association</id>
-    <link href="http://www.zotero.org/styles/american-medical-association" rel="self"/>
-    <link href="http://www.zotero.org/styles/american-medical-association-10th-edition" rel="template"/>
+    <title>AMA Manual of Style 11th edition (no "et al.")</title>
+    <title-short>American Medical Association (numeric/Vancouver/citation-sequence, no "et al.")</title-short>
+    <id>http://www.zotero.org/styles/american-medical-association-no-et-al</id>
+    <link href="http://www.zotero.org/styles/american-medical-association-no-et-al" rel="self"/>
+    <link href="http://www.zotero.org/styles/american-medical-association" rel="template"/>
     <link href="https://doi.org/10.1093/jama/9780190246556.003.0003" rel="documentation"/>
     <link href="https://zotero.org/groups/2205533/collections/QW4JGG9G" rel="documentation"/>
     <author>
@@ -34,7 +34,7 @@
     </contributor>
     <category citation-format="numeric"/>
     <category field="medicine"/>
-    <summary>AMA Manual of Style: A Guide for Authors and Editors (11th ed.)</summary>
+    <summary>AMA Manual of Style: A Guide for Authors and Editors (11th ed.), without et al.</summary>
     <updated>2026-02-06T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -146,7 +146,7 @@
       </group>
     </layout>
   </citation>
-  <bibliography et-al-min="7" et-al-use-first="3" second-field-align="flush">
+  <bibliography hanging-indent="false" second-field-align="flush">
     <layout>
       <text suffix="." variable="citation-number"/>
       <group delimiter=". ">
