--- templates/american-medical-association-template.csl
+++ development/american-medical-association-brackets.csl
@@ -5,11 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style class="in-text" default-locale="en-US" delimiter-precedes-last="always" demote-non-dropping-particle="sort-only" initialize-with="" initialize-with-hyphen="false" name-as-sort-order="all" name-delimiter=", " page-range-format="expanded" sort-separator=" " version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>AMA Manual of Style 11th edition (template)</title>
-    <title-short>American Medical Association (numeric/Vancouver/citation-sequence)</title-short>
-    <id>http://www.zotero.org/styles/american-medical-association</id>
-    <link href="http://www.zotero.org/styles/american-medical-association" rel="self"/>
-    <link href="http://www.zotero.org/styles/american-medical-association-10th-edition" rel="template"/>
+    <title>AMA Manual of Style 11th edition (brackets)</title>
+    <title-short>American Medical Association (numeric/Vancouver/citation-sequence, brackets)</title-short>
+    <id>http://www.zotero.org/styles/american-medical-association-brackets</id>
+    <link href="http://www.zotero.org/styles/american-medical-association-brackets" rel="self"/>
+    <link href="http://www.zotero.org/styles/american-medical-association" rel="template"/>
     <link href="https://doi.org/10.1093/jama/9780190246556.003.0003" rel="documentation"/>
     <author>
       <name>Julian Onions</name>
@@ -137,11 +137,13 @@
     <sort>
       <key variable="citation-number"/>
     </sort>
-    <layout delimiter="," vertical-align="sup">
-      <text variable="citation-number"/>
-      <group prefix="(" suffix=")">
-        <label form="short" strip-periods="true" variable="locator"/>
-        <text variable="locator"/>
+    <layout delimiter="," prefix="[" suffix="]">
+      <group delimiter=", ">
+        <text variable="citation-number"/>
+        <group>
+          <label form="short" strip-periods="true" variable="locator"/>
+          <text variable="locator"/>
+        </group>
       </group>
     </layout>
   </citation>
