--- templates/apa-template.csl
+++ development/apa-notes.csl
@@ -3,13 +3,13 @@
   type="application/relax-ng-compact-syntax" schematypens="http://relaxng.org/ns/compatibility/annotations/1.0"?>
 <?xml-model href="https://raw.githubusercontent.com/citation-style-language/schema/refs/heads/master/schemas/styles/csl.rnc"
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-<style and="symbol" class="in-text" demote-non-dropping-particle="never" initialize-with=". " names-delimiter=", " page-range-format="expanded" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
+<style and="symbol" class="note" demote-non-dropping-particle="never" initialize-with=". " names-delimiter=", " page-range-format="expanded" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>American Psychological Association 7th edition (template)</title>
-    <title-short>APA Style</title-short>
-    <id>http://www.zotero.org/styles/apa</id>
-    <link href="http://www.zotero.org/styles/apa" rel="self"/>
-    <link href="http://www.zotero.org/styles/apa-6th-edition" rel="template"/>
+    <title>American Psychological Association 7th edition (notes)</title>
+    <title-short>APA Style (notes)</title-short>
+    <id>http://www.zotero.org/styles/apa-notes</id>
+    <link href="http://www.zotero.org/styles/apa-notes" rel="self"/>
+    <link href="http://www.zotero.org/styles/apa" rel="template"/>
     <link href="https://apastyle.apa.org/style-grammar-guidelines/references/examples" rel="documentation"/>
     <author>
       <name>Brenton M. Wiernik</name>
@@ -20,7 +20,7 @@
       <name>Andrew Dunning</name>
       <uri>https://orcid.org/0000-0003-0464-5036</uri>
     </author>
-    <category citation-format="author-date"/>
+    <category citation-format="note"/>
     <category field="anthropology"/>
     <category field="communications"/>
     <category field="generic-base"/>
@@ -29,7 +29,7 @@
     <category field="psychology"/>
     <category field="social_science"/>
     <category field="sociology"/>
-    <summary>Author-date system of the Publication Manual of the American Psychological Association (2020)</summary>
+    <summary>APA Style, unofficial variant with author-date citations in notes</summary>
     <updated>2025-04-14T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -2326,16 +2326,12 @@
   </macro>
   <!-- Citation -->
   <citation collapse="year" disambiguate-add-givenname="true" disambiguate-add-names="true" disambiguate-add-year-suffix="true" et-al-min="3" et-al-use-first="1" givenname-disambiguation-rule="primary-name-with-initials">
-    <sort>
-      <key macro="author-sort" names-min="3" names-use-first="1"/>
-      <key macro="date-sort-group"/>
-      <key macro="date-sort"/>
-      <key variable="status"/>
-    </sort>
-    <layout delimiter="; " prefix="(" suffix=")">
+    <layout delimiter="; " suffix=".">
       <group delimiter=", ">
-        <text macro="author-short"/>
-        <text macro="date-short"/>
+        <group delimiter=" ">
+          <text macro="author-short"/>
+          <text macro="date-short" prefix="(" suffix=")"/>
+        </group>
         <text macro="label-locator-capitalized"/>
       </group>
     </layout>
