--- templates/apa-template.csl
+++ development/apa-numeric-superscript.csl
@@ -5,11 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style class="in-text" demote-non-dropping-particle="never" initialize-with=". " names-delimiter=", " page-range-format="expanded" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>APA Style 7th edition (template)</title>
-    <title-short>Publication Manual of the American Psychological Association, with Bluebook</title-short>
-    <id>http://www.zotero.org/styles/apa</id>
-    <link href="http://www.zotero.org/styles/apa" rel="self"/>
-    <link href="http://www.zotero.org/styles/apa-6th-edition" rel="template"/>
+    <title>APA Style 7th edition (numeric, superscript)</title>
+    <title-short>American Psychological Association, with Bluebook (numeric, superscript)</title-short>
+    <id>http://www.zotero.org/styles/apa-numeric-superscript</id>
+    <link href="http://www.zotero.org/styles/apa-numeric-superscript" rel="self"/>
+    <link href="http://www.zotero.org/styles/apa" rel="template"/>
     <link href="https://apastyle.apa.org/style-grammar-guidelines/references/examples" rel="documentation"/>
     <author>
       <name>Brenton M. Wiernik</name>
@@ -20,7 +20,7 @@
       <name>Andrew Dunning</name>
       <uri>https://orcid.org/0000-0003-0464-5036</uri>
     </author>
-    <category citation-format="author-date"/>
+    <category citation-format="numeric"/>
     <category field="anthropology"/>
     <category field="communications"/>
     <category field="generic-base"/>
@@ -29,7 +29,7 @@
     <category field="psychology"/>
     <category field="social_science"/>
     <category field="sociology"/>
-    <summary>Author-date system of the Publication Manual of the American Psychological Association (2020)</summary>
+    <summary>APA Style, unofficial variant with superscript numeric in-text citations</summary>
     <updated>2025-04-14T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -2293,18 +2293,14 @@
     </group>
   </macro>
   <!-- Citation -->
-  <citation collapse="year" disambiguate-add-givenname="true" disambiguate-add-names="true" disambiguate-add-year-suffix="true" et-al-min="3" et-al-use-first="1">
+  <citation collapse="citation-number">
     <sort>
-      <key macro="author-sort" names-min="3" names-use-first="1"/>
-      <key macro="date-sort-group"/>
-      <key macro="date-sort"/>
-      <key variable="status"/>
+      <key variable="citation-number"/>
     </sort>
-    <layout delimiter="; " prefix="(" suffix=")">
-      <group delimiter=", ">
-        <text macro="author-short"/>
-        <text macro="date-short"/>
-        <text macro="label-locator"/>
+    <layout delimiter="," vertical-align="sup">
+      <group>
+        <text variable="citation-number"/>
+        <text macro="label-locator" prefix="(" suffix=")"/>
       </group>
     </layout>
   </citation>
@@ -2346,20 +2342,11 @@
     </choose>
   </macro>
   <bibliography entry-spacing="0" et-al-min="21" et-al-use-first="19" et-al-use-last="true" hanging-indent="true" line-spacing="2">
-    <sort>
-      <key macro="author-sort"/>
-      <key macro="date-sort-group"/>
-      <key macro="date-sort"/>
-      <key variable="status"/>
-      <key macro="title"/>
-      <key variable="volume"/>
-      <key variable="part-number"/>
-      <key variable="event-date"/>
-      <key variable="original-date"/>
-      <key macro="source-archive"/>
-    </sort>
     <layout>
-      <text macro="bibliography-filtered"/>
+      <group delimiter=". ">
+        <text variable="citation-number"/>
+        <text macro="bibliography"/>
+      </group>
     </layout>
   </bibliography>
 </style>
