--- templates/apa-template.csl
+++ development/apa-no-initials.csl
@@ -3,13 +3,13 @@
   type="application/relax-ng-compact-syntax" schematypens="http://relaxng.org/ns/compatibility/annotations/1.0"?>
 <?xml-model href="https://raw.githubusercontent.com/citation-style-language/schema/refs/heads/master/schemas/styles/csl-repository.rnc"
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-<style class="in-text" demote-non-dropping-particle="never" initialize-with=". " names-delimiter=", " page-range-format="expanded" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
+<style class="in-text" demote-non-dropping-particle="never" names-delimiter=", " page-range-format="expanded" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>American Psychological Association 7th edition (template)</title>
-    <title-short>APA Style with Bluebook</title-short>
-    <id>http://www.zotero.org/styles/apa</id>
-    <link href="http://www.zotero.org/styles/apa" rel="self"/>
-    <link href="http://www.zotero.org/styles/apa-6th-edition" rel="template"/>
+    <title>American Psychological Association 7th edition (no initials)</title>
+    <title-short>APA Style with Bluebook (no initials)</title-short>
+    <id>http://www.zotero.org/styles/apa-no-initials</id>
+    <link href="http://www.zotero.org/styles/apa-no-initials" rel="self"/>
+    <link href="http://www.zotero.org/styles/apa" rel="template"/>
     <link href="https://apastyle.apa.org/style-grammar-guidelines/references/examples" rel="documentation"/>
     <author>
       <name>Brenton M. Wiernik</name>
@@ -29,7 +29,7 @@
     <category field="psychology"/>
     <category field="social_science"/>
     <category field="sociology"/>
-    <summary>Author-date system of the Publication Manual of the American Psychological Association (2020)</summary>
+    <summary>APA Style, unofficial variant without initials</summary>
     <updated>2025-04-14T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
