--- templates/apa-template.csl
+++ development/apa-annotated-bibliography.csl
@@ -5,11 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style class="in-text" demote-non-dropping-particle="never" initialize-with=". " names-delimiter=", " page-range-format="expanded" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>APA Style 7th edition (template)</title>
-    <title-short>Publication Manual of the American Psychological Association, with Bluebook</title-short>
-    <id>http://www.zotero.org/styles/apa</id>
-    <link href="http://www.zotero.org/styles/apa" rel="self"/>
-    <link href="http://www.zotero.org/styles/apa-6th-edition" rel="template"/>
+    <title>APA Style 7th edition (annotated bibliography)</title>
+    <title-short>Publication Manual of the American Psychological Association, with Bluebook (annotated bibliography)</title-short>
+    <id>http://www.zotero.org/styles/apa-annotated-bibliography</id>
+    <link href="http://www.zotero.org/styles/apa-annotated-bibliography" rel="self"/>
+    <link href="http://www.zotero.org/styles/apa" rel="template"/>
     <link href="https://apastyle.apa.org/style-grammar-guidelines/references" rel="documentation"/>
     <link href="https://zotero.org/groups/2205533/collections/MR2N872S" rel="documentation"/>
     <author>
@@ -30,7 +30,7 @@
     <category field="psychology"/>
     <category field="social_science"/>
     <category field="sociology"/>
-    <summary>Author-date system of the Publication Manual of the American Psychological Association (2020)</summary>
+    <summary>APA Style, with annotated bibliography</summary>
     <updated>2026-02-07T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -2273,7 +2273,8 @@
       <key macro="source-archive"/>
     </sort>
     <layout>
-      <text macro="bibliography-filtered"/>
+      <text macro="bibliography"/>
+      <text display="block" variable="note"/>
     </layout>
   </bibliography>
 </style>
