--- templates/apa-template.csl
+++ development/apa-no-ampersand.csl
@@ -5,11 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style class="in-text" demote-non-dropping-particle="never" initialize-with=". " names-delimiter=", " page-range-format="expanded" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>APA Style 7th edition (template)</title>
-    <title-short>Publication Manual of the American Psychological Association, with Bluebook</title-short>
-    <id>http://www.zotero.org/styles/apa</id>
-    <link href="http://www.zotero.org/styles/apa" rel="self"/>
-    <link href="http://www.zotero.org/styles/apa-6th-edition" rel="template"/>
+    <title>APA Style 7th edition (no ampersand)</title>
+    <title-short>American Psychological Association, with Bluebook (no ampersand)</title-short>
+    <id>http://www.zotero.org/styles/apa-no-ampersand</id>
+    <link href="http://www.zotero.org/styles/apa-no-ampersand" rel="self"/>
+    <link href="http://www.zotero.org/styles/apa" rel="template"/>
     <link href="https://apastyle.apa.org/style-grammar-guidelines/references/examples" rel="documentation"/>
     <author>
       <name>Brenton M. Wiernik</name>
@@ -29,7 +29,7 @@
     <category field="psychology"/>
     <category field="social_science"/>
     <category field="sociology"/>
-    <summary>Author-date system of the Publication Manual of the American Psychological Association (2020)</summary>
+    <summary>APA Style, unofficial variant without ampersand</summary>
     <updated>2025-04-14T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -38,6 +38,14 @@
       <term name="ad"> C.E.</term>
       <term name="bc"> B.C.E.</term>
       <term form="short" name="circa">ca.</term>
+      <term form="short" name="editor-translator">
+        <single>ed. and trans.</single>
+        <multiple>eds. and trans.</multiple>
+      </term>
+      <term form="short" name="editortranslator">
+        <single>ed. and trans.</single>
+        <multiple>eds. and trans.</multiple>
+      </term>
       <term name="guest">
         <single>guest expert</single>
         <multiple>guest experts</multiple>
@@ -352,7 +360,7 @@
   <macro name="author">
     <!-- Substitutes for missing authors: order prioritizes primary creators (e.g., composer, author) over secondary roles (e.g., editor, curator), with title as the final fallback. -->
     <names variable="composer">
-      <name and="symbol" delimiter-precedes-last="always" name-as-sort-order="all"/>
+      <name and="text" delimiter-precedes-last="always" name-as-sort-order="all"/>
       <label form="short" prefix=" (" suffix=")" text-case="title"/>
       <substitute>
         <names variable="author"/>
@@ -362,24 +370,24 @@
           <if type="broadcast">
             <names variable="script-writer director">
               <!-- Actors/performers and producers [not executive] not cited in APA style. -->
-              <name and="symbol" delimiter-precedes-last="always" name-as-sort-order="all"/>
+              <name and="text" delimiter-precedes-last="always" name-as-sort-order="all"/>
               <label prefix=" (" suffix=")" text-case="title"/>
             </names>
           </if>
         </choose>
         <names variable="director">
           <!-- For non-broadcast items, APA only cites directors and not writers. -->
-          <name and="symbol" delimiter-precedes-last="always" name-as-sort-order="all"/>
+          <name and="text" delimiter-precedes-last="always" name-as-sort-order="all"/>
           <label prefix=" (" suffix=")" text-case="title"/>
         </names>
         <names variable="guest host">
           <!-- TODO: Collapse variables when that becomes available. -->
-          <name and="symbol" delimiter-precedes-last="always" name-as-sort-order="all"/>
+          <name and="text" delimiter-precedes-last="always" name-as-sort-order="all"/>
           <label prefix=" (" suffix=")" text-case="title"/>
         </names>
         <names variable="producer">
           <!-- Producers not cited if there is a writer/director, but use if they are the principal creator. -->
-          <name and="symbol" delimiter-precedes-last="always" name-as-sort-order="all"/>
+          <name and="text" delimiter-precedes-last="always" name-as-sort-order="all"/>
           <label prefix=" (" suffix=")" text-case="title"/>
         </names>
         <choose>
@@ -395,11 +403,11 @@
           </else-if>
         </choose>
         <names variable="executive-producer">
-          <name and="symbol" delimiter-precedes-last="always" name-as-sort-order="all"/>
+          <name and="text" delimiter-precedes-last="always" name-as-sort-order="all"/>
           <label prefix=" (" suffix=")" text-case="title"/>
         </names>
         <names variable="series-creator">
-          <name and="symbol" delimiter-precedes-last="always" name-as-sort-order="all"/>
+          <name and="text" delimiter-precedes-last="always" name-as-sort-order="all"/>
           <label prefix=" (" suffix=")" text-case="title"/>
         </names>
         <names variable="editor-translator"/>
@@ -407,23 +415,23 @@
         <names variable="editor"/>
         <names variable="editorial-director"/>
         <names variable="compiler">
-          <name and="symbol" delimiter-precedes-last="always" name-as-sort-order="all"/>
+          <name and="text" delimiter-precedes-last="always" name-as-sort-order="all"/>
           <label prefix=" (" suffix=")" text-case="title"/>
         </names>
         <choose>
           <if match="any" type="event performance speech">
             <names variable="chair">
-              <name and="symbol" delimiter-precedes-last="always" name-as-sort-order="all"/>
+              <name and="text" delimiter-precedes-last="always" name-as-sort-order="all"/>
               <label prefix=" (" suffix=")" text-case="title"/>
             </names>
             <names variable="organizer">
-              <name and="symbol" delimiter-precedes-last="always" name-as-sort-order="all"/>
+              <name and="text" delimiter-precedes-last="always" name-as-sort-order="all"/>
               <label prefix=" (" suffix=")" text-case="title"/>
             </names>
           </if>
         </choose>
         <names variable="curator">
-          <name and="symbol" delimiter-precedes-last="always" name-as-sort-order="all"/>
+          <name and="text" delimiter-precedes-last="always" name-as-sort-order="all"/>
           <label prefix=" (" suffix=")" text-case="title"/>
         </names>
         <names variable="collection-editor"/>
@@ -445,7 +453,7 @@
         <else-if match="any" type="book musical_score pamphlet report standard">
           <names prefix="(" suffix=")" variable="contributor">
             <label form="verb" suffix=" "/>
-            <name and="symbol" delimiter-precedes-last="always" name-as-sort-order="all"/>
+            <name and="text" delimiter-precedes-last="always" name-as-sort-order="all"/>
           </names>
         </else-if>
       </choose>
@@ -461,7 +469,7 @@
           <!-- These variables indicate that the letter is retrievable by the reader. If not, use the APA in-text-only personal communication format. -->
           <if match="any" variable="archive archive-place container-title DOI number publisher references URL">
             <names variable="author">
-              <name and="symbol" form="short"/>
+              <name and="text" form="short"/>
               <substitute>
                 <text macro="title-and-descriptions-short"/>
               </substitute>
@@ -481,7 +489,7 @@
       </else-if>
       <else>
         <names variable="composer">
-          <name and="symbol" form="short"/>
+          <name and="text" form="short"/>
           <substitute>
             <names variable="author"/>
             <names variable="illustrator"/>
@@ -1017,7 +1025,7 @@
       <choose>
         <if variable="title">
           <names variable="interviewer">
-            <name and="symbol"/>
+            <name and="text"/>
             <label form="short" prefix=", " text-case="title"/>
           </names>
         </if>
@@ -1027,24 +1035,24 @@
           <!-- Handle `container-title` on `post` or `webpage` akin to `publisher` -->
           <names variable="container-author">
             <label form="verb-short" suffix=" " text-case="title"/>
-            <name and="symbol"/>
+            <name and="text"/>
           </names>
           <names delimiter="; " variable="editor translator">
-            <name and="symbol"/>
+            <name and="text"/>
             <label form="short" prefix=", " text-case="title"/>
           </names>
           <names delimiter="; " variable="illustrator narrator">
-            <name and="symbol"/>
+            <name and="text"/>
             <label form="short" prefix=", " text-case="title"/>
           </names>
           <names delimiter="; " variable="compiler chair organizer curator series-creator executive-producer">
-            <name and="symbol"/>
+            <name and="text"/>
             <label prefix=", " text-case="title"/>
           </names>
         </if>
         <else>
           <names delimiter="; " variable="illustrator narrator">
-            <name and="symbol"/>
+            <name and="text"/>
             <label form="short" prefix=", " text-case="title"/>
           </names>
           <choose>
@@ -1052,21 +1060,21 @@
             <else-if variable="container-title">
               <!-- TODO: Check logic once processors start to automatically populate `editor-translator` -->
               <names delimiter="; " variable="translator">
-                <name and="symbol"/>
+                <name and="text"/>
                 <label form="short" prefix=", " text-case="title"/>
               </names>
             </else-if>
             <else>
               <names variable="container-author">
                 <label form="verb-short" suffix=" " text-case="title"/>
-                <name and="symbol"/>
+                <name and="text"/>
               </names>
               <names delimiter="; " variable="editor translator">
-                <name and="symbol"/>
+                <name and="text"/>
                 <label form="short" prefix=", " text-case="title"/>
               </names>
               <names delimiter="; " variable="compiler chair organizer curator series-creator executive-producer">
-                <name and="symbol"/>
+                <name and="text"/>
                 <label prefix=", " text-case="title"/>
               </names>
             </else>
@@ -1080,13 +1088,13 @@
       <choose>
         <if variable="title">
           <names delimiter="; " variable="interviewer">
-            <name and="symbol"/>
+            <name and="text"/>
             <label form="short" prefix=", " text-case="title"/>
           </names>
         </if>
       </choose>
       <names delimiter="; " variable="translator narrator">
-        <name and="symbol"/>
+        <name and="text"/>
         <label form="short" prefix=", " text-case="title"/>
       </names>
     </group>
@@ -1410,7 +1418,7 @@
         <else-if variable="interviewer">
           <names variable="interviewer">
             <label form="verb" suffix=" " text-case="capitalize-first"/>
-            <name and="symbol"/>
+            <name and="text"/>
           </names>
           <text text-case="capitalize-first" variable="medium"/>
         </else-if>
@@ -1423,7 +1431,7 @@
   <macro name="description-interview-short">
     <names variable="interviewer">
       <label form="verb" suffix=" " text-case="capitalize-first"/>
-      <name and="symbol" form="short"/>
+      <name and="text" form="short"/>
       <substitute>
         <text macro="description-format-short"/>
       </substitute>
@@ -1437,7 +1445,7 @@
             <text macro="description-format"/>
             <names variable="recipient">
               <label form="verb" suffix=" "/>
-              <name and="symbol" initialize="false"/>
+              <name and="text" initialize="false"/>
             </names>
           </group>
           <text macro="description-medium"/>
@@ -1455,7 +1463,7 @@
           <text macro="description-format-short"/>
           <names variable="recipient">
             <label form="verb" suffix=" "/>
-            <name and="symbol" form="short"/>
+            <name and="text" form="short"/>
           </names>
         </group>
       </if>
@@ -1504,7 +1512,7 @@
         </group>
         <names variable="reviewed-author">
           <label form="verb-short" suffix=" "/>
-          <name and="symbol"/>
+          <name and="text"/>
         </names>
       </group>
       <text macro="description-medium"/>
@@ -1690,22 +1698,22 @@
   <!-- Monographic source author -->
   <macro name="source-monographic-author">
     <names variable="container-author">
-      <name and="symbol"/>
+      <name and="text"/>
       <label prefix=" (" suffix=")" text-case="title"/>
       <substitute>
         <names variable="executive-producer"/>
         <names variable="series-creator"/>
         <names variable="editor-translator">
-          <name and="symbol"/>
+          <name and="text"/>
           <label form="short" prefix=" (" suffix=")" text-case="title"/>
         </names>
         <!-- TODO: Translator omitted on the assumption that editor-translators are uncommon for chapter citations. If needed, direct entry or automatic population of `editor-translator` can produce combined labels. -->
         <names delimiter="; " variable="editor">
-          <name and="symbol"/>
+          <name and="text"/>
           <label form="short" prefix=" (" suffix=")" text-case="title"/>
         </names>
         <names variable="editorial-director">
-          <name and="symbol"/>
+          <name and="text"/>
           <label form="short" prefix=" (" suffix=")" text-case="title"/>
         </names>
         <names variable="compiler"/>
@@ -1717,7 +1725,7 @@
         </choose>
         <names variable="curator"/>
         <names variable="collection-editor">
-          <name and="symbol"/>
+          <name and="text"/>
           <label form="short" prefix=" (" suffix=")" text-case="title"/>
         </names>
       </substitute>
