--- templates/chicago-template.csl
+++ development/chicago-notes-classic-no-url.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style class="note" demote-non-dropping-particle="display-and-sort" page-range-format="chicago-16" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
-    <title-short>CMOS with Bluebook</title-short>
-    <id>http://www.zotero.org/styles/chicago-notes-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="self"/>
+    <title>Chicago Manual of Style 18th edition (notes without bibliography, classic variants, without URLs)</title>
+    <title-short>CMOS with Bluebook (notes only [13.23, 14.20], classic variants [14.8, 14.30], no URLs [13.6])</title-short>
+    <id>http://www.zotero.org/styles/chicago-notes-classic-no-url</id>
+    <link href="http://www.zotero.org/styles/chicago-notes-classic-no-url" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="template"/>
     <link href="https://www.chicagomanualofstyle.org/" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -30,7 +31,7 @@
     <category field="social_science"/>
     <category field="sociology"/>
     <category field="theology"/>
-    <summary>Chicago-style source citations (with Bluebook for legal citations), template</summary>
+    <summary>Chicago-style source citations (with Bluebook for legal citations), notes system without bibliography, listing up to six authors in a full citation (CMOS18 13.23), displaying number of volumes (CMOS18 14.20), classic variants: chapter page numbers (CMOS18 14.8), place of publication (CMOS18 14.30); without URLs (CMOS18 13.6)</summary>
     <updated>2025-10-27T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -431,12 +432,12 @@
       </choose>
       <text variable="volume"/>
       <!-- always display number of volumes in notes without bibliography (CMOS 14.20): -->
-      <!-- <choose>
+      <choose>
         <if is-numeric="number-of-volumes" variable="volume">
           <text value="of"/>
           <text variable="number-of-volumes"/>
         </if>
-      </choose> -->
+      </choose>
     </group>
   </macro>
   <macro name="label-volume-capitalized">
@@ -3561,9 +3562,9 @@
               <text macro="label-volume"/>
             </if>
             <!-- always display number of volumes in notes without bibliography (CMOS 14.20): -->
-            <!-- <else>
+            <else>
               <text macro="label-number-of-volumes"/>
-            </else> -->
+            </else>
           </choose>
         </else-if>
         <else-if is-numeric="volume" variable="page">
@@ -3571,23 +3572,15 @@
             <if match="any" variable="collection-editor part-number part-title volume-title">
               <text macro="label-volume"/>
             </if>
-            <else-if variable="container-title">
-              <!-- remove condition in styles that print chapter page numbers (CMOS17/classic) -->
-              <text macro="label-volume"/>
-            </else-if>
             <!-- always display number of volumes in notes without bibliography (CMOS 14.20): -->
-            <!-- <else>
+            <else>
               <text macro="label-number-of-volumes"/>
-            </else> -->
+            </else>
           </choose>
         </else-if>
         <else-if variable="volume">
           <text macro="label-volume"/>
         </else-if>
-        <!-- notes only show the number of volumes if there is no physical locator, which should include a volume number, e.g. 4:243; except when using notes without bibliography (CMOS18 14.20) -->
-        <else-if locator="column"/>
-        <else-if locator="folio"/>
-        <else-if locator="page"/>
         <else>
           <text macro="label-number-of-volumes"/>
         </else>
@@ -4336,11 +4329,9 @@
           </if>
         </choose>
       </else-if>
-      <!-- if printing chapter page numbers (CMOS17/classic):
       <else-if variable="chapter-number page title">
         <text term="in" text-case="capitalize-first"/>
       </else-if>
-      -->
       <else-if variable="chapter-number">
         <group delimiter=" ">
           <text macro="label-chapter-number-capitalized"/>
@@ -4372,11 +4363,9 @@
           </if>
         </choose>
       </else-if>
-      <!-- if printing chapter page numbers (CMOS17/classic):
       <else-if variable="chapter-number page title">
         <text term="in"/>
       </else-if>
-      -->
       <else-if variable="chapter-number">
         <group delimiter=" ">
           <text macro="label-chapter-number"/>
@@ -4823,10 +4812,6 @@
             <if is-numeric="volume" match="none">
               <text macro="label-volume"/>
             </if>
-            <else-if variable="container-title">
-              <!-- remove condition in styles that print chapter page numbers (CMOS17/classic) -->
-              <text macro="label-volume"/>
-            </else-if>
             <else-if is-numeric="volume" variable="page">
               <choose>
                 <!-- check for variables that might come between the volume and page number -->
@@ -4868,8 +4853,6 @@
       <else-if variable="locator">
         <text macro="label-locator"/>
       </else-if>
-      <!-- remove `container-title` condition in styles that print chapter page numbers (CMOS17/classic) -->
-      <else-if variable="container-title"/>
       <else-if is-numeric="volume" variable="page">
         <!-- collapse the volume and page number if adjacent (CMOS18 14.18) -->
         <group delimiter=":">
@@ -5642,7 +5625,7 @@
         </if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text text-case="capitalize-first" variable="publisher-place"/> -->
+            <text text-case="capitalize-first" variable="publisher-place"/>
             <text text-case="capitalize-first" variable="publisher"/>
           </group>
         </else-if>
@@ -5668,7 +5651,7 @@
         </if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text variable="publisher-place"/> -->
+            <text variable="publisher-place"/>
             <text variable="publisher"/>
           </group>
         </else-if>
@@ -5690,7 +5673,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text text-case="capitalize-first" variable="original-publisher-place"/> -->
+          <text text-case="capitalize-first" variable="original-publisher-place"/>
           <text text-case="capitalize-first" variable="original-publisher"/>
         </group>
       </if>
@@ -5703,7 +5686,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text variable="original-publisher-place"/> -->
+          <text variable="original-publisher-place"/>
           <text variable="original-publisher"/>
         </group>
       </if>
@@ -6300,7 +6283,7 @@
           </group>
         </else-if>
       </choose>
-      <text macro="source-DOI-URL"/>
+      <!-- <text macro="source-DOI-URL"/> -->
     </group>
   </macro>
   <macro name="source-date-accessed-DOI-URL-note">
@@ -6315,7 +6298,7 @@
           </group>
         </else-if>
       </choose>
-      <text macro="source-DOI-URL"/>
+      <!-- <text macro="source-DOI-URL"/> -->
     </group>
   </macro>
   <macro name="source-DOI-URL">
@@ -7020,7 +7003,7 @@
       </else>
     </choose>
   </macro>
-  <citation disambiguate-add-givenname="true" disambiguate-add-names="true" et-al-min="3" et-al-use-first="1">
+  <citation disambiguate-add-givenname="true" disambiguate-add-names="true" et-al-min="7" et-al-subsequent-min="3" et-al-subsequent-use-first="1" et-al-use-first="3">
     <layout delimiter="; " suffix=".">
       <text macro="citation-notes-full-subsequent-author-title"/>
     </layout>
@@ -7170,19 +7153,4 @@
       </else>
     </choose>
   </macro>
-  <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true">
-    <sort>
-      <key macro="author-sort"/>
-      <key macro="title-and-descriptions-bib"/>
-      <key macro="source-bib"/>
-      <key variable="volume"/>
-      <key variable="part-number"/>
-      <key variable="event-date"/>
-      <key macro="source-date-issued-or-status"/>
-      <key macro="source-archive-bib"/>
-    </sort>
-    <layout suffix=".">
-      <text macro="bibliography-filtered-notes"/>
-    </layout>
-  </bibliography>
 </style>
