--- templates/chicago-template.csl
+++ development/chicago-in-text-full-no-url.csl
@@ -3,18 +3,18 @@
   type="application/relax-ng-compact-syntax" schematypens="http://relaxng.org/ns/compatibility/annotations/1.0"?>
 <?xml-model href="https://raw.githubusercontent.com/citation-style-language/schema/refs/heads/master/schemas/styles/csl-repository.rnc"
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-<style class="note" demote-non-dropping-particle="display-and-sort" page-range-format="chicago-16" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
+<style class="in-text" demote-non-dropping-particle="display-and-sort" page-range-format="chicago-16" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
-    <title-short>CMOS with Bluebook</title-short>
-    <id>http://www.zotero.org/styles/chicago-notes-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="self"/>
+    <title>Chicago Manual of Style 18th edition (in-text full citations, without URLs)</title>
+    <title-short>CMOS with Bluebook (parenthetical full citations [12.76], no URLs [13.6])</title-short>
+    <id>http://www.zotero.org/styles/chicago-in-text-full-no-url</id>
+    <link href="http://www.zotero.org/styles/chicago-in-text-full-no-url" rel="self"/>
     <link href="https://www.chicagomanualofstyle.org/" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
       <uri>https://orcid.org/0000-0003-0464-5036</uri>
     </author>
-    <category citation-format="note"/>
+    <category citation-format="author"/>
     <category field="anthropology"/>
     <category field="communications"/>
     <category field="generic-base"/>
@@ -30,7 +30,7 @@
     <category field="social_science"/>
     <category field="sociology"/>
     <category field="theology"/>
-    <summary>Chicago-style source citations (with Bluebook for legal citations), template</summary>
+    <summary>Chicago-style source citations (with Bluebook for legal citations), full in-text citation (CMOS18 12.76), without URLs (CMOS18 13.6)</summary>
     <updated>2025-10-27T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -4194,24 +4194,24 @@
                 <!-- date appears first if a review `container-title` has been substituted for a missing author (CMOS18 14.89) -->
                 <choose>
                   <if match="any" variable="author chair collection-editor compiler composer curator director editor editor-translator editorial-director executive-producer guest host illustrator organizer producer series-creator translator">
-                    <text macro="source-date-issued-or-status" prefix="(" suffix=")"/>
+                    <text macro="source-date-issued-or-status" prefix="[" suffix="]"/>
                     <!-- for CMOS17 author-date: -->
                     <!-- <text macro="source-date-issued-month-day" prefix="(" suffix=")"/> -->
                   </if>
                   <else-if match="any" variable="reviewed-genre reviewed-title title">
-                    <text macro="source-date-issued-or-status" prefix="(" suffix=")"/>
+                    <text macro="source-date-issued-or-status" prefix="[" suffix="]"/>
                     <!-- for CMOS17 author-date: -->
                     <!-- <text macro="source-date-issued-month-day" prefix="(" suffix=")"/> -->
                   </else-if>
                   <else-if match="none" variable="genre section">
-                    <text macro="source-date-issued-or-status" prefix="(" suffix=")"/>
+                    <text macro="source-date-issued-or-status" prefix="[" suffix="]"/>
                     <!-- for CMOS17 author-date: -->
                     <!-- <text macro="source-date-issued-month-day" prefix="(" suffix=")"/> -->
                   </else-if>
                 </choose>
               </if>
               <else>
-                <text macro="source-date-issued-or-status" prefix="(" suffix=")"/>
+                <text macro="source-date-issued-or-status" prefix="[" suffix="]"/>
                 <!-- for CMOS17 author-date: -->
                 <!-- <text macro="source-date-issued-month-day" prefix="(" suffix=")"/> -->
               </else>
@@ -5256,32 +5256,32 @@
     <!-- Facts of publication with brackets for print monographic formats -->
     <choose>
       <if match="any" type="book chapter classic interview musical_score pamphlet paper-conference report thesis">
-        <text macro="source-monographic-publication-note" prefix="(" suffix=")"/>
+        <text macro="source-monographic-publication-note" prefix="[" suffix="]"/>
       </if>
       <else-if type="article" variable="genre publisher">
         <!-- without brackets for a preprint in a repository (CMOS18 14.76), with brackets for a working paper (CMOS18 14.116) -->
-        <text macro="source-monographic-publication-note" prefix="(" suffix=")"/>
+        <text macro="source-monographic-publication-note" prefix="[" suffix="]"/>
       </else-if>
       <else-if match="any" type="entry entry-dictionary entry-encyclopedia">
         <choose>
           <if match="any" variable="publisher publisher-place">
-            <text macro="source-monographic-publication-note" prefix="(" suffix=")"/>
+            <text macro="source-monographic-publication-note" prefix="[" suffix="]"/>
           </if>
           <!-- online reference sources without a additional publisher details are unbracketed (CMOS18 14.131) -->
           <else-if match="any" variable="DOI URL"/>
           <else>
-            <text macro="source-monographic-publication-note" prefix="(" suffix=")"/>
+            <text macro="source-monographic-publication-note" prefix="[" suffix="]"/>
           </else>
         </choose>
       </else-if>
       <else-if type="motion_picture" variable="publisher">
-        <text macro="source-monographic-publication-note" prefix="(" suffix=")"/>
+        <text macro="source-monographic-publication-note" prefix="[" suffix="]"/>
       </else-if>
       <else-if type="personal_communication">
         <choose>
           <if match="any" variable="collection-editor compiler editor editorial-director publisher">
             <!-- in an edited collection -->
-            <text macro="source-monographic-publication-note" prefix="(" suffix=")"/>
+            <text macro="source-monographic-publication-note" prefix="[" suffix="]"/>
           </if>
         </choose>
       </else-if>
@@ -5324,7 +5324,7 @@
     <!-- mirrors `source-monographic-publication-note-bracketed` -->
     <choose>
       <if disambiguate="true">
-        <group delimiter=", " prefix="(" suffix=")">
+        <group delimiter=", " prefix="[" suffix="]">
           <choose>
             <!-- Facts of publication with brackets for print monographic formats -->
             <if match="any" type="book chapter classic interview musical_score pamphlet paper-conference report thesis">
@@ -5663,7 +5663,7 @@
             </names>
             <text font-style="italic" text-case="title" variable="original-title"/>
           </group>
-          <group delimiter=", " prefix="(" suffix=")">
+          <group delimiter=", " prefix="[" suffix="]">
             <text macro="source-publication-publisher-original-note"/>
             <text macro="source-date-original"/>
           </group>
@@ -6337,7 +6337,7 @@
           </group>
         </else-if>
       </choose>
-      <text macro="source-DOI-URL"/>
+      <!-- <text macro="source-DOI-URL"/> -->
     </group>
   </macro>
   <macro name="source-date-accessed-DOI-URL-note">
@@ -6352,7 +6352,7 @@
           </group>
         </else-if>
       </choose>
-      <text macro="source-DOI-URL"/>
+      <!-- <text macro="source-DOI-URL"/> -->
     </group>
   </macro>
   <macro name="source-DOI-URL">
@@ -7058,8 +7058,8 @@
     </choose>
   </macro>
   <citation disambiguate-add-givenname="true" disambiguate-add-names="true" et-al-min="3" et-al-use-first="1">
-    <layout delimiter="; " suffix=".">
-      <text macro="citation-notes-full-subsequent-author-title"/>
+    <layout delimiter="; " prefix="(" suffix=")">
+      <text macro="citation-notes-full-subsequent-author-only"/>
     </layout>
   </citation>
   <!-- Bibliography -->
