--- templates/chicago-template.csl
+++ development/chicago-notes-bibliography-archive-place-first-no-url.csl
@@ -5,10 +5,10 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style and="text" class="note" demote-non-dropping-particle="display-and-sort" page-range-format="chicago" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
+    <title>Chicago Manual of Style 18th edition (notes and bibliography, archival references with place first, without URLs)</title>
     <title-short>CMOS</title-short>
-    <id>http://www.zotero.org/styles/chicago-notes-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="self"/>
+    <id>http://www.zotero.org/styles/chicago-notes-bibliography-archive-place-first-no-url</id>
+    <link href="http://www.zotero.org/styles/chicago-notes-bibliography-archive-place-first-no-url" rel="self"/>
     <link href="http://www.chicagomanualofstyle.org/tools_citationguide.html" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -22,21 +22,7 @@
     <category field="literature"/>
     <category field="philosophy"/>
     <category field="theology"/>
-    <!-- Categories for author-date:
-    <category citation-format="author-date"/>
-    <category field="anthropology"/>
-    <category field="communications"/>
-    <category field="generic-base"/>
-    <category field="geography"/>
-    <category field="history"/>
-    <category field="humanities"/>
-    <category field="linguistics"/>
-    <category field="political_science"/>
-    <category field="science"/>
-    <category field="social_science"/>
-    <category field="sociology"/>
-    -->
-    <summary>Chicago-style source citations (with Bluebook for legal citations), template</summary>
+    <summary>Chicago-style source citations (with Bluebook for legal citations), notes and bibliography system, archival references with place first (CMOS 14.205), without URLs (CMOS 13.6)</summary>
     <updated>2025-05-19T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -1423,7 +1409,7 @@
       </group>
       <text macro="source-medium-bib"/>
       <text macro="source-archive-bib"/>
-      <text macro="source-DOI-URL-bib"/>
+      <!-- <text macro="source-DOI-URL-bib"/> -->
     </group>
   </macro>
   <macro name="title-and-source-note">
@@ -1476,7 +1462,7 @@
       </choose>
       <text macro="source-medium-note"/>
       <text macro="source-archive-note"/>
-      <text macro="source-DOI-URL-note"/>
+      <!-- <text macro="source-DOI-URL-note"/> -->
     </group>
   </macro>
   <!-- 3.1. Title -->
@@ -5690,7 +5676,7 @@
         <text macro="source-archive-name-first"/>
       </if>
       <else-if match="any" type="collection document manuscript" variable="archive_collection archive-place">
-        <text macro="source-archive-location-first-bib"/>
+        <text macro="source-archive-place-first"/>
       </else-if>
       <else>
         <text macro="source-archive-identifier"/>
@@ -5704,7 +5690,7 @@
         <text macro="source-archive-name-first"/>
       </if>
       <else-if match="any" type="collection document manuscript" variable="archive_collection archive-place">
-        <text macro="source-archive-location-first-note"/>
+        <text macro="source-archive-place-first"/>
       </else-if>
       <!-- Without `archive_collection` or `archive-place`: digital archives (database and identifier) -->
       <else>
@@ -5873,7 +5859,7 @@
         </else>
       </choose>
       <text variable="references"/>
-      <text macro="source-DOI-URL"/>
+      <!-- <text macro="source-DOI-URL"/> -->
     </group>
   </macro>
   <!-- 6.1. Legal author -->
