--- templates/chicago-template.csl
+++ development/chicago-fullnote-bibliography-classic-no-url.csl
@@ -5,11 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style and="text" class="note" demote-non-dropping-particle="display-and-sort" names-delimiter=", " page-range-format="chicago" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
+    <title>Chicago Manual of Style 18th edition (full note with classic variants, without URL)</title>
     <title-short>CMOS</title-short>
-    <id>http://www.zotero.org/styles/chicago-fullnote-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography" rel="self"/>
-    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography-17th-edition" rel="template"/>
+    <id>http://www.zotero.org/styles/chicago-fullnote-bibliography-classic-no-url</id>
+    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography-classic-no-url" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography" rel="template"/>
     <link href="http://www.chicagomanualofstyle.org/tools_citationguide.html" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -59,7 +59,7 @@
     <category field="humanities"/>
     <category field="social_science"/>
     <category field="generic-base"/>
-    <summary>Chicago Manual of Style full notes and bibliography</summary>
+    <summary>Chicago Manual of Style full notes and bibliography, with classic variants (chapter page numbers, place of publication, and 3-em dash), without URL</summary>
     <updated>2025-03-01T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -1836,9 +1836,6 @@
       <else-if match="any" variable="chair compiler container-author contributor curator director editor translator editor-translator editorial-director executive-producer guest host illustrator interviewer narrator organizer original-author performer producer script-writer series-creator volume-title">
         <text macro="volume-labelled-note"/>
       </else-if>
-      <else-if type="chapter">
-        <text macro="volume-labelled-note"/>
-      </else-if>
     </choose>
   </macro>
   <macro name="number-of-volumes-labelled">
@@ -2256,7 +2253,7 @@
           </if>
           <else>
             <group delimiter=": ">
-              <!-- <text variable="publisher-place"/> -->
+              <text variable="publisher-place"/>
               <text variable="publisher"/>
             </group>
           </else>
@@ -2272,7 +2269,7 @@
         </if>
         <else>
           <group delimiter=": ">
-            <!-- <text variable="original-publisher-place"/> -->
+            <text variable="original-publisher-place"/>
             <text variable="original-publisher"/>
           </group>
         </else>
@@ -2695,7 +2692,7 @@
   </macro>
   <macro name="locators">
     <choose>
-      <if match="none" type="chapter" variable="locator">
+      <if match="none" variable="locator">
         <choose>
           <if variable="number">
             <text variable="number"/>
@@ -2905,7 +2902,7 @@
       <if match="any" variable="DOI URL">
         <choose>
           <if match="none" type="bill hearing legal_case legislation regulation treaty">
-            <text macro="URI"/>
+            <!-- <text macro="URI"/> -->
           </if>
         </choose>
       </if>
@@ -3145,7 +3142,7 @@
       </else>
     </choose>
   </macro>
-  <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true">
+  <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true" subsequent-author-substitute="&#8212;&#8212;&#8212;">
     <sort>
       <key macro="author-sort"/>
       <key macro="title-bib"/>
