--- templates/chicago-template.csl
+++ development/chicago-notes-bibliography.csl
@@ -5,8 +5,8 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style class="note" demote-non-dropping-particle="display-and-sort" page-range-format="chicago-16" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
-    <title-short>CMOS with Bluebook</title-short>
+    <title>Chicago Manual of Style 18th edition (notes and bibliography)</title>
+    <title-short>CMOS with Bluebook (notes and bibliography/NB [13.18])</title-short>
     <id>http://www.zotero.org/styles/chicago-notes-bibliography</id>
     <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="self"/>
     <link href="https://www.chicagomanualofstyle.org/" rel="documentation"/>
@@ -30,7 +30,7 @@
     <category field="social_science"/>
     <category field="sociology"/>
     <category field="theology"/>
-    <summary>Chicago-style source citations (with Bluebook for legal citations), template</summary>
+    <summary>Chicago-style source citations (with Bluebook for legal citations), notes and bibliography system</summary>
     <updated>2025-10-27T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
