--- templates/chicago-template.csl
+++ development/chicago-short-notes-bibliography-classic.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style and="text" class="note" demote-non-dropping-particle="display-and-sort" page-range-format="chicago" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
+    <title>Chicago Manual of Style 18th edition (shortened notes and bibliography, with classic variants)</title>
     <title-short>CMOS</title-short>
-    <id>http://www.zotero.org/styles/chicago-notes-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="self"/>
+    <id>http://www.zotero.org/styles/chicago-short-notes-bibliography-classic</id>
+    <link href="http://www.zotero.org/styles/chicago-short-notes-bibliography-classic" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="template"/>
     <link href="http://www.chicagomanualofstyle.org/tools_citationguide.html" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -30,7 +31,7 @@
     <category field="social_science"/>
     <category field="sociology"/>
     <category field="theology"/>
-    <summary>Chicago-style source citations (with Bluebook for legal citations), template</summary>
+    <summary>Chicago-style source citations (with Bluebook for legal citations), shortened author-title notes and bibliography system, with classic variants: chapter page numbers (CMOS 14.8), place of publication (CMOS 14.30), and 3-em dash (CMOS 13.113)</summary>
     <updated>2025-05-19T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -3332,10 +3333,6 @@
           <if match="any" variable="collection-editor part-number part-title volume-title">
             <text macro="label-volume"/>
           </if>
-          <else-if type="chapter">
-            <!-- remove condition in styles that provide chapter page numbers -->
-            <text macro="label-volume"/>
-          </else-if>
         </choose>
       </else-if>
       <else-if variable="volume">
@@ -3984,11 +3981,9 @@
           </if>
         </choose>
       </else-if>
-      <!-- if printing chapter page numbers:
       <else-if variable="chapter-number page">
         <text term="in" text-case="capitalize-first"/>
       </else-if>
-      -->
       <else-if variable="chapter-number">
         <group delimiter=" ">
           <text macro="label-chapter-number-capitalized"/>
@@ -4020,11 +4015,9 @@
           </if>
         </choose>
       </else-if>
-      <!-- if printing chapter page numbers:
       <else-if variable="chapter-number page">
         <text term="in"/>
       </else-if>
-      -->
       <else-if variable="chapter-number">
         <group delimiter=" ">
           <text macro="label-chapter-number"/>
@@ -4417,10 +4410,6 @@
             <if is-numeric="volume" match="none">
               <text macro="label-volume"/>
             </if>
-            <else-if type="chapter">
-              <!-- remove condition in styles that print chapter page numbers -->
-              <text macro="label-volume"/>
-            </else-if>
             <else-if is-numeric="volume" variable="page">
               <choose>
                 <!-- check for variables that might come between the volume and page number -->
@@ -4457,8 +4446,6 @@
       <else-if variable="locator">
         <text macro="label-locator"/>
       </else-if>
-      <!-- Remove `type="chapter"` condition in styles that provide chapter page numbers -->
-      <else-if type="chapter"/>
       <else-if is-numeric="volume" variable="page">
         <!-- collapse the volume and page number if adjacent -->
         <group delimiter=":">
@@ -5173,7 +5160,7 @@
         </if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text text-case="capitalize-first" variable="publisher-place"/> -->
+            <text text-case="capitalize-first" variable="publisher-place"/>
             <text text-case="capitalize-first" variable="publisher"/>
           </group>
         </else-if>
@@ -5199,7 +5186,7 @@
         </if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text variable="publisher-place"/> -->
+            <text variable="publisher-place"/>
             <text variable="publisher"/>
           </group>
         </else-if>
@@ -5221,7 +5208,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text text-case="capitalize-first" variable="original-publisher-place"/> -->
+          <text text-case="capitalize-first" variable="original-publisher-place"/>
           <text text-case="capitalize-first" variable="original-publisher"/>
         </group>
       </if>
@@ -5234,7 +5221,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text variable="original-publisher-place"/> -->
+          <text variable="original-publisher-place"/>
           <text variable="original-publisher"/>
         </group>
       </if>
@@ -6468,7 +6455,7 @@
   </macro>
   <citation disambiguate-add-givenname="true" disambiguate-add-names="true" et-al-min="3" et-al-use-first="1">
     <layout delimiter="; " suffix=".">
-      <text macro="citation-notes-full-subsequent-author-title"/>
+      <text macro="citation-notes-shortened-author-title"/>
     </layout>
   </citation>
   <!-- Bibliography -->
@@ -6626,7 +6613,7 @@
       </else>
     </choose>
   </macro>
-  <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true">
+  <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true" subsequent-author-substitute="&#8212;&#8212;&#8212;">
     <sort>
       <key macro="author-sort"/>
       <key macro="title-bib"/>
@@ -6637,7 +6624,7 @@
       <key macro="source-archive-bib"/>
     </sort>
     <layout suffix=".">
-      <text macro="bibliography-filtered-notes"/>
+      <text macro="bibliography-filtered-notes-shortened"/>
     </layout>
   </bibliography>
 </style>
