--- templates/chicago-template.csl
+++ development/chicago-notes-archive-place-first-no-url.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style class="note" demote-non-dropping-particle="display-and-sort" page-range-format="chicago-16" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
-    <title-short>CMOS with Bluebook</title-short>
-    <id>http://www.zotero.org/styles/chicago-notes-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="self"/>
+    <title>Chicago Manual of Style 18th edition (notes without bibliography, archival references with place first, without URLs)</title>
+    <title-short>CMOS with Bluebook (notes only [13.23, 14.20], archive place first [14.205], no URLs [13.6])</title-short>
+    <id>http://www.zotero.org/styles/chicago-notes-archive-place-first-no-url</id>
+    <link href="http://www.zotero.org/styles/chicago-notes-archive-place-first-no-url" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="template"/>
     <link href="https://www.chicagomanualofstyle.org/" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -30,7 +31,7 @@
     <category field="social_science"/>
     <category field="sociology"/>
     <category field="theology"/>
-    <summary>Chicago-style source citations (with Bluebook for legal citations), template</summary>
+    <summary>Chicago-style source citations (with Bluebook for legal citations), notes system without bibliography, listing up to six authors in a full citation (CMOS18 13.23), displaying number of volumes (CMOS18 14.20), archival references with place first (CMOS18 14.205), without URLs (CMOS18 13.6)</summary>
     <updated>2025-10-27T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -5958,7 +5959,7 @@
           <text macro="source-archive-reference-institution-first"/>
         </if>
         <else>
-          <text macro="source-archive-reference-location-first-bib"/>
+          <text macro="source-archive-reference-place-first"/>
         </else>
       </choose>
       <text macro="source-archive-database-number" prefix="(" suffix=")"/>
@@ -5971,7 +5972,7 @@
           <text macro="source-archive-reference-institution-first"/>
         </if>
         <else>
-          <text macro="source-archive-reference-location-first-note"/>
+          <text macro="source-archive-reference-place-first"/>
         </else>
       </choose>
       <text macro="source-archive-database-number" prefix="(" suffix=")"/>
@@ -6076,7 +6077,7 @@
           </group>
         </else-if>
       </choose>
-      <text macro="source-DOI-URL"/>
+      <!-- <text macro="source-DOI-URL"/> -->
     </group>
   </macro>
   <macro name="source-date-accessed-DOI-URL-note">
@@ -6091,7 +6092,7 @@
           </group>
         </else-if>
       </choose>
-      <text macro="source-DOI-URL"/>
+      <!-- <text macro="source-DOI-URL"/> -->
     </group>
   </macro>
   <macro name="source-DOI-URL">
@@ -6778,7 +6779,7 @@
       </else>
     </choose>
   </macro>
-  <citation disambiguate-add-givenname="true" disambiguate-add-names="true" et-al-min="3" et-al-use-first="1">
+  <citation disambiguate-add-givenname="true" disambiguate-add-names="true" et-al-min="7" et-al-subsequent-min="3" et-al-subsequent-use-first="1" et-al-use-first="3">
     <layout delimiter="; " suffix=".">
       <text macro="citation-notes-full-subsequent-author-title"/>
     </layout>
@@ -6913,19 +6914,4 @@
       </else>
     </choose>
   </macro>
-  <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true">
-    <sort>
-      <key macro="author-sort"/>
-      <key macro="title-and-descriptions-bib"/>
-      <key macro="source-bib"/>
-      <key variable="volume"/>
-      <key variable="part-number"/>
-      <key variable="event-date"/>
-      <key macro="source-date-issued-or-status"/>
-      <key macro="source-archive-bib"/>
-    </sort>
-    <layout suffix=".">
-      <text macro="bibliography-filtered-notes"/>
-    </layout>
-  </bibliography>
 </style>
