--- templates/chicago-template.csl
+++ development/chicago-author-date-17th-edition.csl
@@ -3,18 +3,19 @@
   type="application/relax-ng-compact-syntax" schematypens="http://relaxng.org/ns/compatibility/annotations/1.0"?>
 <?xml-model href="https://raw.githubusercontent.com/citation-style-language/schema/refs/heads/master/schemas/styles/csl-repository.rnc"
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-<style class="note" demote-non-dropping-particle="display-and-sort" page-range-format="chicago-16" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
+<style class="in-text" demote-non-dropping-particle="display-and-sort" page-range-format="chicago-16" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
-    <title-short>CMOS with Bluebook</title-short>
-    <id>http://www.zotero.org/styles/chicago-notes-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="self"/>
+    <title>Chicago Manual of Style 17th edition (author-date)</title>
+    <title-short>CMOS 17e with Bluebook (author-date [15.5])</title-short>
+    <id>http://www.zotero.org/styles/chicago-author-date-17th-edition</id>
+    <link href="http://www.zotero.org/styles/chicago-author-date-17th-edition" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="template"/>
     <link href="https://www.chicagomanualofstyle.org/" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
       <uri>https://orcid.org/0000-0003-0464-5036</uri>
     </author>
-    <category citation-format="note"/>
+    <category citation-format="author-date"/>
     <category field="anthropology"/>
     <category field="communications"/>
     <category field="generic-base"/>
@@ -30,7 +31,7 @@
     <category field="social_science"/>
     <category field="sociology"/>
     <category field="theology"/>
-    <summary>Chicago-style source citations (with Bluebook for legal citations), template</summary>
+    <summary>Chicago-style source citations (with Bluebook for legal citations), author-date system, 17th edition</summary>
     <updated>2025-10-27T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -63,9 +64,6 @@
       <term name="manuscript">unpublished manuscript</term>
       <term name="original-work-published">originally published as</term>
       <term form="short" name="paper-conference">paper</term>
-      <!-- 'under' replaces 's.v.' from CMOS17 and earlier (CMOS18 14.130) -->
-      <term name="sub-verbo">under</term>
-      <term form="short" name="sub-verbo">under</term>
       <term name="timestamp">at</term>
       <term form="verb-short" name="translator">trans.</term>
     </terms>
@@ -3332,10 +3330,6 @@
           <if match="any" variable="collection-editor part-number part-title volume-title">
             <text macro="label-volume"/>
           </if>
-          <else-if variable="container-title">
-            <!-- remove condition in styles that print chapter page numbers (CMOS17/classic) -->
-            <text macro="label-volume"/>
-          </else-if>
         </choose>
       </else-if>
       <else-if variable="volume">
@@ -3602,9 +3596,7 @@
       <if match="any" type="article-magazine article-newspaper">
         <group delimiter=", ">
           <group delimiter=". ">
-            <text macro="source-serial-identifier-volume-bib"/>
-            <!-- for author-date: -->
-            <!-- <text macro="source-serial-identifier-volume-author-date"/> -->
+            <text macro="source-serial-identifier-volume-author-date"/>
             <!-- newspaper edition always capitalized (CMOS18 14.89) -->
             <text macro="label-edition-capitalized"/>
           </group>
@@ -3613,17 +3605,13 @@
       </if>
       <else-if match="any" variable="issue supplement-number volume">
         <group delimiter=": ">
-          <text macro="source-serial-identifier-volume-bib"/>
-          <!-- for author-date: -->
-          <!-- <text macro="source-serial-identifier-volume-author-date"/> -->
+          <text macro="source-serial-identifier-volume-author-date"/>
           <text macro="source-serial-locator"/>
         </group>
       </else-if>
       <else>
         <group delimiter=", ">
-          <text macro="source-serial-identifier-volume-bib"/>
-          <!-- for author-date: -->
-          <!-- <text macro="source-serial-identifier-volume-author-date"/> -->
+          <text macro="source-serial-identifier-volume-author-date"/>
           <text macro="source-serial-locator"/>
         </group>
       </else>
@@ -3677,9 +3665,7 @@
                     <!-- nothing to substitute if there is no issue/supplement/volume -->
                     <else-if match="none" variable="issue volume"/>
                     <else>
-                      <text macro="source-date-bib"/>
-                      <!-- for CMOS17 author-date: -->
-                      <!-- <text macro="source-date-issued-month-day"/> -->
+                      <text macro="source-date-issued-month-day"/>
                     </else>
                   </choose>
                 </if>
@@ -3721,16 +3707,12 @@
                       <!-- date for unsigned reviews only appears here if it did not earlier (CMOS18 14.102) -->
                       <choose>
                         <if match="any" variable="author chair collection-editor compiler composer curator director editor editor-translator editorial-director executive-producer guest host illustrator organizer producer series-creator translator">
-                          <text macro="source-date-bib"/>
-                          <!-- for CMOS17 author-date: -->
-                          <!-- <text macro="source-date-issued-month-day"/> -->
+                          <text macro="source-date-issued-month-day"/>
                         </if>
                       </choose>
                     </else-if>
                     <else>
-                      <text macro="source-date-bib"/>
-                      <!-- for CMOS17 author-date: -->
-                      <!-- <text macro="source-date-issued-month-day"/> -->
+                      <text macro="source-date-issued-month-day"/>
                     </else>
                   </choose>
                 </group>
@@ -3743,9 +3725,7 @@
                   </group>
                   <choose>
                     <if match="any" variable="author chair collection-editor compiler composer curator director editor editor-translator editorial-director executive-producer guest host illustrator organizer producer series-creator translator">
-                      <text macro="source-date-bib" prefix="(" suffix=")"/>
-                      <!-- for CMOS17 author-date: -->
-                      <!-- <text macro="source-date-issued-month-day" prefix="(" suffix=")"/> -->
+                      <text macro="source-date-issued-month-day" prefix="(" suffix=")"/>
                     </if>
                   </choose>
                 </group>
@@ -3780,9 +3760,7 @@
               <!-- date appears first if the review `container-title` has been substituted for a missing author (CMOS18 14.87, 14.102) -->
               <choose>
                 <if match="none" variable="author chair collection-editor compiler composer curator director editor editor-translator editorial-director executive-producer guest host illustrator organizer producer series-creator translator">
-                  <text macro="source-date-bib"/>
-                  <!-- for CMOS17 author-date: -->
-                  <!-- <text macro="source-date-issued-month-day"/> -->
+                  <text macro="source-date-issued-month-day"/>
                 </if>
               </choose>
             </if>
@@ -3855,9 +3833,7 @@
                 <else-if match="any" variable="reviewed-genre reviewed-title title"/>
                 <else-if match="none" variable="genre section"/>
                 <else>
-                  <text macro="source-date-issued-or-status"/>
-                  <!-- for CMOS17 author-date: -->
-                  <!-- <text macro="source-date-issued-month-day"/> -->
+                  <text macro="source-date-issued-month-day"/>
                 </else>
               </choose>
             </if>
@@ -3903,26 +3879,18 @@
                 <!-- date appears first if a review `container-title` has been substituted for a missing author (CMOS18 14.89) -->
                 <choose>
                   <if match="any" variable="author chair collection-editor compiler composer curator director editor editor-translator editorial-director executive-producer guest host illustrator organizer producer series-creator translator">
-                    <text macro="source-date-issued-or-status" prefix="(" suffix=")"/>
-                    <!-- for CMOS17 author-date: -->
-                    <!-- <text macro="source-date-issued-month-day" prefix="(" suffix=")"/> -->
+                    <text macro="source-date-issued-month-day" prefix="(" suffix=")"/>
                   </if>
                   <else-if match="any" variable="reviewed-genre reviewed-title title">
-                    <text macro="source-date-issued-or-status" prefix="(" suffix=")"/>
-                    <!-- for CMOS17 author-date: -->
-                    <!-- <text macro="source-date-issued-month-day" prefix="(" suffix=")"/> -->
+                    <text macro="source-date-issued-month-day" prefix="(" suffix=")"/>
                   </else-if>
                   <else-if match="none" variable="genre section">
-                    <text macro="source-date-issued-or-status" prefix="(" suffix=")"/>
-                    <!-- for CMOS17 author-date: -->
-                    <!-- <text macro="source-date-issued-month-day" prefix="(" suffix=")"/> -->
+                    <text macro="source-date-issued-month-day" prefix="(" suffix=")"/>
                   </else-if>
                 </choose>
               </if>
               <else>
-                <text macro="source-date-issued-or-status" prefix="(" suffix=")"/>
-                <!-- for CMOS17 author-date: -->
-                <!-- <text macro="source-date-issued-month-day" prefix="(" suffix=")"/> -->
+                <text macro="source-date-issued-month-day" prefix="(" suffix=")"/>
               </else>
             </choose>
           </group>
@@ -3950,9 +3918,7 @@
       </choose>
       <group delimiter=" ">
         <text macro="source-date-status-bib"/>
-        <text macro="source-date-bib"/>
-        <!-- for CMOS17 author-date: -->
-        <!-- <text macro="source-date-issued-full-serial"/> -->
+        <text macro="source-date-issued-full-serial"/>
       </group>
     </group>
   </macro>
@@ -4059,11 +4025,9 @@
           </if>
         </choose>
       </else-if>
-      <!-- if printing chapter page numbers (CMOS17/classic):
       <else-if variable="chapter-number page title">
         <text term="in" text-case="capitalize-first"/>
       </else-if>
-      -->
       <else-if variable="chapter-number">
         <group delimiter=" ">
           <text macro="label-chapter-number-capitalized"/>
@@ -4095,11 +4059,9 @@
           </if>
         </choose>
       </else-if>
-      <!-- if printing chapter page numbers (CMOS17/classic):
       <else-if variable="chapter-number page title">
         <text term="in"/>
       </else-if>
-      -->
       <else-if variable="chapter-number">
         <group delimiter=" ">
           <text macro="label-chapter-number"/>
@@ -4492,10 +4454,6 @@
             <if is-numeric="volume" match="none">
               <text macro="label-volume"/>
             </if>
-            <else-if variable="container-title">
-              <!-- remove condition in styles that print chapter page numbers (CMOS17/classic) -->
-              <text macro="label-volume"/>
-            </else-if>
             <else-if is-numeric="volume" variable="page">
               <choose>
                 <!-- check for variables that might come between the volume and page number -->
@@ -4536,8 +4494,6 @@
       <else-if variable="locator">
         <text macro="label-locator"/>
       </else-if>
-      <!-- remove `container-title` condition in styles that print chapter page numbers (CMOS17/classic) -->
-      <else-if variable="container-title"/>
       <else-if is-numeric="volume" variable="page">
         <!-- collapse the volume and page number if adjacent -->
         <group delimiter=":">
@@ -5304,7 +5260,7 @@
         </if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text text-case="capitalize-first" variable="publisher-place"/> -->
+            <text text-case="capitalize-first" variable="publisher-place"/>
             <text text-case="capitalize-first" variable="publisher"/>
           </group>
         </else-if>
@@ -5330,7 +5286,7 @@
         </if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text variable="publisher-place"/> -->
+            <text variable="publisher-place"/>
             <text variable="publisher"/>
           </group>
         </else-if>
@@ -5352,7 +5308,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text text-case="capitalize-first" variable="original-publisher-place"/> -->
+          <text text-case="capitalize-first" variable="original-publisher-place"/>
           <text text-case="capitalize-first" variable="original-publisher"/>
         </group>
       </if>
@@ -5365,7 +5321,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text variable="original-publisher-place"/> -->
+          <text variable="original-publisher-place"/>
           <text variable="original-publisher"/>
         </group>
       </if>
@@ -5387,9 +5343,7 @@
         <date date-parts="year" form="text" variable="available-date"/>
       </else-if>
       <else>
-        <text macro="source-date-issued-or-status"/>
-        <!-- for author-date: -->
-        <!-- <text macro="source-date-issued-month-day"/> -->
+        <text macro="source-date-issued-month-day"/>
       </else>
     </choose>
   </macro>
@@ -5450,14 +5404,10 @@
         <text macro="date-issued-full"/>
       </if>
       <else-if variable="issue volume">
-        <text macro="date-issued-year"/>
-        <!-- for CMOS17: -->
-        <!-- <text macro="date-issued-year-month"/> -->
+        <text macro="date-issued-year-month"/>
       </else-if>
       <else-if variable="supplement-number">
-        <text macro="date-issued-year"/>
-        <!-- for CMOS17: -->
-        <!-- <text macro="date-issued-year-month"/> -->
+        <text macro="date-issued-year-month"/>
       </else-if>
       <else-if match="any" variable="issue volume">
         <text macro="date-issued-year-month"/>
@@ -5606,14 +5556,10 @@
   <macro name="source-date-original-full-serial">
     <choose>
       <if variable="issue volume">
-        <text macro="date-original-year"/>
-        <!-- for CMOS17: -->
-        <!-- <text macro="date-original-year-month"/> -->
+        <text macro="date-original-year-month"/>
       </if>
       <else-if variable="supplement-number">
-        <text macro="date-original-year"/>
-        <!-- for CMOS17: -->
-        <!-- <text macro="date-original-year-month"/> -->
+        <text macro="date-original-year-month"/>
       </else-if>
       <else-if match="any" variable="issue volume">
         <text macro="date-original-year-month"/>
@@ -6672,9 +6618,9 @@
       </else>
     </choose>
   </macro>
-  <citation disambiguate-add-givenname="true" disambiguate-add-names="true" et-al-min="3" et-al-use-first="1">
-    <layout delimiter="; " suffix=".">
-      <text macro="citation-notes-full-subsequent-author-title"/>
+  <citation after-collapse-delimiter="; " collapse="year" disambiguate-add-givenname="true" disambiguate-add-names="true" disambiguate-add-year-suffix="true" et-al-min="4" et-al-use-first="1">
+    <layout delimiter="; " prefix="(" suffix=")">
+      <text macro="citation-author-date"/>
     </layout>
   </citation>
   <!-- Bibliography -->
@@ -6822,19 +6768,22 @@
       </else>
     </choose>
   </macro>
-  <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true">
+  <bibliography et-al-min="11" et-al-use-first="7" hanging-indent="true" subsequent-author-substitute="&#8212;&#8212;&#8212;">
     <sort>
       <key macro="author-sort"/>
+      <key macro="date-sort-group"/>
+      <key macro="date-sort-year"/>
+      <key macro="date"/>
       <key macro="title-and-descriptions-bib"/>
       <key macro="source-bib"/>
       <key variable="volume"/>
       <key variable="part-number"/>
       <key variable="event-date"/>
-      <key macro="source-date-issued-or-status"/>
+      <key variable="issued"/>
       <key macro="source-archive-bib"/>
     </sort>
     <layout suffix=".">
-      <text macro="bibliography-filtered-notes"/>
+      <text macro="bibliography-filtered-author-date"/>
     </layout>
   </bibliography>
 </style>
