--- templates/chicago-template.csl
+++ development/chicago-fullnote.csl
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style and="text" class="note" demote-non-dropping-particle="display-and-sort" names-delimiter=", " page-range-format="chicago" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
+    <title>Chicago Manual of Style 18th edition (full note without bibliography)</title>
     <title-short>CMOS</title-short>
-    <id>http://www.zotero.org/styles/chicago-fullnote-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography" rel="self"/>
-    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography-17th-edition" rel="template"/>
+    <id>http://www.zotero.org/styles/chicago-fullnote</id>
+    <link href="http://www.zotero.org/styles/chicago-fullnote" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography" rel="template"/>
     <link href="http://www.chicagomanualofstyle.org/tools_citationguide.html" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -55,7 +55,7 @@
     <category field="humanities"/>
     <category field="social_science"/>
     <category field="generic-base"/>
-    <summary>Chicago Manual of Style full notes and bibliography</summary>
+    <summary>Chicago Manual of Style full notes without bibliography</summary>
     <updated>2025-03-01T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -2879,7 +2879,7 @@
       </choose>
     </group>
   </macro>
-  <citation disambiguate-add-names="true" et-al-min="3" et-al-use-first="1">
+  <citation disambiguate-add-names="true" et-al-min="7" et-al-use-first="3">
     <layout delimiter="; " suffix=".">
       <text macro="citation-fullnote"/>
     </layout>
@@ -2953,15 +2953,4 @@
       </else>
     </choose>
   </macro>
-  <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true">
-    <sort>
-      <key macro="author-bib"/>
-      <key variable="title"/>
-      <key variable="genre"/>
-      <key variable="issued"/>
-    </sort>
-    <layout suffix=".">
-      <text macro="bibliography-filtered"/>
-    </layout>
-  </bibliography>
 </style>
