--- templates/chicago-template.csl
+++ development/chicago-notes-classic-no-url.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style and="text" class="note" demote-non-dropping-particle="display-and-sort" page-range-format="chicago" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
+    <title>Chicago Manual of Style 18th edition (notes without bibliography, with place of publication, without URLs)</title>
     <title-short>CMOS</title-short>
-    <id>http://www.zotero.org/styles/chicago-notes-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="self"/>
+    <id>http://www.zotero.org/styles/chicago-notes-classic-no-url</id>
+    <link href="http://www.zotero.org/styles/chicago-notes-classic-no-url" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="template"/>
     <link href="http://www.chicagomanualofstyle.org/tools_citationguide.html" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -22,21 +23,7 @@
     <category field="literature"/>
     <category field="philosophy"/>
     <category field="theology"/>
-    <!-- Categories for author-date:
-    <category citation-format="author-date"/>
-    <category field="anthropology"/>
-    <category field="communications"/>
-    <category field="generic-base"/>
-    <category field="geography"/>
-    <category field="history"/>
-    <category field="humanities"/>
-    <category field="linguistics"/>
-    <category field="political_science"/>
-    <category field="science"/>
-    <category field="social_science"/>
-    <category field="sociology"/>
-    -->
-    <summary>Chicago-style source citations (with Bluebook for legal citations), template</summary>
+    <summary>Chicago-style source citations (with Bluebook for legal citations), notes system without bibliography, listing up to six authors in a full citation (CMOS 13.23), with classic variants: chapter page numbers (CMOS 14.8), place of publication (CMOS 14.30); without URLs (CMOS 13.6)</summary>
     <updated>2025-05-19T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -1186,7 +1173,7 @@
       <text macro="source-monographic-publication-bib"/>
       <text macro="source-medium-bib"/>
       <text macro="source-archive-bib"/>
-      <text macro="source-DOI-URL-bib"/>
+      <!-- <text macro="source-DOI-URL-bib"/> -->
     </group>
   </macro>
   <macro name="title-and-source-note">
@@ -1239,7 +1226,7 @@
       </choose>
       <text macro="source-medium-note"/>
       <text macro="source-archive-note"/>
-      <text macro="source-DOI-URL-note"/>
+      <!-- <text macro="source-DOI-URL-note"/> -->
     </group>
   </macro>
   <!-- 3.1. Title -->
@@ -3596,11 +3583,9 @@
           </if>
         </choose>
       </else-if>
-      <!-- if printing chapter page numbers:
       <else-if variable="chapter-number page">
         <text term="in" text-case="capitalize-first"/>
       </else-if>
-      -->
       <else-if variable="chapter-number">
         <group delimiter=" ">
           <text macro="label-chapter-number-capitalized"/>
@@ -3632,11 +3617,9 @@
           </if>
         </choose>
       </else-if>
-      <!-- if printing chapter page numbers:
       <else-if variable="chapter-number page">
         <text term="in"/>
       </else-if>
-      -->
       <else-if variable="chapter-number">
         <group delimiter=" ">
           <text macro="label-chapter-number"/>
@@ -4015,10 +3998,6 @@
             <if is-numeric="volume" match="none">
               <text macro="label-volume"/>
             </if>
-            <else-if type="chapter">
-              <!-- remove condition in styles that print chapter page numbers -->
-              <text macro="label-volume"/>
-            </else-if>
             <else-if is-numeric="volume" variable="page">
               <choose>
                 <!-- check for variables that might come between the volume and page number -->
@@ -4758,7 +4737,7 @@
         </if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text text-case="capitalize-first" variable="publisher-place"/> -->
+            <text text-case="capitalize-first" variable="publisher-place"/>
             <text text-case="capitalize-first" variable="publisher"/>
           </group>
         </else-if>
@@ -4784,7 +4763,7 @@
         </if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text variable="publisher-place"/> -->
+            <text variable="publisher-place"/>
             <text variable="publisher"/>
           </group>
         </else-if>
@@ -4806,7 +4785,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text text-case="capitalize-first" variable="original-publisher-place"/> -->
+          <text text-case="capitalize-first" variable="original-publisher-place"/>
           <text text-case="capitalize-first" variable="original-publisher"/>
         </group>
       </if>
@@ -4819,7 +4798,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text variable="original-publisher-place"/> -->
+          <text variable="original-publisher-place"/>
           <text variable="original-publisher"/>
         </group>
       </if>
@@ -5446,7 +5425,7 @@
         </else>
       </choose>
       <text variable="references"/>
-      <text macro="source-DOI-URL"/>
+      <!-- <text macro="source-DOI-URL"/> -->
     </group>
   </macro>
   <!-- 6.1. Legal author -->
@@ -6014,7 +5993,7 @@
       </else>
     </choose>
   </macro>
-  <citation disambiguate-add-givenname="true" disambiguate-add-names="true" et-al-min="3" et-al-use-first="1">
+  <citation disambiguate-add-givenname="true" disambiguate-add-names="true" et-al-min="7" et-al-subsequent-min="3" et-al-subsequent-use-first="1" et-al-use-first="3">
     <layout delimiter="; " suffix=".">
       <text macro="citation-notes-full-subsequent-author-title"/>
     </layout>
@@ -6174,18 +6153,4 @@
       </else>
     </choose>
   </macro>
-  <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true">
-    <sort>
-      <key macro="author-sort"/>
-      <key macro="title-bib"/>
-      <key variable="part-number"/>
-      <key variable="volume"/>
-      <key macro="date-sort-group"/>
-      <key macro="date-sort"/>
-      <key variable="status"/>
-    </sort>
-    <layout suffix=".">
-      <text macro="bibliography-filtered-notes"/>
-    </layout>
-  </bibliography>
 </style>
