--- templates/chicago-template.csl
+++ development/chicago-notes-bibliography-annotated-abstract.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style class="note" demote-non-dropping-particle="display-and-sort" page-range-format="chicago-16" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
-    <title-short>CMOS with Bluebook</title-short>
-    <id>http://www.zotero.org/styles/chicago-notes-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="self"/>
+    <title>Chicago Manual of Style 18th edition (notes and annotated bibliography with abstract)</title>
+    <title-short>CMOS with Bluebook (notes and annotated bibliography with abstract [13.68])</title-short>
+    <id>http://www.zotero.org/styles/chicago-notes-bibliography-annotated-abstract</id>
+    <link href="http://www.zotero.org/styles/chicago-notes-bibliography-annotated-abstract" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="template"/>
     <link href="https://www.chicagomanualofstyle.org/" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -30,7 +31,7 @@
     <category field="social_science"/>
     <category field="sociology"/>
     <category field="theology"/>
-    <summary>Chicago-style source citations (with Bluebook for legal citations), template</summary>
+    <summary>Chicago-style source citations (with Bluebook for legal citations), notes and bibliography system, bibliography annotated with abstracts (CMOS18 13.68)</summary>
     <updated>2025-10-27T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -7174,8 +7175,9 @@
       <key macro="source-date-issued-or-status"/>
       <key macro="source-archive-bib"/>
     </sort>
-    <layout suffix=".">
-      <text macro="bibliography-filtered-notes"/>
+    <layout>
+      <text macro="bibliography-notes" suffix="."/>
+      <text display="block" variable="abstract"/>
     </layout>
   </bibliography>
 </style>
