--- templates/chicago-template.csl
+++ development/chicago-shortened-notes-bibliography-access-dates.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style class="note" demote-non-dropping-particle="display-and-sort" page-range-format="chicago-16" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
-    <title-short>CMOS with Bluebook</title-short>
-    <id>http://www.zotero.org/styles/chicago-notes-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="self"/>
+    <title>Chicago Manual of Style 18th edition (shortened notes and bibliography, access dates)</title>
+    <title-short>CMOS with Bluebook (shortened author-title notes [13.32], access dates [13.15])</title-short>
+    <id>http://www.zotero.org/styles/chicago-shortened-notes-bibliography-access-dates</id>
+    <link href="http://www.zotero.org/styles/chicago-shortened-notes-bibliography-access-dates" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="template"/>
     <link href="https://www.chicagomanualofstyle.org/" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -30,7 +31,7 @@
     <category field="social_science"/>
     <category field="sociology"/>
     <category field="theology"/>
-    <summary>Chicago-style source citations (with Bluebook for legal citations), template</summary>
+    <summary>Chicago-style source citations (with Bluebook for legal citations), shortened author-title notes and bibliography system, with access dates for all electronic sources (CMOS18 13.15)</summary>
     <updated>2025-10-27T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -5953,9 +5954,8 @@
   <macro name="source-date-accessed-DOI-URL-bib">
     <group delimiter=". ">
       <choose>
-        <if variable="DOI"/>
-        <else-if match="any" variable="available-date event-date issued status"/>
-        <else-if variable="accessed URL">
+        <if match="none" variable="DOI URL"/>
+        <else-if variable="accessed">
           <group delimiter=" ">
             <text term="accessed" text-case="capitalize-first"/>
             <date form="text" variable="accessed"/>
@@ -6685,7 +6685,7 @@
   </macro>
   <citation disambiguate-add-givenname="true" disambiguate-add-names="true" et-al-min="3" et-al-use-first="1">
     <layout delimiter="; " suffix=".">
-      <text macro="citation-notes-full-subsequent-author-title"/>
+      <text macro="citation-notes-shortened-author-title"/>
     </layout>
   </citation>
   <!-- Bibliography -->
@@ -6845,7 +6845,7 @@
       <key macro="source-archive-bib"/>
     </sort>
     <layout suffix=".">
-      <text macro="bibliography-filtered-notes"/>
+      <text macro="bibliography-filtered-notes-shortened"/>
     </layout>
   </bibliography>
 </style>
