--- templates/chicago-template.csl
+++ development/chicago-notes.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style class="note" demote-non-dropping-particle="display-and-sort" page-range-format="chicago-16" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
-    <title-short>CMOS with Bluebook</title-short>
-    <id>http://www.zotero.org/styles/chicago-notes-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="self"/>
+    <title>Chicago Manual of Style 18th edition (notes without bibliography)</title>
+    <title-short>CMOS with Bluebook (notes only [13.23, 14.20])</title-short>
+    <id>http://www.zotero.org/styles/chicago-notes</id>
+    <link href="http://www.zotero.org/styles/chicago-notes" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="template"/>
     <link href="https://www.chicagomanualofstyle.org/" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -30,7 +31,7 @@
     <category field="social_science"/>
     <category field="sociology"/>
     <category field="theology"/>
-    <summary>Chicago-style source citations (with Bluebook for legal citations), template</summary>
+    <summary>Chicago-style source citations (with Bluebook for legal citations), notes system without bibliography, listing up to six authors in a full citation (CMOS18 13.23), displaying number of volumes (CMOS18 14.20)</summary>
     <updated>2025-10-27T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -426,12 +427,12 @@
       </choose>
       <text variable="volume"/>
       <!-- always display number of volumes in notes without bibliography (CMOS18 14.20): -->
-      <!-- <choose>
+      <choose>
         <if is-numeric="number-of-volumes" variable="volume">
           <text value="of"/>
           <text variable="number-of-volumes"/>
         </if>
-      </choose> -->
+      </choose>
     </group>
   </macro>
   <macro name="label-volume-capitalized">
@@ -3313,9 +3314,6 @@
       <choose>
         <!-- notes only show the number of volumes if there is no physical locator, which should include a volume number, e.g. 4:243; except when using notes without bibliography (CMOS18 14.20) -->
         <if match="any" variable="part-number part-title volume volume-title"/>
-        <else-if locator="column"/>
-        <else-if locator="folio"/>
-        <else-if locator="page"/>
         <else>
           <text macro="label-number-of-volumes"/>
         </else>
@@ -3551,9 +3549,9 @@
               <text macro="label-volume"/>
             </else-if>
             <!-- always display number of volumes in notes without bibliography (CMOS18 14.20): -->
-            <!-- <else>
+            <else>
               <text macro="label-number-of-volumes"/>
-            </else> -->
+            </else>
           </choose>
         </else-if>
         <else-if is-numeric="volume" variable="page">
@@ -3569,9 +3567,9 @@
               <text macro="label-volume"/>
             </else-if>
             <!-- always display number of volumes in notes without bibliography (CMOS18 14.20): -->
-            <!-- <else>
+            <else>
               <text macro="label-number-of-volumes"/>
-            </else> -->
+            </else>
           </choose>
         </else-if>
         <else-if variable="volume">
@@ -6874,7 +6872,7 @@
       </else>
     </choose>
   </macro>
-  <citation disambiguate-add-givenname="true" disambiguate-add-names="true" et-al-min="3" et-al-use-first="1">
+  <citation disambiguate-add-givenname="true" disambiguate-add-names="true" et-al-min="7" et-al-subsequent-min="3" et-al-subsequent-use-first="1" et-al-use-first="3">
     <layout delimiter="; " suffix=".">
       <text macro="citation-notes-full-subsequent-author-title"/>
     </layout>
@@ -7009,19 +7007,4 @@
       </else>
     </choose>
   </macro>
-  <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true">
-    <sort>
-      <key macro="author-sort"/>
-      <key macro="title-and-descriptions-bib"/>
-      <key macro="source-bib"/>
-      <key variable="volume"/>
-      <key variable="part-number"/>
-      <key variable="event-date"/>
-      <key macro="source-date-issued-or-status"/>
-      <key macro="source-archive-bib"/>
-    </sort>
-    <layout suffix=".">
-      <text macro="bibliography-filtered-notes"/>
-    </layout>
-  </bibliography>
 </style>
