--- templates/chicago-template.csl
+++ development/chicago-shortened-notes-bibliography-classic-no-url.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style class="note" demote-non-dropping-particle="display-and-sort" page-range-format="chicago-16" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
-    <title-short>CMOS with Bluebook</title-short>
-    <id>http://www.zotero.org/styles/chicago-notes-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="self"/>
+    <title>Chicago Manual of Style 18th edition (shortened notes and bibliography, classic variants, without URLs)</title>
+    <title-short>CMOS with Bluebook (shortened author-title notes [13.32], classic variants [13.113, 14.8, 14.30], no URLs [13.6])</title-short>
+    <id>http://www.zotero.org/styles/chicago-shortened-notes-bibliography-classic-no-url</id>
+    <link href="http://www.zotero.org/styles/chicago-shortened-notes-bibliography-classic-no-url" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="template"/>
     <link href="https://www.chicagomanualofstyle.org/" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -30,7 +31,7 @@
     <category field="social_science"/>
     <category field="sociology"/>
     <category field="theology"/>
-    <summary>Chicago-style source citations (with Bluebook for legal citations), template</summary>
+    <summary>Chicago-style source citations (with Bluebook for legal citations), shortened author-title notes and bibliography system, with classic variants: 3-em dash for repeated names (CMOS18 13.113), chapter page numbers (CMOS18 14.8), and place of publication (CMOS18 14.30); without URLs (CMOS18 13.6)</summary>
     <updated>2025-10-27T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -3519,10 +3520,6 @@
             <else-if match="any" variable="container-author editor editor-translator translator">
               <text macro="label-volume"/>
             </else-if>
-            <else-if variable="container-title">
-              <!-- remove condition in styles that print chapter page numbers (CMOS17/classic) -->
-              <text macro="label-volume"/>
-            </else-if>
             <!-- always display number of volumes in notes without bibliography (CMOS18 14.20): -->
             <!-- <else>
               <text macro="label-number-of-volumes"/>
@@ -4188,11 +4185,9 @@
           </if>
         </choose>
       </else-if>
-      <!-- if printing chapter page numbers (CMOS17/classic):
       <else-if variable="chapter-number page title">
         <text term="in" text-case="capitalize-first"/>
       </else-if>
-      -->
       <else-if variable="chapter-number">
         <group delimiter=" ">
           <text macro="label-chapter-number-capitalized"/>
@@ -4227,11 +4222,9 @@
           </if>
         </choose>
       </else-if>
-      <!-- if printing chapter page numbers (CMOS17/classic):
       <else-if variable="chapter-number page title">
         <text term="in"/>
       </else-if>
-      -->
       <else-if variable="chapter-number">
         <group delimiter=" ">
           <text macro="label-chapter-number"/>
@@ -4600,10 +4593,6 @@
         <else-if is-numeric="volume" match="none">
           <text macro="label-volume"/>
         </else-if>
-        <else-if variable="container-title">
-          <!-- remove condition in styles that print chapter page numbers (CMOS17/classic) -->
-          <text macro="label-volume"/>
-        </else-if>
         <else-if is-numeric="volume" variable="page">
           <choose>
             <!-- check for variables that might come between the volume and page number -->
@@ -4643,8 +4632,6 @@
       <else-if variable="locator">
         <text macro="label-locator"/>
       </else-if>
-      <!-- remove `container-title` condition in styles that print chapter page numbers (CMOS17/classic) -->
-      <else-if variable="container-title"/>
       <else-if is-numeric="volume" variable="page">
         <!-- collapse the volume and page number if adjacent (CMOS18 14.18) -->
         <group delimiter=":">
@@ -5409,7 +5396,7 @@
         </if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text text-case="capitalize-first" variable="publisher-place"/> -->
+            <text text-case="capitalize-first" variable="publisher-place"/>
             <text text-case="capitalize-first" variable="publisher"/>
           </group>
         </else-if>
@@ -5435,7 +5422,7 @@
         </if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text variable="publisher-place"/> -->
+            <text variable="publisher-place"/>
             <text variable="publisher"/>
           </group>
         </else-if>
@@ -5457,7 +5444,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text text-case="capitalize-first" variable="original-publisher-place"/> -->
+          <text text-case="capitalize-first" variable="original-publisher-place"/>
           <text text-case="capitalize-first" variable="original-publisher"/>
         </group>
       </if>
@@ -5470,7 +5457,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text variable="original-publisher-place"/> -->
+          <text variable="original-publisher-place"/>
           <text variable="original-publisher"/>
         </group>
       </if>
@@ -6054,7 +6041,7 @@
           </group>
         </else-if>
       </choose>
-      <text macro="source-DOI-URL"/>
+      <!-- <text macro="source-DOI-URL"/> -->
     </group>
   </macro>
   <macro name="source-date-accessed-DOI-URL-note">
@@ -6069,7 +6056,7 @@
           </group>
         </else-if>
       </choose>
-      <text macro="source-DOI-URL"/>
+      <!-- <text macro="source-DOI-URL"/> -->
     </group>
   </macro>
   <macro name="source-DOI-URL">
@@ -6758,7 +6745,7 @@
   </macro>
   <citation disambiguate-add-givenname="true" disambiguate-add-names="true" et-al-min="3" et-al-use-first="1">
     <layout delimiter="; " suffix=".">
-      <text macro="citation-notes-full-subsequent-author-title"/>
+      <text macro="citation-notes-shortened-author-title"/>
     </layout>
   </citation>
   <!-- Bibliography -->
@@ -6891,7 +6878,7 @@
       </else>
     </choose>
   </macro>
-  <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true">
+  <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true" subsequent-author-substitute="&#8212;&#8212;&#8212;">
     <sort>
       <key macro="author-sort"/>
       <key macro="title-and-descriptions-bib"/>
@@ -6903,7 +6890,7 @@
       <key macro="source-archive-bib"/>
     </sort>
     <layout suffix=".">
-      <text macro="bibliography-filtered-notes"/>
+      <text macro="bibliography-filtered-notes-shortened"/>
     </layout>
   </bibliography>
 </style>
