--- templates/chicago-template.csl
+++ development/chicago-short-notes-bibliography-subsequent-ibid.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style and="text" class="note" demote-non-dropping-particle="display-and-sort" names-delimiter=", " page-range-format="chicago" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
+    <title>Chicago Manual of Style 18th edition (shortened author-title notes and bibliography, with Ibid. subsequent)</title>
     <title-short>CMOS</title-short>
-    <id>http://www.zotero.org/styles/chicago-notes-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="self"/>
+    <id>http://www.zotero.org/styles/chicago-short-notes-bibliography-subsequent-ibid</id>
+    <link href="http://www.zotero.org/styles/chicago-short-notes-bibliography-subsequent-ibid" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="template"/>
     <link href="http://www.chicagomanualofstyle.org/tools_citationguide.html" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -22,21 +23,7 @@
     <category field="literature"/>
     <category field="philosophy"/>
     <category field="theology"/>
-    <!-- Categories for author-date:
-    <category citation-format="author-date"/>
-    <category field="anthropology"/>
-    <category field="communications"/>
-    <category field="generic-base"/>
-    <category field="geography"/>
-    <category field="history"/>
-    <category field="humanities"/>
-    <category field="linguistics"/>
-    <category field="political_science"/>
-    <category field="science"/>
-    <category field="social_science"/>
-    <category field="sociology"/>
-    -->
-    <summary>Chicago-style source citations, template</summary>
+    <summary>Chicago-style source citations, shortened author-title notes and bibliography system, abbreviating subsequent references with Ibid. (CMOS 13.37)</summary>
     <updated>2025-05-19T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -2497,7 +2484,7 @@
     </group>
   </macro>
   <macro name="identifier-contributors-monographic-note">
-    <group delimiter=", ">
+    <group delimiter=". ">
       <choose>
         <if variable="title">
           <names variable="interviewer">
@@ -2509,7 +2496,7 @@
       <choose>
         <if match="none" type="post webpage">
           <!-- Handle `container-title` on `post` or `webpage` as `publisher` -->
-          <group delimiter=", ">
+          <group delimiter=". ">
             <choose>
               <if match="none" variable="container-title">
                 <names variable="container-author">
@@ -2586,7 +2573,7 @@
           </group>
         </if>
         <else>
-          <group delimiter=", ">
+          <group delimiter=". ">
             <names variable="container-author">
               <label form="verb-short" suffix=" "/>
               <name/>
@@ -5574,7 +5561,7 @@
   </macro>
   <citation disambiguate-add-names="true" et-al-min="3" et-al-use-first="1">
     <layout delimiter="; " suffix=".">
-      <text macro="citation-notes-full-subsequent-author-title"/>
+      <text macro="citation-notes-shortened-subsequent-ibid"/>
     </layout>
   </citation>
   <!-- Bibliography -->
@@ -5741,7 +5728,7 @@
       <key variable="status"/>
     </sort>
     <layout suffix=".">
-      <text macro="bibliography-filtered-notes"/>
+      <text macro="bibliography-filtered-notes-shortened"/>
     </layout>
   </bibliography>
 </style>
