--- templates/chicago-template.csl
+++ development/chicago-author-date.csl
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style and="text" class="note" demote-non-dropping-particle="display-and-sort" names-delimiter=", " page-range-format="chicago" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
+<style and="text" class="in-text" demote-non-dropping-particle="display-and-sort" names-delimiter=", " page-range-format="chicago" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
+    <title>Chicago Manual of Style 18th edition (author-date)</title>
     <title-short>CMOS</title-short>
-    <id>http://www.zotero.org/styles/chicago-fullnote-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography" rel="self"/>
-    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography-17th-edition" rel="template"/>
+    <id>http://www.zotero.org/styles/chicago-author-date</id>
+    <link href="http://www.zotero.org/styles/chicago-author-date" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography" rel="template"/>
     <link href="http://www.chicagomanualofstyle.org/tools_citationguide.html" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -51,11 +51,11 @@
       <name>Zeping Lee</name>
       <uri>https://github.com/zepinglee</uri>
     </contributor>
-    <category citation-format="note"/>
+    <category citation-format="author-date"/>
     <category field="humanities"/>
     <category field="social_science"/>
     <category field="generic-base"/>
-    <summary>Chicago Manual of Style full notes and bibliography</summary>
+    <summary>Chicago Manual of Style author-date system</summary>
     <updated>2025-03-01T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -98,7 +98,6 @@
         </if>
       </choose>
       <text variable="event-place"/>
-      <date form="text" variable="event-date"/>
     </group>
   </macro>
   <macro name="personal-communication-note">
@@ -159,7 +158,7 @@
   <macro name="author-short">
     <group delimiter=" ">
       <names variable="composer">
-        <name form="short"/>
+        <name form="short" initialize-with=". "/>
         <substitute>
           <names variable="author"/>
           <choose>
@@ -645,7 +644,7 @@
   <macro name="date-disambiguate">
     <choose>
       <if disambiguate="true">
-        <text macro="issued-date"/>
+        <text macro="issued-date-filtered-year"/>
       </if>
       <else-if type="personal_communication">
         <choose>
@@ -653,7 +652,7 @@
             <date form="text" variable="event-date"/>
           </if>
           <else>
-            <text macro="issued-date"/>
+            <text macro="issued-date-filtered-year"/>
           </else>
         </choose>
       </else-if>
@@ -781,7 +780,7 @@
         <group delimiter=". ">
           <choose>
             <if match="none" variable="original-publisher original-publisher-place publisher publisher-place">
-              <text macro="issued-date"/>
+              <text macro="issued-date-filtered-year"/>
             </if>
           </choose>
           <text text-case="capitalize-first" variable="scale"/>
@@ -799,7 +798,7 @@
         <group delimiter=", ">
           <choose>
             <if match="none" variable="original-publisher original-publisher-place publisher publisher-place">
-              <text macro="issued-date"/>
+              <text macro="issued-date-filtered-year"/>
             </if>
           </choose>
           <text variable="scale"/>
@@ -1703,7 +1702,7 @@
     <group delimiter=", ">
       <choose>
         <if match="any" type="article-journal periodical">
-          <text macro="volume-journal"/>
+          <text macro="volume-journal-issue-bracketed"/>
         </if>
         <else-if match="any" variable="volume volume-title">
           <choose>
@@ -2131,7 +2130,6 @@
           </group>
         </else>
       </choose>
-      <text macro="date-original-year"/>
     </group>
   </macro>
   <macro name="publisher">
@@ -2160,17 +2158,17 @@
   <macro name="publication-facts-journal">
     <choose>
       <if match="any" variable="volume issue">
-        <text macro="issued-date" prefix="(" suffix=")"/>
+        <text macro="issued-date-filtered-year" prefix="(" suffix=")"/>
       </if>
       <else>
-        <text macro="issued-date"/>
+        <text macro="issued-date-filtered-year"/>
       </else>
     </choose>
   </macro>
   <macro name="publication-facts-note">
     <choose>
       <if match="any" type="bill hearing legal_case legislation regulation treaty">
-        <text macro="issued-date" prefix="(" suffix=")"/>
+        <text macro="issued-date-filtered-year" prefix="(" suffix=")"/>
       </if>
       <else-if match="any" type="article-journal periodical">
         <text macro="publication-facts-journal"/>
@@ -2207,7 +2205,7 @@
                 <text value="released"/>
               </if>
             </choose>
-            <text macro="issued-date"/>
+            <text macro="issued-date-filtered-year"/>
           </group>
         </group>
       </else-if>
@@ -2223,7 +2221,7 @@
               <group delimiter="; ">
                 <group delimiter=", ">
                   <text macro="publisher"/>
-                  <text macro="issued-date"/>
+                  <text macro="issued-date-filtered-year"/>
                 </group>
                 <group delimiter=" ">
                   <text term="original-work-published"/>
@@ -2252,20 +2250,20 @@
                   <text macro="publisher"/>
                 </group>
               </group>
-              <text macro="issued-date"/>
+              <text macro="issued-date-filtered-year"/>
             </else>
           </choose>
         </group>
       </else-if>
       <else-if match="none" type="graphic map">
-        <text macro="issued-date"/>
+        <text macro="issued-date-filtered-year"/>
       </else-if>
     </choose>
   </macro>
   <macro name="publication-facts-bib">
     <choose>
       <if match="any" type="bill hearing legal_case legislation regulation treaty">
-        <text macro="issued-date" prefix="(" suffix=")"/>
+        <text macro="issued-date-filtered-year" prefix="(" suffix=")"/>
       </if>
       <else-if match="any" type="article-journal periodical">
         <text macro="publication-facts-journal"/>
@@ -2302,7 +2300,7 @@
                 <text text-case="capitalize-first" value="released"/>
               </if>
             </choose>
-            <text macro="issued-date"/>
+            <text macro="issued-date-filtered-year"/>
           </group>
         </group>
       </else-if>
@@ -2318,7 +2316,7 @@
               <group delimiter=". ">
                 <group delimiter=", ">
                   <text macro="publisher"/>
-                  <text macro="issued-date"/>
+                  <text macro="issued-date-filtered-year"/>
                 </group>
                 <group delimiter=" ">
                   <text term="original-work-published" text-case="capitalize-first"/>
@@ -2347,13 +2345,13 @@
                   <text macro="publisher"/>
                 </group>
               </group>
-              <text macro="issued-date"/>
+              <text macro="issued-date-filtered-year"/>
             </else>
           </choose>
         </group>
       </else-if>
       <else-if match="none" type="graphic map">
-        <text macro="issued-date"/>
+        <text macro="issued-date-filtered-year"/>
       </else-if>
     </choose>
   </macro>
@@ -2879,9 +2877,9 @@
       </choose>
     </group>
   </macro>
-  <citation disambiguate-add-names="true" et-al-min="3" et-al-use-first="1">
-    <layout delimiter="; " suffix=".">
-      <text macro="citation-fullnote"/>
+  <citation after-collapse-delimiter="; " collapse="year" disambiguate-add-givenname="true" disambiguate-add-names="true" disambiguate-add-year-suffix="true" et-al-min="3" et-al-use-first="1" givenname-disambiguation-rule="primary-name">
+    <layout delimiter="; " prefix="(" suffix=")">
+      <text macro="citation-inline"/>
     </layout>
   </citation>
   <macro name="bibliography">
@@ -2894,6 +2892,7 @@
                 <group delimiter=", ">
                   <group delimiter=". ">
                     <text macro="author-bib"/>
+                    <text macro="date"/>
                     <text macro="title-bib"/>
                     <text macro="description-bib"/>
                     <text macro="secondary-contributors-bib"/>
@@ -2956,9 +2955,9 @@
   <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true">
     <sort>
       <key macro="author-bib"/>
+      <key macro="date"/>
       <key variable="title"/>
       <key variable="genre"/>
-      <key variable="issued"/>
     </sort>
     <layout suffix=".">
       <text macro="bibliography-filtered"/>
