--- chicago-fullnote-bibliography.csl	2025-02-07 10:41:27
+++ chicago-fullnote-bibliography-short-title-subsequent.csl	2025-02-07 10:41:27
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" and="text" class="note" demote-non-dropping-particle="display-and-sort" names-delimiter=", " page-range-format="chicago" version="1.0">
   <info>
-    <title>Chicago Manual of Style, 18th edition (full note)</title>
+    <title>Chicago Manual of Style, 18th edition (full note, short title subsequent)</title>
     <title-short>CMOS</title-short>
-    <id>http://www.zotero.org/styles/chicago-fullnote-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography" rel="self"/>
+    <id>http://www.zotero.org/styles/chicago-fullnote-bibliography-short-title-subsequent</id>
+    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography-short-title-subsequent" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography" rel="template"/>
     <link href="http://www.chicagomanualofstyle.org/tools_citationguide.html" rel="documentation"/>
     <author>
       <name>Julian Onions</name>
@@ -1344,28 +1345,7 @@
   <citation disambiguate-add-names="true" et-al-min="3" et-al-use-first="1">
     <layout delimiter="; " suffix=".">
       <choose>
-        <if match="any" position="ibid ibid-with-locator">
-          <group delimiter=", ">
-            <text macro="contributors-short"/>
-            <group delimiter=" ">
-              <group delimiter=", ">
-                <choose>
-                  <if match="none" variable="author editor translator">
-                    <text macro="title-short"/>
-                  </if>
-                </choose>
-                <text macro="locator-case-subsequent"/>
-              </group>
-              <text macro="point-locator-case-subsequent"/>
-            </group>
-            <choose>
-              <if match="none" type="legal_case">
-                <text macro="point-locators-subsequent"/>
-              </if>
-            </choose>
-          </group>
-        </if>
-        <else-if position="subsequent">
+        <if position="subsequent">
           <group delimiter=", ">
             <text macro="contributors-short"/>
             <group delimiter=" ">
@@ -1382,7 +1362,7 @@
               </if>
             </choose>
           </group>
-        </else-if>
+        </if>
         <else>
           <group delimiter=", ">
             <group delimiter=": ">
