--- templates/chicago-template.csl
+++ development/chicago-shortened-notes-bibliography-classic-archive-place-first.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style class="note" demote-non-dropping-particle="display-and-sort" page-range-format="chicago-16" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
-    <title-short>CMOS with Bluebook</title-short>
-    <id>http://www.zotero.org/styles/chicago-notes-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="self"/>
+    <title>Chicago Manual of Style 18th edition (shortened notes and bibliography, classic variants, archival references with place first)</title>
+    <title-short>CMOS with Bluebook (shortened author-title notes [13.32], classic variants [13.113, 14.8, 14.30], archive place first [14.205])</title-short>
+    <id>http://www.zotero.org/styles/chicago-shortened-notes-bibliography-classic-archive-place-first</id>
+    <link href="http://www.zotero.org/styles/chicago-shortened-notes-bibliography-classic-archive-place-first" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="template"/>
     <link href="https://www.chicagomanualofstyle.org/" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -30,7 +31,7 @@
     <category field="social_science"/>
     <category field="sociology"/>
     <category field="theology"/>
-    <summary>Chicago-style source citations (with Bluebook for legal citations), template</summary>
+    <summary>Chicago-style source citations (with Bluebook for legal citations), shortened author-title notes and bibliography system, with classic variants: 3-em dash for repeated names (CMOS18 13.113), chapter page numbers (CMOS18 14.8), and place of publication (CMOS18 14.30); archival references with place first (CMOS18 14.205)</summary>
     <updated>2025-10-27T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -3523,10 +3524,6 @@
             <else-if match="any" variable="container-author editor editor-translator translator">
               <text macro="label-volume"/>
             </else-if>
-            <else-if variable="container-title">
-              <!-- remove condition in styles that print chapter page numbers (CMOS17/classic) -->
-              <text macro="label-volume"/>
-            </else-if>
             <!-- always display number of volumes in notes without bibliography (CMOS18 14.20): -->
             <!-- <else>
               <text macro="label-number-of-volumes"/>
@@ -4184,11 +4181,9 @@
           </if>
         </choose>
       </else-if>
-      <!-- if printing chapter page numbers (CMOS17/classic):
       <else-if variable="chapter-number page title">
         <text term="in" text-case="capitalize-first"/>
       </else-if>
-      -->
       <else-if variable="chapter-number">
         <group delimiter=" ">
           <text macro="label-chapter-number-capitalized"/>
@@ -4223,11 +4218,9 @@
           </if>
         </choose>
       </else-if>
-      <!-- if printing chapter page numbers (CMOS17/classic):
       <else-if variable="chapter-number page title">
         <text term="in"/>
       </else-if>
-      -->
       <else-if variable="chapter-number">
         <group delimiter=" ">
           <text macro="label-chapter-number"/>
@@ -4596,10 +4589,6 @@
         <else-if is-numeric="volume" match="none">
           <text macro="label-volume"/>
         </else-if>
-        <else-if variable="container-title">
-          <!-- remove condition in styles that print chapter page numbers (CMOS17/classic) -->
-          <text macro="label-volume"/>
-        </else-if>
         <else-if is-numeric="volume" variable="page">
           <choose>
             <!-- check for variables that might come between the volume and page number -->
@@ -4639,8 +4628,6 @@
       <else-if variable="locator">
         <text macro="label-locator"/>
       </else-if>
-      <!-- remove `container-title` condition in styles that print chapter page numbers (CMOS17/classic) -->
-      <else-if variable="container-title"/>
       <else-if is-numeric="volume" variable="page">
         <!-- collapse the volume and page number if adjacent (CMOS18 14.18) -->
         <group delimiter=":">
@@ -5405,7 +5392,7 @@
         </if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text text-case="capitalize-first" variable="publisher-place"/> -->
+            <text text-case="capitalize-first" variable="publisher-place"/>
             <text text-case="capitalize-first" variable="publisher"/>
           </group>
         </else-if>
@@ -5431,7 +5418,7 @@
         </if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text variable="publisher-place"/> -->
+            <text variable="publisher-place"/>
             <text variable="publisher"/>
           </group>
         </else-if>
@@ -5453,7 +5440,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text text-case="capitalize-first" variable="original-publisher-place"/> -->
+          <text text-case="capitalize-first" variable="original-publisher-place"/>
           <text text-case="capitalize-first" variable="original-publisher"/>
         </group>
       </if>
@@ -5466,7 +5453,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text variable="original-publisher-place"/> -->
+          <text variable="original-publisher-place"/>
           <text variable="original-publisher"/>
         </group>
       </if>
@@ -5932,7 +5919,7 @@
           <text macro="source-archive-reference-institution-first"/>
         </if>
         <else>
-          <text macro="source-archive-reference-location-first-bib"/>
+          <text macro="source-archive-reference-place-first"/>
         </else>
       </choose>
       <text macro="source-archive-database-number" prefix="(" suffix=")"/>
@@ -5945,7 +5932,7 @@
           <text macro="source-archive-reference-institution-first"/>
         </if>
         <else>
-          <text macro="source-archive-reference-location-first-note"/>
+          <text macro="source-archive-reference-place-first"/>
         </else>
       </choose>
       <text macro="source-archive-database-number" prefix="(" suffix=")"/>
@@ -6754,7 +6741,7 @@
   </macro>
   <citation disambiguate-add-givenname="true" disambiguate-add-names="true" et-al-min="3" et-al-use-first="1">
     <layout delimiter="; " suffix=".">
-      <text macro="citation-notes-full-subsequent-author-title"/>
+      <text macro="citation-notes-shortened-author-title"/>
     </layout>
   </citation>
   <!-- Bibliography -->
@@ -6902,7 +6889,7 @@
       </else>
     </choose>
   </macro>
-  <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true">
+  <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true" subsequent-author-substitute="&#8212;&#8212;&#8212;">
     <sort>
       <key macro="author-sort"/>
       <key macro="title-and-descriptions-bib"/>
@@ -6914,7 +6901,7 @@
       <key macro="source-archive-bib"/>
     </sort>
     <layout suffix=".">
-      <text macro="bibliography-filtered-notes"/>
+      <text macro="bibliography-filtered-notes-shortened"/>
     </layout>
   </bibliography>
 </style>
