--- templates/chicago-template.csl
+++ development/chicago-author-date-17th-edition.csl
@@ -3,19 +3,20 @@
   type="application/relax-ng-compact-syntax" schematypens="http://relaxng.org/ns/compatibility/annotations/1.0"?>
 <?xml-model href="https://raw.githubusercontent.com/citation-style-language/schema/refs/heads/master/schemas/styles/csl-repository.rnc"
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-<style class="note" demote-non-dropping-particle="display-and-sort" initialize-with=". " page-range-format="chicago-16" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
+<style class="in-text" demote-non-dropping-particle="display-and-sort" initialize-with=". " page-range-format="chicago-16" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
-    <title-short>CMOS/CMS with Bluebook</title-short>
-    <id>http://www.zotero.org/styles/chicago-notes-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="self"/>
+    <title>Chicago Manual of Style 17th edition (author-date)</title>
+    <title-short>CMOS 17e with Bluebook (author-date/AD [15.5])</title-short>
+    <id>http://www.zotero.org/styles/chicago-author-date-17th-edition</id>
+    <link href="http://www.zotero.org/styles/chicago-author-date-17th-edition" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="template"/>
     <link href="https://www.chicagomanualofstyle.org/" rel="documentation"/>
     <link href="https://zotero.org/groups/2205533/collections/5V67EPX3" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
       <uri>https://orcid.org/0000-0003-0464-5036</uri>
     </author>
-    <category citation-format="note"/>
+    <category citation-format="author-date"/>
     <category field="anthropology"/>
     <category field="communications"/>
     <category field="generic-base"/>
@@ -31,7 +32,7 @@
     <category field="social_science"/>
     <category field="sociology"/>
     <category field="theology"/>
-    <summary>Chicago-style source citations (with Bluebook for legal citations), template</summary>
+    <summary>Chicago-style source citations (with Bluebook for legal citations), author-date system, 17th edition</summary>
     <updated>2025-02-09T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -64,9 +65,6 @@
       <term name="manuscript">unpublished manuscript</term>
       <term name="original-work-published">originally published as</term>
       <term form="short" name="paper-conference">paper</term>
-      <!-- 'under' replaces 's.v.' from CMOS17 and earlier (CMOS18 14.130) -->
-      <term name="sub-verbo">under</term>
-      <term form="short" name="sub-verbo">under</term>
       <term name="timestamp">at</term>
       <term form="verb-short" name="translator">trans.</term>
     </terms>
@@ -474,13 +472,6 @@
           </else-if>
         </choose>
         <names variable="illustrator"/>
-        <choose>
-          <!-- list an edited multivolume work under the name of the series editor; remove this condition for author-date or notes and bibliography leading with volume title (CMOS18 14.21) -->
-          <if variable="container-title"/>
-          <else-if match="any" variable="number-of-volumes part-number part-title volume volume-title">
-            <names variable="collection-editor"/>
-          </else-if>
-        </choose>
         <choose>
           <if match="none" type="standard">
             <names variable="editor-translator"/>
@@ -1062,9 +1053,7 @@
       </else-if>
       <else-if type="webpage">
         <!-- list a webpage under the website title if there is no owner or sponsor (CMOS18 14.104) -->
-        <text text-case="title" variable="container-title"/>
-        <!-- for author-date: -->
-        <!-- <text text-case="title" variable="container-title-short"/> -->
+        <text text-case="title" variable="container-title-short"/>
       </else-if>
     </choose>
   </macro>
@@ -1436,9 +1425,7 @@
       <choose>
         <if type="broadcast" variable="container-title number title">
           <!-- Bespoke `broadcast` format (CMOS18 14.165, 14.168) -->
-          <text font-style="italic" text-case="title" variable="container-title"/>
-          <!-- for author-date or notes and bibliography leading with volume title (CMOS18 14.21): -->
-          <!-- <text macro="source-monographic-title-specific-title-first"/> -->
+          <text macro="source-monographic-title-specific-title-first"/>
           <group delimiter=", ">
             <text macro="identifier-number-bib"/>
             <text macro="title-bib"/>
@@ -1570,18 +1557,14 @@
       </else-if>
       <else-if match="any" variable="collection-editor compiler editor editorial-director">
         <!-- monographic types -->
-        <text macro="title-primary"/>
-        <!-- for author-date or notes and bibliography leading with volume title (CMOS18 14.21): -->
-        <!-- <text macro="title-monographic-bib-specific-title-first"/> -->
+        <text macro="title-monographic-bib-specific-title-first"/>
       </else-if>
       <else-if match="any" type="interview paper-conference">
         <!-- serial types -->
         <text macro="title-and-part-filter-review-bib"/>
       </else-if>
       <else>
-        <text macro="title-primary"/>
-        <!-- for author-date or notes and bibliography leading with volume title (CMOS18 14.21): -->
-        <!-- <text macro="title-monographic-bib-specific-title-first"/> -->
+        <text macro="title-monographic-bib-specific-title-first"/>
       </else>
     </choose>
   </macro>
@@ -3216,9 +3199,7 @@
       </if>
       <else-if match="any" variable="collection-editor compiler editor editorial-director">
         <!-- monographic types -->
-        <text macro="identifier-contributors-and-volume-monographic-bib"/>
-        <!-- for author-date or notes and bibliography leading with volume title (CMOS18 14.21): -->
-        <!-- <text macro="identifier-contributors-and-volume-monographic-bib-specific-title-first"/> -->
+        <text macro="identifier-contributors-and-volume-monographic-bib-specific-title-first"/>
       </else-if>
       <else-if match="any" type="interview paper-conference">
         <!-- serial types -->
@@ -3226,9 +3207,7 @@
       </else-if>
       <else>
         <!-- monographic types -->
-        <text macro="identifier-contributors-and-volume-monographic-bib"/>
-        <!-- for author-date or notes and bibliography leading with volume title (CMOS18 14.21): -->
-        <!-- <text macro="identifier-contributors-and-volume-monographic-bib-specific-title-first"/> -->
+        <text macro="identifier-contributors-and-volume-monographic-bib-specific-title-first"/>
       </else>
     </choose>
   </macro>
@@ -3568,10 +3547,6 @@
             <else-if match="any" variable="container-author editor editor-translator translator">
               <text macro="label-volume"/>
             </else-if>
-            <else-if variable="container-title">
-              <!-- remove condition in styles that print chapter page numbers (CMOS17/classic) -->
-              <text macro="label-volume"/>
-            </else-if>
             <!-- always display number of volumes in notes without bibliography (CMOS18 14.20): -->
             <!-- <else>
               <text macro="label-number-of-volumes"/>
@@ -3790,9 +3765,7 @@
       <choose>
         <if match="any" type="article-magazine article-newspaper">
           <group delimiter=". ">
-            <text macro="source-serial-identifier-volume-bib"/>
-            <!-- for author-date: -->
-            <!-- <text macro="source-serial-identifier-volume-author-date"/> -->
+            <text macro="source-serial-identifier-volume-author-date"/>
             <!-- newspaper edition always capitalized (CMOS18 14.89) -->
             <text macro="label-edition-capitalized"/>
           </group>
@@ -3800,16 +3773,12 @@
         </if>
         <else-if match="any" variable="issue supplement-number volume">
           <group delimiter=": ">
-            <text macro="source-serial-identifier-volume-bib"/>
-            <!-- for author-date: -->
-            <!-- <text macro="source-serial-identifier-volume-author-date"/> -->
+            <text macro="source-serial-identifier-volume-author-date"/>
             <text macro="source-serial-locator"/>
           </group>
         </else-if>
         <else>
-          <text macro="source-serial-identifier-volume-bib"/>
-          <!-- for author-date: -->
-          <!-- <text macro="source-serial-identifier-volume-author-date"/> -->
+          <text macro="source-serial-identifier-volume-author-date"/>
           <text macro="source-serial-locator"/>
         </else>
         <!-- TODO: If CSL adds `date-part` detection, add two further conditions to address CMOS18 14.74: delimiting with ":" if there is a `volume` and no month or `issue` or `supplement number`; delimiting with ", " or there is an `issue` or `supplement number` and no month -->
@@ -4114,9 +4083,7 @@
       </choose>
       <group delimiter=" ">
         <text macro="source-date-status-bib"/>
-        <text macro="source-date-issued-or-status"/>
-        <!-- for author-date or notes and bibliography leading with volume title (CMOS18 14.21): -->
-        <!-- <text macro="source-date-specific-title-first"/> -->
+        <text macro="source-date-specific-title-first"/>
       </group>
     </group>
   </macro>
@@ -4173,9 +4140,7 @@
           <group delimiter=", ">
             <group delimiter=" ">
               <text macro="source-monographic-preposition-bib"/>
-              <text font-style="italic" text-case="title" variable="container-title"/>
-              <!-- for author-date or notes and bibliography leading with volume title (CMOS18 14.21): -->
-              <!-- <text macro="source-monographic-title-specific-title-first"/> -->
+              <text macro="source-monographic-title-specific-title-first"/>
             </group>
             <text macro="source-monographic-description"/>
             <text macro="source-monographic-identifier-bib"/>
@@ -4231,11 +4196,9 @@
           </if>
         </choose>
       </else-if>
-      <!-- if printing chapter page numbers (CMOS17/classic):
       <else-if variable="chapter-number page title">
         <text term="in" text-case="capitalize-first"/>
       </else-if>
-      -->
       <else-if variable="chapter-number">
         <group delimiter=" ">
           <text macro="label-chapter-number-capitalized"/>
@@ -4271,11 +4234,9 @@
           </if>
         </choose>
       </else-if>
-      <!-- if printing chapter page numbers (CMOS17/classic):
       <else-if variable="chapter-number page title">
         <text term="in"/>
       </else-if>
-      -->
       <else-if variable="chapter-number">
         <group delimiter=" ">
           <text macro="label-chapter-number"/>
@@ -4336,9 +4297,7 @@
           <text macro="label-version"/>
           <!-- use note form as `edition` is not capitalized here -->
           <text macro="identifier-edition-note"/>
-          <text macro="source-monographic-identifier-contributors-and-volume-bib"/>
-          <!-- for author-date or notes and bibliography leading with volume title (CMOS18 14.21): -->
-          <!-- <text macro="source-monographic-identifier-contributors-and-volume-bib-specific-title-first"/> -->
+          <text macro="source-monographic-identifier-contributors-and-volume-bib-specific-title-first"/>
         </group>
       </if>
     </choose>
@@ -4709,10 +4668,6 @@
         <else-if is-numeric="volume" match="none">
           <text macro="label-volume"/>
         </else-if>
-        <else-if variable="container-title">
-          <!-- remove condition in styles that print chapter page numbers (CMOS17/classic) -->
-          <text macro="label-volume"/>
-        </else-if>
         <else-if is-numeric="volume" variable="page">
           <choose>
             <!-- check for variables that might come between the volume and page number -->
@@ -4739,9 +4694,7 @@
         <group delimiter=":">
           <choose>
             <if match="any" variable="part-number part-title volume-title"/>
-            <else-if match="any" variable="container-author editor editor-translator translator"/>
-            <!-- for author-date or notes and bibliography leading with volume title (CMOS18 14.21), check for name variables falling after the general title: -->
-            <!-- <else-if variable="collection-editor"/> -->
+            <else-if variable="collection-editor"/>
             <else>
               <text variable="volume"/>
             </else>
@@ -4752,17 +4705,13 @@
       <else-if variable="locator">
         <text macro="label-locator"/>
       </else-if>
-      <!-- remove `container-title` condition in styles that print chapter page numbers (CMOS17/classic) -->
-      <else-if variable="container-title"/>
       <else-if is-numeric="volume" variable="page">
         <!-- collapse the volume and page number if adjacent (CMOS18 14.18) -->
         <group delimiter=":">
           <choose>
             <!-- check for variables that might come between the volume and page number -->
             <if match="any" variable="part-number part-title volume-title"/>
-            <else-if match="any" variable="container-author editor editor-translator translator"/>
-            <!-- for author-date or notes and bibliography leading with volume title (CMOS18 14.21), check for name variables falling after the general title: -->
-            <!-- <else-if variable="collection-editor"/> -->
+            <else-if variable="collection-editor"/>
             <else>
               <text variable="volume"/>
             </else>
@@ -5288,9 +5237,7 @@
       </choose>
       <group delimiter=" ">
         <text macro="source-date-status-bib"/>
-        <text macro="source-date-issued-or-status"/>
-        <!-- for author-date or notes and bibliography leading with volume title (CMOS18 14.21): -->
-        <!-- <text macro="source-date-specific-title-first"/> -->
+        <text macro="source-date-specific-title-first"/>
       </group>
       <choose>
         <if type="broadcast" variable="URL"/>
@@ -5518,7 +5465,7 @@
         </if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text text-case="capitalize-first" variable="publisher-place"/> -->
+            <text text-case="capitalize-first" variable="publisher-place"/>
             <text text-case="capitalize-first" variable="publisher"/>
           </group>
         </else-if>
@@ -5544,7 +5491,7 @@
         </if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text variable="publisher-place"/> -->
+            <text variable="publisher-place"/>
             <text variable="publisher"/>
           </group>
         </else-if>
@@ -5566,7 +5513,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text text-case="capitalize-first" variable="original-publisher-place"/> -->
+          <text text-case="capitalize-first" variable="original-publisher-place"/>
           <text text-case="capitalize-first" variable="original-publisher"/>
         </group>
       </if>
@@ -5579,7 +5526,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text variable="original-publisher-place"/> -->
+          <text variable="original-publisher-place"/>
           <text variable="original-publisher"/>
         </group>
       </if>
@@ -5601,9 +5548,7 @@
       </else-if>
       <else-if match="any" type="article-journal article-magazine article-newspaper periodical post-weblog review review-book">
         <!-- serial types -->
-        <text macro="source-date-issued-month-day"/>
-        <!-- for CMOS17 author-date: -->
-        <!-- <text macro="source-date-issued-full-serial"/> -->
+        <text macro="source-date-issued-full-serial"/>
         <!-- for notes and bibliography leading with volume title (CMOS18 14.21): -->
         <!-- <text macro="source-date-issued-or-status"/> -->
       </else-if>
@@ -5615,9 +5560,7 @@
       </else-if>
       <else-if match="any" type="interview paper-conference">
         <!-- serial types -->
-        <text macro="source-date-issued-month-day"/>
-        <!-- for CMOS17 author-date: -->
-        <!-- <text macro="source-date-issued-full-serial"/> -->
+        <text macro="source-date-issued-full-serial"/>
         <!-- for notes and bibliography leading with volume title (CMOS18 14.21): -->
         <!-- <text macro="source-date-issued-or-status"/> -->
       </else-if>
@@ -5683,14 +5626,10 @@
         <text macro="date-issued-full"/>
       </if>
       <else-if variable="issue volume">
-        <text macro="date-issued-year"/>
-        <!-- for CMOS17: -->
-        <!-- <text macro="date-issued-year-month"/> -->
+        <text macro="date-issued-year-month"/>
       </else-if>
       <else-if variable="supplement-number">
-        <text macro="date-issued-year"/>
-        <!-- for CMOS17: -->
-        <!-- <text macro="date-issued-year-month"/> -->
+        <text macro="date-issued-year-month"/>
       </else-if>
       <else-if match="any" variable="issue volume">
         <text macro="date-issued-year-month"/>
@@ -5835,14 +5774,10 @@
   <macro name="source-date-original-full-serial">
     <choose>
       <if variable="issue volume">
-        <text macro="date-original-year"/>
-        <!-- for CMOS17: -->
-        <!-- <text macro="date-original-year-month"/> -->
+        <text macro="date-original-year-month"/>
       </if>
       <else-if variable="supplement-number">
-        <text macro="date-original-year"/>
-        <!-- for CMOS17: -->
-        <!-- <text macro="date-original-year-month"/> -->
+        <text macro="date-original-year-month"/>
       </else-if>
       <else-if match="any" variable="issue volume">
         <text macro="date-original-year-month"/>
@@ -6898,9 +6833,9 @@
       </else>
     </choose>
   </macro>
-  <citation disambiguate-add-givenname="true" disambiguate-add-names="true" et-al-min="3" et-al-use-first="1">
-    <layout delimiter="; " suffix=".">
-      <text macro="citation-notes-full-subsequent-author-title"/>
+  <citation collapse="year" disambiguate-add-givenname="true" disambiguate-add-names="true" disambiguate-add-year-suffix="true" et-al-min="4" et-al-use-first="1" givenname-disambiguation-rule="primary-name">
+    <layout delimiter="; " prefix="(" suffix=")">
+      <text macro="citation-author-date"/>
     </layout>
   </citation>
   <!-- Bibliography -->
@@ -7042,19 +6977,22 @@
       </else>
     </choose>
   </macro>
-  <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true">
+  <bibliography et-al-min="11" et-al-use-first="7" hanging-indent="true" subsequent-author-substitute="&#8212;&#8212;&#8212;">
     <sort>
       <key macro="author-sort"/>
+      <key macro="date-sort-group"/>
+      <key macro="date-sort-year"/>
+      <key macro="date"/>
       <key macro="title-and-descriptions-bib"/>
       <key macro="source-bib"/>
       <key variable="volume"/>
       <key variable="part-number"/>
       <key variable="event-date"/>
-      <key macro="source-date-issued-or-status"/>
+      <key variable="issued"/>
       <key macro="source-archive-bib"/>
     </sort>
     <layout suffix=".">
-      <text macro="bibliography-filtered-notes"/>
+      <text macro="bibliography-filtered-author-date"/>
     </layout>
   </bibliography>
 </style>
