--- templates/chicago-template.csl
+++ development/chicago-author-date-classic.csl
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style and="text" class="note" demote-non-dropping-particle="display-and-sort" names-delimiter=", " page-range-format="chicago" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
+<style and="text" class="in-text" demote-non-dropping-particle="display-and-sort" names-delimiter=", " page-range-format="chicago" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
+    <title>Chicago Manual of Style 18th edition (author-date with classic variants)</title>
     <title-short>CMOS</title-short>
-    <id>http://www.zotero.org/styles/chicago-fullnote-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography" rel="self"/>
-    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography-17th-edition" rel="template"/>
+    <id>http://www.zotero.org/styles/chicago-author-date-classic</id>
+    <link href="http://www.zotero.org/styles/chicago-author-date-classic" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography" rel="template"/>
     <link href="http://www.chicagomanualofstyle.org/tools_citationguide.html" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -51,11 +51,11 @@
       <name>Zeping Lee</name>
       <uri>https://github.com/zepinglee</uri>
     </contributor>
-    <category citation-format="note"/>
+    <category citation-format="author-date"/>
     <category field="humanities"/>
     <category field="social_science"/>
     <category field="generic-base"/>
-    <summary>Chicago Manual of Style full notes and bibliography</summary>
+    <summary>Chicago Manual of Style author-date system, with classic variants (chapter page numbers, place of publication, and 3-em dash)</summary>
     <updated>2025-03-01T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -98,7 +98,6 @@
         </if>
       </choose>
       <text variable="event-place"/>
-      <date form="text" variable="event-date"/>
     </group>
   </macro>
   <macro name="personal-communication-note">
@@ -159,7 +158,7 @@
   <macro name="author-short">
     <group delimiter=" ">
       <names variable="composer">
-        <name form="short"/>
+        <name form="short" initialize-with=". "/>
         <substitute>
           <names variable="author"/>
           <choose>
@@ -760,7 +759,7 @@
   <macro name="date-disambiguate">
     <choose>
       <if disambiguate="true">
-        <text macro="issued-date"/>
+        <text macro="issued-date-filtered-year"/>
       </if>
       <else-if type="personal_communication">
         <choose>
@@ -768,7 +767,7 @@
             <date form="text" variable="event-date"/>
           </if>
           <else>
-            <text macro="issued-date"/>
+            <text macro="issued-date-filtered-year"/>
           </else>
         </choose>
       </else-if>
@@ -1717,9 +1716,6 @@
       <else-if match="any" variable="chair compiler container-author contributor curator director editor translator editor-translator editorial-director executive-producer guest host illustrator interviewer narrator organizer original-author performer producer script-writer series-creator volume-title">
         <text macro="volume-label-number-note"/>
       </else-if>
-      <else-if type="chapter">
-        <text macro="volume-label-number-note"/>
-      </else-if>
     </choose>
   </macro>
   <macro name="volume-label-number-note">
@@ -1756,7 +1752,7 @@
     <group delimiter=", ">
       <choose>
         <if match="any" type="article-journal periodical">
-          <text macro="volume-journal"/>
+          <text macro="volume-journal-issue-bracketed"/>
         </if>
         <else-if match="any" variable="volume volume-title">
           <choose>
@@ -2203,12 +2199,11 @@
         </if>
         <else>
           <group delimiter=": ">
-            <!-- <text variable="original-publisher-place"/> -->
+            <text variable="original-publisher-place"/>
             <text variable="original-publisher"/>
           </group>
         </else>
       </choose>
-      <text macro="date-original-year"/>
     </group>
   </macro>
   <macro name="publisher">
@@ -2226,7 +2221,7 @@
           </if>
           <else>
             <group delimiter=": ">
-              <!-- <text variable="publisher-place"/> -->
+              <text variable="publisher-place"/>
               <text variable="publisher"/>
             </group>
           </else>
@@ -2237,17 +2232,17 @@
   <macro name="publication-facts-journal">
     <choose>
       <if match="any" variable="volume issue">
-        <text macro="issued-date" prefix="(" suffix=")"/>
+        <text macro="issued-date-filtered-year" prefix="(" suffix=")"/>
       </if>
       <else>
-        <text macro="issued-date"/>
+        <text macro="issued-date-filtered-year"/>
       </else>
     </choose>
   </macro>
   <macro name="publication-facts-note">
     <choose>
       <if match="any" type="bill hearing legal_case legislation regulation treaty">
-        <text macro="issued-date" prefix="(" suffix=")"/>
+        <text macro="issued-date-filtered-year" prefix="(" suffix=")"/>
       </if>
       <else-if match="any" type="article-journal periodical">
         <text macro="publication-facts-journal"/>
@@ -2284,7 +2279,7 @@
                 <text value="released"/>
               </if>
             </choose>
-            <text macro="issued-date"/>
+            <text macro="issued-date-filtered-year"/>
           </group>
         </group>
       </else-if>
@@ -2300,7 +2295,7 @@
               <group delimiter="; ">
                 <group delimiter=", ">
                   <text macro="publisher"/>
-                  <text macro="issued-date"/>
+                  <text macro="issued-date-filtered-year"/>
                 </group>
                 <group delimiter=" ">
                   <text term="original-work-published"/>
@@ -2329,20 +2324,20 @@
                   <text macro="publisher"/>
                 </group>
               </group>
-              <text macro="issued-date"/>
+              <text macro="issued-date-filtered-year"/>
             </else>
           </choose>
         </group>
       </else-if>
       <else>
-        <text macro="issued-date"/>
+        <text macro="issued-date-filtered-year"/>
       </else>
     </choose>
   </macro>
   <macro name="publication-facts-bib">
     <choose>
       <if match="any" type="bill hearing legal_case legislation regulation treaty">
-        <text macro="issued-date" prefix="(" suffix=")"/>
+        <text macro="issued-date-filtered-year" prefix="(" suffix=")"/>
       </if>
       <else-if match="any" type="article-journal periodical">
         <text macro="publication-facts-journal"/>
@@ -2379,7 +2374,7 @@
                 <text text-case="capitalize-first" value="released"/>
               </if>
             </choose>
-            <text macro="issued-date"/>
+            <text macro="issued-date-filtered-year"/>
           </group>
         </group>
       </else-if>
@@ -2395,7 +2390,7 @@
               <group delimiter=". ">
                 <group delimiter=", ">
                   <text macro="publisher"/>
-                  <text macro="issued-date"/>
+                  <text macro="issued-date-filtered-year"/>
                 </group>
                 <group delimiter=" ">
                   <text term="original-work-published" text-case="capitalize-first"/>
@@ -2424,13 +2419,13 @@
                   <text macro="publisher"/>
                 </group>
               </group>
-              <text macro="issued-date"/>
+              <text macro="issued-date-filtered-year"/>
             </else>
           </choose>
         </group>
       </else-if>
       <else>
-        <text macro="issued-date"/>
+        <text macro="issued-date-filtered-year"/>
       </else>
     </choose>
   </macro>
@@ -2565,7 +2560,7 @@
   </macro>
   <macro name="locators">
     <choose>
-      <if match="none" type="chapter" variable="locator">
+      <if match="none" variable="locator">
         <choose>
           <if is-numeric="page">
             <choose>
@@ -2947,9 +2942,9 @@
       </choose>
     </group>
   </macro>
-  <citation disambiguate-add-names="true" et-al-min="3" et-al-use-first="1">
-    <layout delimiter="; " suffix=".">
-      <text macro="citation-fullnote"/>
+  <citation after-collapse-delimiter="; " collapse="year" disambiguate-add-givenname="true" disambiguate-add-names="true" disambiguate-add-year-suffix="true" et-al-min="3" et-al-use-first="1" givenname-disambiguation-rule="primary-name">
+    <layout delimiter="; " prefix="(" suffix=")">
+      <text macro="citation-intext"/>
     </layout>
   </citation>
   <macro name="bibliography">
@@ -2962,6 +2957,7 @@
                 <group delimiter=", ">
                   <group delimiter=". ">
                     <text macro="author-bib"/>
+                    <text macro="date"/>
                     <text macro="title-bib"/>
                     <text macro="description-bib"/>
                     <text macro="secondary-contributors-bib"/>
@@ -3021,13 +3017,13 @@
       </else>
     </choose>
   </macro>
-  <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true">
+  <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true" subsequent-author-substitute="&#8212;&#8212;&#8212;">
     <sort>
       <key macro="author-sort"/>
-      <key macro="title-bib"/>
       <key macro="date-sort-group" sort="ascending"/>
       <key macro="date-sort" sort="ascending"/>
       <key variable="status"/>
+      <key macro="title-bib"/>
     </sort>
     <layout suffix=".">
       <text macro="bibliography-filtered"/>
