--- templates/chicago-template.csl
+++ development/chicago-author-date-17th-edition.csl
@@ -3,26 +3,18 @@
   type="application/relax-ng-compact-syntax" schematypens="http://relaxng.org/ns/compatibility/annotations/1.0"?>
 <?xml-model href="https://raw.githubusercontent.com/citation-style-language/schema/refs/heads/master/schemas/styles/csl.rnc"
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-<style and="text" class="note" demote-non-dropping-particle="display-and-sort" names-delimiter=", " page-range-format="chicago" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
+<style and="text" class="in-text" demote-non-dropping-particle="display-and-sort" names-delimiter=", " page-range-format="chicago" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
+    <title>Chicago Manual of Style 17th edition (author-date)</title>
     <title-short>CMOS</title-short>
-    <id>http://www.zotero.org/styles/chicago-notes-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="self"/>
-    <link href="http://www.chicagomanualofstyle.org/tools_citationguide.html" rel="documentation"/>
+    <id>http://www.zotero.org/styles/chicago-author-date-17th-edition</id>
+    <link href="http://www.zotero.org/styles/chicago-author-date-17th-edition" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="template"/>
+    <link href="https://web.archive.org/web/20240801040138/https://www.chicagomanualofstyle.org/tools_citationguide.html" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
       <uri>https://orcid.org/0000-0003-0464-5036</uri>
     </author>
-    <category citation-format="note"/>
-    <category field="generic-base"/>
-    <category field="history"/>
-    <category field="humanities"/>
-    <category field="law"/>
-    <category field="literature"/>
-    <category field="philosophy"/>
-    <category field="theology"/>
-    <!-- Categories for author-date:
     <category citation-format="author-date"/>
     <category field="anthropology"/>
     <category field="communications"/>
@@ -35,8 +27,7 @@
     <category field="science"/>
     <category field="social_science"/>
     <category field="sociology"/>
-    -->
-    <summary>Chicago-style source citations, template</summary>
+    <summary>Chicago-style source citations, author-date system, 17th edition</summary>
     <updated>2025-05-19T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -557,9 +548,7 @@
   </macro>
   <macro name="author-substitute-short">
     <names variable="composer">
-      <name form="short"/>
-      <!-- for author-date:
-      <name form="short" initialize-with=". "/> -->
+      <name form="short" initialize-with=". "/>
       <substitute>
         <names variable="author"/>
         <choose>
@@ -3067,14 +3056,14 @@
     <choose>
       <if match="any" variable="issue supplement-number volume">
         <group delimiter=": ">
-          <text macro="source-serial-identifier-volume-bib"/>
+          <text macro="source-serial-identifier-volume-author-date"/>
           <text macro="source-serial-locators"/>
         </group>
       </if>
       <else>
         <group delimiter=", ">
           <group delimiter=". ">
-            <text macro="source-serial-identifier-volume-bib"/>
+            <text macro="source-serial-identifier-volume-author-date"/>
             <!-- periodical edition always capitalized (CMOS 14.89) -->
             <text macro="label-edition-capitalized"/>
           </group>
@@ -3345,11 +3334,9 @@
           </if>
         </choose>
       </else-if>
-      <!-- if printing chapter page numbers:
       <else-if variable="chapter-number page">
         <text term="in" text-case="capitalize-first"/>
       </else-if>
-      -->
       <else-if variable="chapter-number">
         <group delimiter=" ">
           <text macro="label-chapter-number-capitalized"/>
@@ -3385,11 +3372,9 @@
           </if>
         </choose>
       </else-if>
-      <!-- if printing chapter page numbers:
       <else-if variable="chapter-number page">
         <text term="in"/>
       </else-if>
-      -->
       <else-if variable="chapter-number">
         <group delimiter=" ">
           <text macro="label-chapter-number"/>
@@ -3732,10 +3717,6 @@
             <if is-numeric="volume" match="none">
               <text macro="label-volume"/>
             </if>
-            <else-if type="chapter">
-              <!-- remove condition in styles that print chapter page numbers -->
-              <text macro="label-volume"/>
-            </else-if>
             <else-if is-numeric="volume" variable="page">
               <choose>
                 <if match="any" variable="collection-editor part-number part-title volume-title">
@@ -3763,8 +3744,7 @@
   <!-- Monographic source locators -->
   <macro name="source-monographic-locators-bib">
     <choose>
-      <if type="chapter"/>
-      <else-if is-numeric="volume" variable="page">
+      <if is-numeric="volume" variable="page">
         <!-- collapse the volume and page number if adjacent -->
         <group delimiter=":">
           <choose>
@@ -3775,7 +3755,7 @@
           </choose>
           <text variable="page"/>
         </group>
-      </else-if>
+      </if>
       <else>
         <text variable="page"/>
       </else>
@@ -3800,7 +3780,7 @@
   </macro>
   <macro name="source-monographic-locators-note-item">
     <choose>
-      <if match="none" type="chapter" variable="locator">
+      <if match="none" variable="locator">
         <text variable="page"/>
       </if>
       <else>
@@ -4330,7 +4310,7 @@
         </if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text variable="publisher-place"/> -->
+            <text variable="publisher-place"/>
             <text variable="publisher"/>
           </group>
         </else-if>
@@ -4355,7 +4335,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text variable="original-publisher-place"/> -->
+          <text variable="original-publisher-place"/>
           <text variable="original-publisher"/>
         </group>
       </if>
@@ -4388,15 +4368,9 @@
       </if>
       <else-if match="any" type="article-journal article-magazine article-newspaper periodical post-weblog review review-book">
         <choose>
-          <if variable="issue volume">
-            <date date-parts="year" form="text" variable="issued"/>
-          </if>
-          <else-if variable="supplement-number volume">
-            <date date-parts="year" form="text" variable="issued"/>
-          </else-if>
-          <else-if variable="volume">
+          <if variable="volume">
             <date date-parts="year-month" form="text" variable="issued"/>
-          </else-if>
+          </if>
           <else>
             <text macro="date-issued-full"/>
           </else>
@@ -5430,9 +5404,9 @@
       </else>
     </choose>
   </macro>
-  <citation disambiguate-add-names="true" et-al-min="3" et-al-use-first="1">
-    <layout delimiter="; " suffix=".">
-      <text macro="citation-notes-full-subsequent-author-title"/>
+  <citation after-collapse-delimiter="; " collapse="year" disambiguate-add-givenname="true" disambiguate-add-names="true" disambiguate-add-year-suffix="true" et-al-min="4" et-al-use-first="1" givenname-disambiguation-rule="primary-name-with-initials">
+    <layout delimiter="; " prefix="(" suffix=")">
+      <text macro="citation-author-date"/>
     </layout>
   </citation>
   <!-- Bibliography -->
@@ -5555,16 +5529,16 @@
       </else>
     </choose>
   </macro>
-  <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true">
+  <bibliography et-al-min="11" et-al-use-first="7" hanging-indent="true" subsequent-author-substitute="&#8212;&#8212;&#8212;">
     <sort>
       <key macro="author-sort"/>
-      <key macro="title-bib"/>
       <key macro="date-sort-group"/>
       <key macro="date-sort"/>
       <key variable="status"/>
+      <key macro="title-bib"/>
     </sort>
     <layout suffix=".">
-      <text macro="bibliography-filtered-notes"/>
+      <text macro="bibliography-filtered-author-date"/>
     </layout>
   </bibliography>
 </style>
