--- templates/mhra-template.csl
+++ development/instrumenta-patristica-et-mediaevalia.csl
@@ -3,31 +3,36 @@
   type="application/relax-ng-compact-syntax" schematypens="http://relaxng.org/ns/compatibility/annotations/1.0"?>
 <?xml-model href="https://raw.githubusercontent.com/citation-style-language/schema/refs/heads/master/schemas/styles/csl-repository.rnc"
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-<style and="text" class="note" default-locale="en-GB" demote-non-dropping-particle="display-and-sort" et-al-min="4" et-al-use-first="1" initialize-with=". " page-range-format="minimal-two" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
+<style class="note" default-locale="en-GB" demote-non-dropping-particle="display-and-sort" et-al-min="4" et-al-use-first="1" initialize-with=". " page-range-format="expanded" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>MHRA Style Guide 4th edition (template)</title>
-    <title-short>Modern Humanities Research Association</title-short>
-    <id>http://www.zotero.org/styles/mhra-notes</id>
-    <link href="http://www.zotero.org/styles/mhra-notes" rel="self"/>
-    <link href="https://www.mhra.org.uk/style/" rel="documentation"/>
+    <title>Instrumenta Patristica et Mediaevalia</title>
+    <title-short>IPM</title-short>
+    <id>http://www.zotero.org/styles/instrumenta-patristica-et-mediaevalia</id>
+    <link href="http://www.zotero.org/styles/instrumenta-patristica-et-mediaevalia" rel="self"/>
+    <link href="http://www.zotero.org/styles/mhra-notes" rel="template"/>
+    <link href="https://www.corpuschristianorum.org/author-info" rel="documentation"/>
+    <link href="https://www.brepols.net/permalink/stylesheet-full-refs" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
       <uri>https://orcid.org/0000-0003-0464-5036</uri>
     </author>
     <category citation-format="note"/>
-    <category field="generic-base"/>
-    <category field="history"/>
     <category field="humanities"/>
-    <category field="linguistics"/>
     <category field="literature"/>
     <category field="philosophy"/>
     <category field="theology"/>
-    <summary>MHRA source citations, template</summary>
+    <issn>1379-9878</issn>
+    <eissn>2294-8457</eissn>
+    <summary>Instrumenta Patristica et Mediaevalia (Brepols), interpreted as a light modification of MHRA.</summary>
     <updated>2025-02-04T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="en">
     <terms>
+      <term name="open-quote">“</term>
+      <term name="close-quote">”</term>
+      <term name="open-inner-quote">‘</term>
+      <term name="close-inner-quote">’</term>
       <term name="advance-online-publication">published online</term>
       <term form="short" name="editor-translator">
         <single>ed. and trans.</single>
@@ -39,7 +44,6 @@
       </term>
       <term form="verb-short" name="editor-translator">ed. and trans. by</term>
       <term form="verb-short" name="editortranslator">ed. and trans. by</term>
-      <term name="et-al">and others</term>
       <term name="manuscript">unpublished manuscript</term>
       <term name="original-work-published">originally published as</term>
       <term form="short" name="paper-conference">paper</term>
@@ -370,7 +374,8 @@
   <macro name="author-bib">
     <group delimiter=", ">
       <names variable="composer">
-        <name delimiter-precedes-et-al="after-inverted-name" delimiter-precedes-last="always" initialize="false" name-as-sort-order="first"/>
+        <name delimiter-precedes-et-al="after-inverted-name" delimiter-precedes-last="always" font-variant="small-caps" name-as-sort-order="first"/>
+        <et-al font-variant="normal"/>
         <label form="short" prefix=" (" suffix=")"/>
         <substitute>
           <choose>
@@ -379,7 +384,17 @@
               <names variable="interviewer"/>
             </if>
           </choose>
-          <names variable="author"/>
+          <choose>
+            <if match="any" type="classic document manuscript personal_communication">
+              <names variable="author">
+                <name delimiter-precedes-et-al="after-inverted-name" delimiter-precedes-last="always" initialize="false" name-as-sort-order="first"/>
+                <!-- do not initialize names in primary sources -->
+              </names>
+            </if>
+            <else>
+              <names variable="author"/>
+            </else>
+          </choose>
           <names variable="host"/>
           <names variable="guest"/>
           <choose>
@@ -529,7 +544,8 @@
   <macro name="author-note">
     <group delimiter=", ">
       <names variable="composer">
-        <name initialize="false"/>
+        <name font-variant="small-caps" initialize="true"/>
+        <et-al font-variant="normal"/>
         <label form="short" prefix=" (" suffix=")"/>
         <substitute>
           <choose>
@@ -538,7 +554,17 @@
               <names variable="interviewer"/>
             </if>
           </choose>
-          <names variable="author"/>
+          <choose>
+            <if match="any" type="classic document manuscript personal_communication">
+              <names variable="author">
+                <name initialize="false"/>
+                <!-- do not initialize names in primary sources -->
+              </names>
+            </if>
+            <else>
+              <names variable="author"/>
+            </else>
+          </choose>
           <names variable="illustrator"/>
           <choose>
             <if type="song">
@@ -577,7 +603,8 @@
       </if>
       <else>
         <names variable="composer">
-          <name form="short" initialize="true"/>
+          <name font-variant="small-caps" form="short" initialize="true"/>
+          <et-al font-variant="normal"/>
           <substitute>
             <choose>
               <if match="any" variable="container-title DOI issue number publisher publisher-place references URL volume">
@@ -660,7 +687,8 @@
       <else-if match="any" type="book broadcast chapter classic entry interview motion_picture musical_score pamphlet paper-conference report software song standard thesis">
         <names variable="contributor">
           <label form="verb" suffix=" "/>
-          <name initialize="false"/>
+          <name font-variant="small-caps" initialize="true"/>
+          <et-al font-variant="normal"/>
         </names>
       </else-if>
     </choose>
@@ -714,19 +742,22 @@
             <if match="any" variable="container-title DOI issue number publisher publisher-place references URL volume">
               <!-- cite published interviews under interviewer name (MHRA 7.11) -->
               <names variable="interviewer">
-                <name form="short" initialize="true"/>
+                <name font-variant="small-caps" form="short" initialize="true"/>
+                <et-al font-variant="normal"/>
               </names>
             </if>
             <else>
               <names variable="author">
-                <name form="short" initialize="true"/>
+                <name font-variant="small-caps" form="short" initialize="true"/>
+                <et-al font-variant="normal"/>
               </names>
             </else>
           </choose>
         </else-if>
         <else-if variable="author recipient">
           <names variable="author">
-            <name form="short" initialize="true"/>
+            <name font-variant="small-caps" form="short" initialize="true"/>
+            <et-al font-variant="normal"/>
           </names>
           <choose>
             <if match="none" variable="genre">
@@ -739,7 +770,8 @@
         </else-if>
         <else>
           <names variable="author">
-            <name form="short" initialize="true"/>
+            <name font-variant="small-caps" form="short" initialize="true"/>
+            <et-al font-variant="normal"/>
             <substitute>
               <text macro="title-and-descriptions-short"/>
             </substitute>
@@ -1175,8 +1207,23 @@
               </else>
             </choose>
             <text macro="source-monographic-publication-unbracketed"/>
+            <text macro="source-monographic-publication-bracketed"/>
           </group>
-          <text macro="source-monographic-publication-bracketed"/>
+          <choose>
+            <!-- Series, for monographic types only -->
+            <!-- omit serial types -->
+            <if match="any" type="article-journal article-magazine article-newspaper periodical post-weblog review review-book"/>
+            <else-if match="any" variable="collection-editor container-author editor editorial-director">
+              <!-- monographic types -->
+              <text macro="source-series" prefix="(" suffix=")"/>
+            </else-if>
+            <!-- omit serial types -->
+            <else-if match="any" type="interview paper-conference"/>
+            <else>
+              <!-- monographic types -->
+              <text macro="source-series" prefix="(" suffix=")"/>
+            </else>
+          </choose>
         </group>
         <choose>
           <!-- Locator for monographic types only; serial locator handled in `source-serial-locator` -->
@@ -1195,9 +1242,8 @@
         </choose>
         <text macro="source-medium"/>
         <text macro="source-archive"/>
-        <text prefix="doi:" variable="DOI"/>
       </group>
-      <text macro="source-URL"/>
+      <text macro="source-accessed"/>
     </group>
   </macro>
   <!-- 3.1. Title -->
@@ -1231,12 +1277,12 @@
         <!-- If a review has no `reviewed-title`, assume that `title` contains the title of the reviewed work; the description provides it. -->
         <choose>
           <if variable="reviewed-genre title">
-            <!-- Quotes, title case -->
-            <text form="short" quotes="true" text-case="title" variable="title"/>
+            <!-- Quotes, sentence case -->
+            <text form="short" quotes="true" text-case="capitalize-first" variable="title"/>
           </if>
           <else-if variable="reviewed-genre reviewed-title title">
-            <!-- Quotes, title case -->
-            <text form="short" quotes="true" text-case="title" variable="title"/>
+            <!-- Quotes, sentence case -->
+            <text form="short" quotes="true" text-case="capitalize-first" variable="title"/>
           </else-if>
           <else>
             <text macro="description-short"/>
@@ -1278,16 +1324,16 @@
         <text form="short" text-case="capitalize-first" variable="part-title"/>
       </if>
       <else-if match="any" type="bill collection legislation regulation treaty">
-        <!-- No italics or quotes, title case -->
-        <text text-case="title" variable="part-title"/>
+        <!-- No italics or quotes, sentence case -->
+        <text text-case="capitalize-first" variable="part-title"/>
       </else-if>
       <else-if match="any" type="book classic graphic hearing map periodical">
         <!-- Italicized, title case (regardless of `container-title`) -->
         <text font-style="italic" text-case="title" variable="part-title"/>
       </else-if>
       <else-if type="entry-encyclopedia" variable="author container-title">
-        <!-- Signed encyclopedia entry in quotes, title case (cf. CMOS18 14.132) -->
-        <text quotes="true" text-case="title" variable="part-title"/>
+        <!-- Signed encyclopedia entry in quotes, sentence case (cf. CMOS18 14.132) -->
+        <text quotes="true" text-case="capitalize-first" variable="part-title"/>
       </else-if>
       <else-if type="entry-dictionary" variable="container-title">
         <!-- Quotes, sentence case -->
@@ -1309,8 +1355,8 @@
       <else-if variable="container-title">
         <choose>
           <if match="any" type="article-journal article-magazine article-newspaper periodical post-weblog review review-book">
-            <!-- Quotes, title case -->
-            <text quotes="true" text-case="title" variable="part-title"/>
+            <!-- Quotes, sentence case -->
+            <text quotes="true" text-case="capitalize-first" variable="part-title"/>
           </if>
           <else-if match="any" variable="collection-editor compiler editor editorial-director">
             <!-- monographic types -->
@@ -1319,8 +1365,8 @@
           </else-if>
           <else-if match="any" type="interview paper-conference">
             <!-- serial types -->
-            <!-- Quotes, title case -->
-            <text quotes="true" text-case="title" variable="part-title"/>
+            <!-- Quotes, sentence case -->
+            <text quotes="true" text-case="capitalize-first" variable="part-title"/>
           </else-if>
           <else>
             <!-- monographic types -->
@@ -1331,8 +1377,8 @@
       </else-if>
       <else-if match="any" type="article dataset document interview manuscript paper-conference personal_communication speech thesis webpage">
         <!-- Container-like but not necessarily with `container-title` -->
-        <!-- Quotes, title case -->
-        <text quotes="true" text-case="title" variable="part-title"/>
+        <!-- Quotes, sentence case -->
+        <text quotes="true" text-case="capitalize-first" variable="part-title"/>
       </else-if>
       <else>
         <!-- Italicized, title case (default) -->
@@ -1347,16 +1393,16 @@
         <text form="short" text-case="capitalize-first" variable="title"/>
       </if>
       <else-if match="any" type="bill collection legislation regulation treaty">
-        <!-- No italics or quotes, title case -->
-        <text text-case="title" variable="title"/>
+        <!-- No italics or quotes, sentence case -->
+        <text text-case="capitalize-first" variable="title"/>
       </else-if>
       <else-if match="any" type="book classic graphic hearing map periodical">
         <!-- Italicized, title case (regardless of `container-title`) -->
         <text font-style="italic" text-case="title" variable="title"/>
       </else-if>
       <else-if type="entry-encyclopedia" variable="author container-title">
-        <!-- Signed encyclopedia entry in quotes, title case (cf. CMOS18 14.132) -->
-        <text quotes="true" text-case="title" variable="title"/>
+        <!-- Signed encyclopedia entry in quotes, sentence case (cf. CMOS18 14.132) -->
+        <text quotes="true" text-case="capitalize-first" variable="title"/>
       </else-if>
       <else-if type="entry-dictionary" variable="container-title">
         <!-- Quotes, sentence case -->
@@ -1372,13 +1418,13 @@
       </else-if>
       <!-- Other types are formatted based on presence of `container-title` -->
       <else-if variable="container-title">
-        <!-- Quotes, title case -->
-        <text quotes="true" text-case="title" variable="title"/>
+        <!-- Quotes, sentence case -->
+        <text quotes="true" text-case="capitalize-first" variable="title"/>
       </else-if>
       <else-if match="any" type="article dataset document interview manuscript paper-conference personal_communication speech thesis webpage">
         <!-- Container-like but not necessarily with `container-title` -->
-        <!-- Quotes, title case -->
-        <text quotes="true" text-case="title" variable="title"/>
+        <!-- Quotes, sentence case -->
+        <text quotes="true" text-case="capitalize-first" variable="title"/>
       </else-if>
       <else>
         <!-- Italicized, title case (default) -->
@@ -1393,16 +1439,16 @@
         <text form="short" text-case="capitalize-first" variable="title"/>
       </if>
       <else-if match="any" type="bill collection legislation regulation treaty">
-        <!-- No italics or quotes, title case -->
-        <text form="short" text-case="title" variable="title"/>
+        <!-- No italics or quotes, sentence case -->
+        <text form="short" text-case="capitalize-first" variable="title"/>
       </else-if>
       <else-if match="any" type="book classic graphic hearing map periodical">
         <!-- Italicized, title case (regardless of `container-title`) -->
         <text font-style="italic" form="short" text-case="title" variable="title"/>
       </else-if>
       <else-if type="entry-encyclopedia" variable="author container-title">
-        <!-- Signed encyclopedia entry in quotes, title case (cf. CMOS18 14.132) -->
-        <text form="short" quotes="true" text-case="title" variable="title"/>
+        <!-- Signed encyclopedia entry in quotes, sentence case (cf. CMOS18 14.132) -->
+        <text form="short" quotes="true" text-case="capitalize-first" variable="title"/>
       </else-if>
       <else-if type="entry-dictionary" variable="container-title">
         <!-- Quotes, sentence case -->
@@ -1418,13 +1464,13 @@
       </else-if>
       <!-- Other types are formatted based on presence of `container-title` -->
       <else-if variable="container-title">
-        <!-- Quotes, title case -->
-        <text form="short" quotes="true" text-case="title" variable="title"/>
+        <!-- Quotes, sentence case -->
+        <text form="short" quotes="true" text-case="capitalize-first" variable="title"/>
       </else-if>
       <else-if match="any" type="article dataset document interview manuscript paper-conference personal_communication speech thesis webpage">
         <!-- Container-like but not necessarily with `container-title` -->
-        <!-- Quotes, title case -->
-        <text form="short" quotes="true" text-case="title" variable="title"/>
+        <!-- Quotes, sentence case -->
+        <text form="short" quotes="true" text-case="capitalize-first" variable="title"/>
       </else-if>
       <else>
         <!-- Italicized, title case (default) -->
@@ -1692,12 +1738,12 @@
               <if match="any" variable="container-title DOI issue number publisher publisher-place references URL volume">
                 <!-- interviewee appears in the description of a published interview only (MHRA 7.11) -->
                 <names variable="author">
-                  <name initialize="false"/>
+                  <name initialize="true"/>
                 </names>
               </if>
               <else>
                 <names variable="interviewer">
-                  <name initialize="false"/>
+                  <name initialize="true"/>
                 </names>
               </else>
             </choose>
@@ -1709,12 +1755,12 @@
               <if match="any" variable="container-title DOI issue number publisher publisher-place references URL volume">
                 <!-- interviewee appears in the description of a published interview only (MHRA 7.11) -->
                 <names variable="author">
-                  <name initialize="false"/>
+                  <name initialize="true"/>
                 </names>
               </if>
               <else>
                 <names variable="interviewer">
-                  <name initialize="false"/>
+                  <name initialize="true"/>
                 </names>
               </else>
             </choose>
@@ -1726,12 +1772,12 @@
               <if match="any" variable="container-title DOI issue number publisher publisher-place references URL volume">
                 <!-- interviewee appears in the description of a published interview only (MHRA 7.11) -->
                 <names variable="author">
-                  <name initialize="false"/>
+                  <name initialize="true"/>
                 </names>
               </if>
               <else>
                 <names variable="interviewer">
-                  <name initialize="false"/>
+                  <name initialize="true"/>
                 </names>
               </else>
             </choose>
@@ -1757,12 +1803,12 @@
               <if match="any" variable="container-title DOI issue number publisher publisher-place references URL volume">
                 <!-- interviewee appears in the description of a published interview only (MHRA 7.11) -->
                 <names variable="author">
-                  <name initialize="false"/>
+                  <name initialize="true"/>
                 </names>
               </if>
               <else>
                 <names variable="interviewer">
-                  <name initialize="false"/>
+                  <name initialize="true"/>
                 </names>
               </else>
             </choose>
@@ -1774,12 +1820,12 @@
               <if match="any" variable="container-title DOI issue number publisher publisher-place references URL volume">
                 <!-- interviewee appears in the description of a published interview only (MHRA 7.11) -->
                 <names variable="author">
-                  <name initialize="false"/>
+                  <name initialize="true"/>
                 </names>
               </if>
               <else>
                 <names variable="interviewer">
-                  <name initialize="false"/>
+                  <name initialize="true"/>
                 </names>
               </else>
             </choose>
@@ -1791,12 +1837,12 @@
               <if match="any" variable="container-title DOI issue number publisher publisher-place references URL volume">
                 <!-- interviewee appears in the description of a published interview only (MHRA 7.11) -->
                 <names variable="author">
-                  <name initialize="false"/>
+                  <name initialize="true"/>
                 </names>
               </if>
               <else>
                 <names variable="interviewer">
-                  <name initialize="false"/>
+                  <name initialize="true"/>
                 </names>
               </else>
             </choose>
@@ -1854,7 +1900,8 @@
               <!-- `genre` appears with `number` -->
               <names variable="recipient">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name font-variant="small-caps" initialize="true"/>
+                <et-al font-variant="normal"/>
               </names>
             </if>
             <else>
@@ -1862,7 +1909,8 @@
                 <text macro="description-format"/>
                 <names variable="recipient">
                   <label form="verb" suffix=" "/>
-                  <name initialize="false"/>
+                  <name font-variant="small-caps" initialize="true"/>
+                  <et-al font-variant="normal"/>
                 </names>
               </group>
             </else>
@@ -1885,7 +1933,8 @@
               <!-- `genre` appears with `number` -->
               <names variable="recipient">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name font-variant="small-caps" initialize="true"/>
+                <et-al font-variant="normal"/>
               </names>
             </if>
             <else>
@@ -1893,7 +1942,8 @@
                 <text macro="description-format-capitalized"/>
                 <names variable="recipient">
                   <label form="verb" suffix=" "/>
-                  <name initialize="false"/>
+                  <name font-variant="small-caps" initialize="true"/>
+                  <et-al font-variant="normal"/>
                 </names>
               </group>
             </else>
@@ -1979,12 +2029,12 @@
         </choose>
         <group delimiter=", ">
           <names variable="reviewed-author">
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <text macro="description-review-title"/>
           <text variable="reviewed-genre"/>
+          <text macro="source-event-place-first"/>
         </group>
-        <text macro="source-event-place-first" prefix="(" suffix=")"/>
       </group>
       <text macro="label-section"/>
     </group>
@@ -2003,7 +2053,7 @@
         </choose>
         <group delimiter=", ">
           <names variable="reviewed-author">
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <text macro="description-review-title"/>
           <text variable="reviewed-genre"/>
@@ -2023,7 +2073,7 @@
           <!-- cite under author if available (MHRA 7.12) -->
           <text term="review-of"/>
           <names variable="reviewed-author">
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <!-- use full name, not short -->
         </else-if>
@@ -2097,7 +2147,8 @@
         </if>
       </choose>
       <names variable="author">
-        <name delimiter-precedes-last="always" initialize="false"/>
+        <name delimiter-precedes-last="always" font-variant="small-caps"/>
+        <et-al font-variant="normal"/>
         <substitute>
           <names variable="performer"/>
         </substitute>
@@ -2116,7 +2167,7 @@
         </if>
       </choose>
       <names variable="author">
-        <name delimiter-precedes-last="always" initialize="false"/>
+        <name delimiter-precedes-last="always" initialize="true"/>
         <substitute>
           <names variable="performer"/>
         </substitute>
@@ -2159,7 +2210,8 @@
               <!-- if there is a volume or part, contributors except for the series editor follow the corresponding volume -->
               <names variable="collection-editor">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name font-variant="small-caps" initialize="true"/>
+                <et-al font-variant="normal"/>
               </names>
             </if>
             <else-if match="any" variable="number-of-volumes part-number volume">
@@ -2167,7 +2219,8 @@
                 <if variable="collection-editor">
                   <names variable="collection-editor">
                     <label form="verb-short" suffix=" "/>
-                    <name initialize="false"/>
+                    <name font-variant="small-caps" initialize="true"/>
+                    <et-al font-variant="normal"/>
                   </names>
                 </if>
                 <else>
@@ -2206,51 +2259,63 @@
         <if match="any" type="post webpage">
           <names variable="container-author">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name font-variant="small-caps" initialize="true"/>
+            <et-al font-variant="normal"/>
           </names>
           <names variable="editor-translator">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name font-variant="small-caps" initialize="true"/>
+            <et-al font-variant="normal"/>
           </names>
           <names delimiter=", " variable="editor translator">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name font-variant="small-caps" initialize="true"/>
+            <et-al font-variant="normal"/>
           </names>
           <names variable="editorial-director">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name font-variant="small-caps" initialize="true"/>
+            <et-al font-variant="normal"/>
           </names>
           <names variable="host">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name font-variant="small-caps" initialize="true"/>
+            <et-al font-variant="normal"/>
           </names>
           <names variable="guest">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name font-variant="small-caps" initialize="true"/>
+            <et-al font-variant="normal"/>
           </names>
           <names delimiter=", " variable="illustrator narrator">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name font-variant="small-caps" initialize="true"/>
+            <et-al font-variant="normal"/>
           </names>
           <names delimiter=", " variable="compiler chair organizer curator">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name font-variant="small-caps" initialize="true"/>
+            <et-al font-variant="normal"/>
           </names>
           <names delimiter=", " variable="series-creator executive-producer">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name font-variant="small-caps" initialize="true"/>
+            <et-al font-variant="normal"/>
           </names>
           <names variable="producer">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name font-variant="small-caps" initialize="true"/>
+            <et-al font-variant="normal"/>
           </names>
           <names variable="director">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name font-variant="small-caps" initialize="true"/>
+            <et-al font-variant="normal"/>
           </names>
           <names variable="performer">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name font-variant="small-caps" initialize="true"/>
+            <et-al font-variant="normal"/>
           </names>
           <choose>
             <if match="any" type="book broadcast chapter classic entry interview motion_picture musical_score pamphlet paper-conference report software"/>
@@ -2258,7 +2323,8 @@
             <else>
               <names variable="contributor">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name font-variant="small-caps" initialize="true"/>
+                <et-al font-variant="normal"/>
               </names>
             </else>
           </choose>
@@ -2269,59 +2335,72 @@
             <if match="none" variable="container-title">
               <names variable="container-author">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name font-variant="small-caps" initialize="true"/>
+                <et-al font-variant="normal"/>
               </names>
               <names variable="editor-translator">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name font-variant="small-caps" initialize="true"/>
+                <et-al font-variant="normal"/>
               </names>
               <names delimiter=", " variable="editor translator">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name font-variant="small-caps" initialize="true"/>
+                <et-al font-variant="normal"/>
               </names>
               <names variable="editorial-director">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name font-variant="small-caps" initialize="true"/>
+                <et-al font-variant="normal"/>
               </names>
               <names variable="host">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name font-variant="small-caps" initialize="true"/>
+                <et-al font-variant="normal"/>
               </names>
               <names variable="guest">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name font-variant="small-caps" initialize="true"/>
+                <et-al font-variant="normal"/>
               </names>
             </if>
           </choose>
           <names delimiter=", " variable="illustrator">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name font-variant="small-caps" initialize="true"/>
+            <et-al font-variant="normal"/>
           </names>
           <names delimiter=", " variable="narrator">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name font-variant="small-caps" initialize="true"/>
+            <et-al font-variant="normal"/>
           </names>
           <choose>
             <if match="none" variable="container-title">
               <names delimiter=", " variable="compiler chair organizer curator">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name font-variant="small-caps" initialize="true"/>
+                <et-al font-variant="normal"/>
               </names>
               <names delimiter=", " variable="series-creator executive-producer">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name font-variant="small-caps" initialize="true"/>
+                <et-al font-variant="normal"/>
               </names>
               <names variable="producer">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name font-variant="small-caps" initialize="true"/>
+                <et-al font-variant="normal"/>
               </names>
               <names variable="director">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name font-variant="small-caps" initialize="true"/>
+                <et-al font-variant="normal"/>
               </names>
               <names variable="performer">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name font-variant="small-caps" initialize="true"/>
+                <et-al font-variant="normal"/>
               </names>
               <choose>
                 <if match="any" type="book broadcast chapter classic entry interview motion_picture musical_score pamphlet paper-conference report software"/>
@@ -2330,7 +2409,8 @@
                 <else>
                   <names variable="contributor">
                     <label form="verb-short" suffix=" "/>
-                    <name initialize="false"/>
+                    <name font-variant="small-caps" initialize="true"/>
+                    <et-al font-variant="normal"/>
                   </names>
                 </else>
               </choose>
@@ -2341,7 +2421,8 @@
               <!-- Song contributors attached to album (cf. CMOS18 14.163) -->
               <names variable="contributor">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name font-variant="small-caps" initialize="true"/>
+                <et-al font-variant="normal"/>
               </names>
             </if>
           </choose>
@@ -2355,34 +2436,41 @@
         <if type="periodical">
           <names variable="editor">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name font-variant="small-caps" initialize="true"/>
+            <et-al font-variant="normal"/>
           </names>
         </if>
       </choose>
       <names delimiter=", " variable="translator narrator">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name font-variant="small-caps" initialize="true"/>
+        <et-al font-variant="normal"/>
       </names>
       <names delimiter=", " variable="compiler chair organizer curator">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name font-variant="small-caps" initialize="true"/>
+        <et-al font-variant="normal"/>
       </names>
       <names delimiter=", " variable="series-creator executive-producer">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name font-variant="small-caps" initialize="true"/>
+        <et-al font-variant="normal"/>
       </names>
       <names variable="producer">
         <!-- not abbreviated (cf. CMOS18 14.168) -->
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name font-variant="small-caps" initialize="true"/>
+        <et-al font-variant="normal"/>
       </names>
       <names variable="director">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name font-variant="small-caps" initialize="true"/>
+        <et-al font-variant="normal"/>
       </names>
       <names variable="performer">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name font-variant="small-caps" initialize="true"/>
+        <et-al font-variant="normal"/>
       </names>
     </group>
   </macro>
@@ -2671,7 +2759,6 @@
           </group>
         </if>
       </choose>
-      <text macro="source-series"/>
       <text macro="source-event"/>
       <text macro="label-version"/>
       <text macro="identifier-edition"/>
@@ -2741,7 +2828,8 @@
               <!-- if there is a volume or part, contributors except for the series editor follow the corresponding volume -->
               <names variable="collection-editor">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name font-variant="small-caps" initialize="true"/>
+                <et-al font-variant="normal"/>
               </names>
             </if>
             <else-if match="any" variable="number-of-volumes part-number volume">
@@ -2749,7 +2837,8 @@
                 <if variable="collection-editor">
                   <names variable="collection-editor">
                     <label form="verb-short" suffix=" "/>
-                    <name initialize="false"/>
+                    <name font-variant="small-caps" initialize="true"/>
+                    <et-al font-variant="normal"/>
                   </names>
                 </if>
                 <else>
@@ -2769,47 +2858,58 @@
     <group delimiter=", ">
       <names variable="editor-translator">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name font-variant="small-caps" initialize="true"/>
+        <et-al font-variant="normal"/>
       </names>
       <names delimiter=", " variable="editor translator">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name font-variant="small-caps" initialize="true"/>
+        <et-al font-variant="normal"/>
       </names>
       <names variable="host">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name font-variant="small-caps" initialize="true"/>
+        <et-al font-variant="normal"/>
       </names>
       <names variable="guest">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name font-variant="small-caps" initialize="true"/>
+        <et-al font-variant="normal"/>
       </names>
       <names delimiter=", " variable="chair organizer">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name font-variant="small-caps" initialize="true"/>
+        <et-al font-variant="normal"/>
       </names>
       <names delimiter=", " variable="illustrator narrator compiler curator">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name font-variant="small-caps" initialize="true"/>
+        <et-al font-variant="normal"/>
       </names>
       <names delimiter=", " variable="series-creator executive-producer">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name font-variant="small-caps" initialize="true"/>
+        <et-al font-variant="normal"/>
       </names>
       <names variable="producer">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name font-variant="small-caps" initialize="true"/>
+        <et-al font-variant="normal"/>
       </names>
       <names variable="editorial-director">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name font-variant="small-caps" initialize="true"/>
+        <et-al font-variant="normal"/>
       </names>
       <names variable="director">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name font-variant="small-caps" initialize="true"/>
+        <et-al font-variant="normal"/>
       </names>
       <names variable="performer">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name font-variant="small-caps" initialize="true"/>
+        <et-al font-variant="normal"/>
       </names>
       <choose>
         <if match="any" type="book broadcast chapter classic entry interview motion_picture musical_score pamphlet paper-conference report software"/>
@@ -2819,7 +2919,8 @@
           <!-- Song contributors attached to album (cf. CMOS18 14.163) -->
           <names variable="contributor">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name font-variant="small-caps" initialize="true"/>
+            <et-al font-variant="normal"/>
           </names>
         </else>
       </choose>
@@ -2858,7 +2959,7 @@
     </group>
   </macro>
   <macro name="source-monographic-locator-volume">
-    <group delimiter=" ">
+    <group delimiter=", ">
       <group delimiter=", ">
         <text macro="label-volume"/>
         <text macro="title-volume"/>
@@ -2886,13 +2987,13 @@
       <!-- remove volume `issued` date from this position in author-date -->
       <choose>
         <if variable="available-date issued volume">
-          <text macro="date-issued-year" prefix="(" suffix=")"/>
+          <text macro="date-issued-year"/>
         </if>
         <else-if variable="available-date issued part-number">
-          <text macro="date-issued-year" prefix="(" suffix=")"/>
+          <text macro="date-issued-year"/>
         </else-if>
         <else-if type="broadcast" variable="available-date issued number">
-          <text macro="date-issued-year" prefix="(" suffix=")"/>
+          <text macro="date-issued-year"/>
         </else-if>
       </choose>
     </group>
@@ -3018,19 +3119,17 @@
   </macro>
   <macro name="source-event-unpublished-paper-conference">
     <!-- place and date are rendered in brackets for an unpublished conference paper (MHRA 7.10.vi) -->
-    <group delimiter=" ">
+    <group delimiter=", ">
       <text macro="source-event-title"/>
-      <group delimiter=", " prefix="(" suffix=")">
-        <text variable="event-place"/>
-        <choose>
-          <if variable="event-date">
-            <text macro="date-event-full"/>
-          </if>
-          <else>
-            <text macro="date-issued-full"/>
-          </else>
-        </choose>
-      </group>
+      <text variable="event-place"/>
+      <choose>
+        <if variable="event-date">
+          <text macro="date-event-full"/>
+        </if>
+        <else>
+          <text macro="date-issued-full"/>
+        </else>
+      </choose>
     </group>
   </macro>
   <!-- 4.5. Facts of publication -->
@@ -3059,24 +3158,24 @@
     <!-- Facts of publication with brackets for print monographic formats -->
     <choose>
       <if match="any" type="book broadcast chapter classic entry interview motion_picture musical_score pamphlet paper-conference report software song standard thesis">
-        <text macro="source-monographic-publication" prefix="(" suffix=")"/>
+        <text macro="source-monographic-publication"/>
       </if>
       <else-if type="article" variable="genre publisher">
         <!-- without brackets for a preprint in a repository (cf. CMOS18 14.76), with brackets for a working paper (cf. CMOS18 14.116) -->
-        <text macro="source-monographic-publication" prefix="(" suffix=")"/>
+        <text macro="source-monographic-publication"/>
       </else-if>
       <!-- dictionary/encyclopedia entries probably are bracketed only with publisher (MHRA 7.5) -->
       <else-if type="entry-dictionary" variable="publisher">
-        <text macro="source-monographic-publication" prefix="(" suffix=")"/>
+        <text macro="source-monographic-publication"/>
       </else-if>
       <else-if type="entry-encyclopedia" variable="publisher">
-        <text macro="source-monographic-publication" prefix="(" suffix=")"/>
+        <text macro="source-monographic-publication"/>
       </else-if>
       <else-if type="personal_communication">
         <choose>
           <if match="any" variable="collection-editor compiler editor editorial-director issue page supplement-number volume">
             <!-- in an edited collection -->
-            <text macro="source-monographic-publication" prefix="(" suffix=")"/>
+            <text macro="source-monographic-publication"/>
           </if>
         </choose>
       </else-if>
@@ -3104,7 +3203,7 @@
   </macro>
   <macro name="source-monographic-publication-short-bracketed">
     <!-- mirrors `source-monographic-publication-bracketed` -->
-    <group delimiter=", " prefix="(" suffix=")">
+    <group delimiter=", ">
       <choose>
         <if disambiguate="true" match="none"/>
         <!-- Facts of publication with brackets for print monographic formats -->
@@ -3248,11 +3347,9 @@
           <text term="original-work-published"/>
           <group delimiter=", ">
             <names variable="original-author">
-              <name initialize="false"/>
+              <name initialize="true"/>
             </names>
             <text font-style="italic" text-case="title" variable="original-title"/>
-          </group>
-          <group delimiter=", " prefix="(" suffix=")">
             <text macro="source-publication-publisher-original"/>
             <text macro="source-date-original"/>
           </group>
@@ -3263,26 +3360,21 @@
   <macro name="source-publication-publisher">
     <group delimiter=" ">
       <choose>
-        <if type="thesis" variable="publisher">
+        <if match="any" type="book chapter classic entry entry-dictionary entry-encyclopedia interview musical_score pamphlet paper-conference report">
+          <group delimiter=": ">
+            <choose>
+              <if variable="publisher-place">
+                <text variable="publisher-place"/>
+              </if>
+              <else-if match="none" type="classic entry entry-dictionary entry-encyclopedia" variable="original-publisher-place">
+                <text form="short" term="no-place"/>
+              </else-if>
+            </choose>
+            <!-- <text variable="publisher"/> -->
+          </group>
+        </if>
+        <else>
           <text variable="publisher"/>
-        </if>
-        <else-if type="motion_picture" variable="publisher-place">
-          <!-- prioritize the country of origin for films (MHRA 7.8) -->
-          <text variable="publisher-place"/>
-        </else-if>
-        <else-if type="motion_picture" variable="event-place">
-          <text variable="event-place"/>
-        </else-if>
-        <else-if variable="publisher">
-          <group delimiter=": ">
-            <!-- <text variable="publisher-place"/> -->
-            <text variable="publisher"/>
-          </group>
-        </else-if>
-        <!-- TODO: remove conditional when Zotero fixes double-mapping of `event-place` -->
-        <else-if match="any" variable="event-date event-title"/>
-        <else>
-          <text variable="publisher-place"/>
         </else>
       </choose>
       <choose>
@@ -3297,7 +3389,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text variable="original-publisher-place"/> -->
+          <text variable="original-publisher-place"/>
           <text variable="original-publisher"/>
         </group>
       </if>
@@ -3872,7 +3964,8 @@
         <if disambiguate="true">
           <names variable="editor-translator">
             <label form="verb-short" suffix=" "/>
-            <name form="short" initialize="true"/>
+            <name font-variant="small-caps" form="short" initialize="true"/>
+            <et-al font-variant="normal"/>
             <substitute>
               <names variable="editor"/>
               <names variable="translator"/>
@@ -4020,7 +4113,7 @@
       <key macro="source-archive"/>
     </sort>
     <layout>
-      <text macro="bibliography-filtered-notes"/>
+      <text macro="bibliography-notes"/>
     </layout>
   </bibliography>
 </style>
