--- templates/mhra-template.csl
+++ development/mhra-notes-publisher-place-no-url.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style and="text" class="note" default-locale="en-GB" demote-non-dropping-particle="display-and-sort" et-al-min="4" et-al-use-first="1" page-range-format="minimal-two" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>MHRA Style Guide 4th edition (template)</title>
-    <title-short>Modern Humanities Research Association</title-short>
-    <id>http://www.zotero.org/styles/mhra-notes</id>
-    <link href="http://www.zotero.org/styles/mhra-notes" rel="self"/>
+    <title>MHRA Style Guide 4th edition (notes, place of publication, without URLs)</title>
+    <title-short>Modern Humanities Research Association (notes, place of publication, no URLs)</title-short>
+    <id>http://www.zotero.org/styles/mhra-notes-publisher-place-no-url</id>
+    <link href="http://www.zotero.org/styles/mhra-notes-publisher-place-no-url" rel="self"/>
+    <link href="http://www.zotero.org/styles/mhra-notes" rel="template"/>
     <link href="https://www.mhra.org.uk/style/" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -22,7 +23,7 @@
     <category field="literature"/>
     <category field="philosophy"/>
     <category field="theology"/>
-    <summary>MHRA source citations, template</summary>
+    <summary>MHRA source citations, notes system, with place of publication (MHRA 7.3), without URLs</summary>
     <updated>2025-11-23T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -1146,9 +1147,7 @@
         </choose>
         <text macro="source-medium"/>
         <text macro="source-archive"/>
-        <text prefix="doi:" variable="DOI"/>
       </group>
-      <text macro="source-URL"/>
     </group>
   </macro>
   <!-- 3.1. Title -->
@@ -3293,7 +3292,7 @@
         </else-if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text variable="publisher-place"/> -->
+            <text variable="publisher-place"/>
             <text variable="publisher"/>
           </group>
         </else-if>
@@ -3315,7 +3314,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text variable="original-publisher-place"/> -->
+          <text variable="original-publisher-place"/>
           <text variable="original-publisher"/>
         </group>
       </if>
