--- templates/modern-humanities-research-association-template.csl
+++ development/modern-humanities-research-association-notes-no-url.csl
@@ -5,10 +5,10 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style and="text" class="note" default-locale="en-GB" demote-non-dropping-particle="display-and-sort" et-al-min="4" et-al-use-first="1" page-range-format="minimal-two" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Modern Humanities Research Association 4th edition (template)</title>
+    <title>Modern Humanities Research Association 4th edition (notes, without URLs)</title>
     <title-short>MHRA Style Guide</title-short>
-    <id>http://www.zotero.org/styles/modern-humanities-research-association-notes</id>
-    <link href="http://www.zotero.org/styles/modern-humanities-research-association-notes" rel="self"/>
+    <id>http://www.zotero.org/styles/modern-humanities-research-association-notes-no-url</id>
+    <link href="http://www.zotero.org/styles/modern-humanities-research-association-notes-no-url" rel="self"/>
     <link href="https://www.mhra.org.uk/style/" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -22,7 +22,7 @@
     <category field="literature"/>
     <category field="philosophy"/>
     <category field="theology"/>
-    <summary>MHRA source citations, template</summary>
+    <summary>MHRA source citations, notes system, without URLs</summary>
     <updated>2025-05-19T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -1240,9 +1240,7 @@
         </choose>
         <text macro="source-medium"/>
         <text macro="source-archive"/>
-        <text macro="source-DOI"/>
       </group>
-      <text macro="source-URL"/>
     </group>
   </macro>
   <!-- 3.1. Title -->
