--- templates/modern-humanities-research-association-template.csl
+++ development/modern-humanities-research-association-publisher-place.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style and="text" class="note" default-locale="en-GB" demote-non-dropping-particle="sort-only" et-al-min="4" et-al-use-first="1" page-range-format="minimal-two" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Modern Humanities Research Association 4th edition (template)</title>
+    <title>Modern Humanities Research Association 4th edition (notes with place of publication)</title>
     <title-short>MHRA Style Guide</title-short>
-    <id>http://www.zotero.org/styles/modern-humanities-research-association</id>
-    <link href="http://www.zotero.org/styles/modern-humanities-research-association" rel="self"/>
+    <id>http://www.zotero.org/styles/modern-humanities-research-association-publisher-place</id>
+    <link href="http://www.zotero.org/styles/modern-humanities-research-association-publisher-place" rel="self"/>
+    <link href="http://www.zotero.org/styles/modern-humanities-research-association" rel="template"/>
     <link href="https://www.mhra.org.uk/style/" rel="documentation"/>
     <author>
       <name>Rintze Zelle</name>
@@ -430,7 +431,7 @@
       </if>
       <else-if variable="publisher">
         <group delimiter=": ">
-          <!-- <text variable="publisher-place"/> -->
+          <text variable="publisher-place"/>
           <text variable="publisher"/>
         </group>
       </else-if>
