--- templates/modern-language-association-template.csl
+++ development/modern-language-association-no-url.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style and="text" class="in-text" demote-non-dropping-particle="never" initialize-with=". " page-range-format="minimal-two" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>MLA Handbook 9th edition</title>
-    <title-short>Modern Language Association</title-short>
-    <id>http://www.zotero.org/styles/modern-language-association</id>
-    <link href="http://www.zotero.org/styles/modern-language-association" rel="self"/>
+    <title>MLA Handbook 9th edition (in-text citations, without URLs)</title>
+    <title-short>Modern Language Association (in-text citations, no URLs)</title-short>
+    <id>http://www.zotero.org/styles/modern-language-association-no-url</id>
+    <link href="http://www.zotero.org/styles/modern-language-association-no-url" rel="self"/>
+    <link href="http://www.zotero.org/styles/modern-language-association" rel="template"/>
     <link href="https://style.mla.org/" rel="documentation"/>
     <author>
       <name>Sebastian Karcher</name>
@@ -25,7 +26,7 @@
     <category field="generic-base"/>
     <category field="humanities"/>
     <category field="literature"/>
-    <summary>MLA source citations</summary>
+    <summary>MLA source citations, in-text citations system (chapter 6), without URLs (5.84)</summary>
     <updated>2025-07-15T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -867,11 +868,6 @@
       <text variable="archive_collection"/>
       <text variable="archive_location"/>
       <text macro="label-page"/>
-      <choose>
-        <if match="none" variable="source">
-          <text macro="container1-location-URI"/>
-        </if>
-      </choose>
     </group>
   </macro>
   <macro name="container1-location-event">
@@ -1168,7 +1164,6 @@
       <key macro="supplemental-element-after-title"/>
       <key macro="container1"/>
       <key macro="supplemental-element-between"/>
-      <key macro="container2"/>
       <key macro="supplemental-element-end"/>
     </sort>
     <layout suffix=".">
@@ -1178,7 +1173,6 @@
         <text macro="supplemental-element-after-title"/>
         <text macro="container1"/>
         <text macro="supplemental-element-between"/>
-        <text macro="container2"/>
         <text macro="supplemental-element-end"/>
       </group>
     </layout>
