--- templates/modern-humanities-research-association-template.csl
+++ development/modern-humanities-research-association-author-date.csl
@@ -3,12 +3,13 @@
   type="application/relax-ng-compact-syntax" schematypens="http://relaxng.org/ns/compatibility/annotations/1.0"?>
 <?xml-model href="https://raw.githubusercontent.com/citation-style-language/schema/refs/heads/master/schemas/styles/csl.rnc"
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-<style and="text" class="note" default-locale="en-GB" demote-non-dropping-particle="sort-only" et-al-min="4" et-al-use-first="1" page-range-format="minimal-two" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
+<style and="text" class="in-text" default-locale="en-GB" demote-non-dropping-particle="sort-only" et-al-min="4" et-al-use-first="1" page-range-format="minimal-two" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Modern Humanities Research Association 4th edition (template)</title>
+    <title>Modern Humanities Research Association 4th edition (author-date)</title>
     <title-short>MHRA Style Guide</title-short>
-    <id>http://www.zotero.org/styles/modern-humanities-research-association</id>
-    <link href="http://www.zotero.org/styles/modern-humanities-research-association" rel="self"/>
+    <id>http://www.zotero.org/styles/modern-humanities-research-association-author-date</id>
+    <link href="http://www.zotero.org/styles/modern-humanities-research-association-author-date" rel="self"/>
+    <link href="http://www.zotero.org/styles/modern-humanities-research-association" rel="template"/>
     <link href="https://www.mhra.org.uk/style/" rel="documentation"/>
     <author>
       <name>Rintze Zelle</name>
@@ -25,7 +26,7 @@
     <contributor>
       <name>Patrick O'Brien</name>
     </contributor>
-    <category citation-format="note"/>
+    <category citation-format="author-date"/>
     <category field="generic-base"/>
     <category field="history"/>
     <category field="humanities"/>
@@ -33,7 +34,7 @@
     <category field="literature"/>
     <category field="philosophy"/>
     <category field="theology"/>
-    <summary>MHRA Style Guide, notes system</summary>
+    <summary>MHRA Style Guide, author-date system</summary>
     <updated>2025-05-25T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -108,7 +109,11 @@
   <macro name="label-number-of-volumes">
     <group delimiter=" ">
       <text variable="number-of-volumes"/>
-      <label form="short" variable="number-of-volumes"/>
+      <choose>
+        <if is-numeric="number-of-volumes">
+          <label form="short" variable="number-of-volumes"/>
+        </if>
+      </choose>
     </group>
   </macro>
   <macro name="label-page">
@@ -228,10 +233,7 @@
   <macro name="author-short">
     <group delimiter=" ">
       <names variable="author">
-        <name form="short"/>
-        <!-- for author-date:
         <name form="short" initialize-with=". "/>
-        -->
         <substitute>
           <names variable="editor"/>
           <names variable="translator"/>
@@ -451,16 +453,7 @@
       <if match="any" type="report article-newspaper article-magazine personal_communication">
         <date form="text" variable="issued"/>
       </if>
-      <else-if match="any" type="article-journal book bill chapter classic legislation motion_picture pamphlet paper-conference song thesis">
-        <choose>
-          <if is-uncertain-date="issued">
-            <date date-parts="year" form="numeric" prefix="[" suffix="?]" variable="issued"/>
-          </if>
-          <else>
-            <date date-parts="year" form="numeric" variable="issued"/>
-          </else>
-        </choose>
-      </else-if>
+      <else-if match="any" type="article-journal book bill chapter classic legislation motion_picture pamphlet paper-conference song thesis"/>
       <else>
         <choose>
           <if is-uncertain-date="issued">
@@ -699,9 +692,9 @@
       </else>
     </choose>
   </macro>
-  <citation disambiguate-add-givenname="true" disambiguate-add-names="true">
-    <layout delimiter="; " suffix=".">
-      <text macro="citation-notes"/>
+  <citation collapse="year" disambiguate-add-givenname="true" disambiguate-add-year-suffix="true">
+    <layout delimiter="; " prefix="(" suffix=")">
+      <text macro="citation-author-date"/>
     </layout>
   </citation>
   <macro name="bibliography-author-date">
@@ -758,11 +751,11 @@
   <bibliography delimiter-precedes-et-al="after-inverted-name" hanging-indent="true" subsequent-author-substitute="&#8212;&#8212;">
     <sort>
       <key macro="author-bib"/>
+      <key variable="issued"/>
       <key variable="title"/>
-      <key variable="issued"/>
     </sort>
     <layout>
-      <text macro="bibliography-notes"/>
+      <text macro="bibliography-author-date"/>
     </layout>
   </bibliography>
 </style>
