--- templates/modern-humanities-research-association-template.csl
+++ development/modern-humanities-research-association.csl
@@ -5,7 +5,7 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style and="text" class="note" default-locale="en-GB" demote-non-dropping-particle="sort-only" et-al-min="4" et-al-use-first="1" page-range-format="minimal-two" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Modern Humanities Research Association 4th edition (template)</title>
+    <title>Modern Humanities Research Association 4th edition (note with bibliography)</title>
     <title-short>MHRA Style Guide</title-short>
     <id>http://www.zotero.org/styles/modern-humanities-research-association</id>
     <link href="http://www.zotero.org/styles/modern-humanities-research-association" rel="self"/>
