--- templates/modern-language-association-template.csl
+++ development/modern-language-association-notes.csl
@@ -3,12 +3,13 @@
   type="application/relax-ng-compact-syntax" schematypens="http://relaxng.org/ns/compatibility/annotations/1.0"?>
 <?xml-model href="https://raw.githubusercontent.com/citation-style-language/schema/refs/heads/master/schemas/styles/csl-repository.rnc"
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-<style and="text" class="in-text" demote-non-dropping-particle="never" page-range-format="minimal-two" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
+<style and="text" class="note" demote-non-dropping-particle="never" page-range-format="minimal-two" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>MLA Handbook 9th edition</title>
-    <title-short>Modern Language Association</title-short>
-    <id>http://www.zotero.org/styles/modern-language-association</id>
-    <link href="http://www.zotero.org/styles/modern-language-association" rel="self"/>
+    <title>MLA Handbook 9th edition (notes)</title>
+    <title-short>Modern Language Association (notes)</title-short>
+    <id>http://www.zotero.org/styles/modern-language-association-notes</id>
+    <link href="http://www.zotero.org/styles/modern-language-association-notes" rel="self"/>
+    <link href="http://www.zotero.org/styles/modern-language-association" rel="template"/>
     <link href="https://style.mla.org/" rel="documentation"/>
     <author>
       <name>Sebastian Karcher</name>
@@ -21,11 +22,11 @@
     <contributor>
       <name>Patrick O'Brien</name>
     </contributor>
-    <category citation-format="author"/>
+    <category citation-format="note"/>
     <category field="generic-base"/>
     <category field="humanities"/>
     <category field="literature"/>
-    <summary>MLA source citations</summary>
+    <summary>MLA source citations, notes system (chapter 7)</summary>
     <updated>2025-07-15T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -1183,7 +1184,7 @@
     </group>
   </macro>
   <citation disambiguate-add-givenname="true" disambiguate-add-names="true" et-al-min="3" et-al-use-first="1">
-    <layout delimiter="; " prefix="(" suffix=")">
+    <layout delimiter="; " suffix=".">
       <choose>
         <if locator="page line" match="any">
           <group delimiter=" ">
