--- templates/modern-humanities-research-association-template.csl
+++ development/modern-humanities-research-association-notes-publisher-place.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style and="text" class="note" default-locale="en-GB" demote-non-dropping-particle="display-and-sort" et-al-min="4" et-al-use-first="1" page-range-format="minimal-two" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Modern Humanities Research Association 4th edition (template)</title>
-    <title-short>MHRA Style Guide</title-short>
-    <id>http://www.zotero.org/styles/modern-humanities-research-association-notes</id>
-    <link href="http://www.zotero.org/styles/modern-humanities-research-association-notes" rel="self"/>
+    <title>Modern Humanities Research Association 4th edition (notes with place of publication)</title>
+    <title-short>MHRA Style Guide (notes with place of publication)</title-short>
+    <id>http://www.zotero.org/styles/modern-humanities-research-association-notes-publisher-place</id>
+    <link href="http://www.zotero.org/styles/modern-humanities-research-association-notes-publisher-place" rel="self"/>
+    <link href="http://www.zotero.org/styles/modern-humanities-research-association-notes" rel="template"/>
     <link href="https://www.mhra.org.uk/style/" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -22,7 +23,7 @@
     <category field="literature"/>
     <category field="philosophy"/>
     <category field="theology"/>
-    <summary>MHRA source citations, template</summary>
+    <summary>MHRA source citations, notes system, with place of publication (MHRA 7.3)</summary>
     <updated>2025-05-19T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -3294,7 +3295,7 @@
         </if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text variable="publisher-place"/> -->
+            <text variable="publisher-place"/>
             <text variable="publisher"/>
           </group>
         </else-if>
@@ -3316,7 +3317,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text variable="original-publisher-place"/> -->
+          <text variable="original-publisher-place"/>
           <text variable="original-publisher"/>
         </group>
       </if>
