--- templates/modern-language-association-template.csl
+++ development/modern-language-association.csl
@@ -5,7 +5,7 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style and="text" class="in-text" demote-non-dropping-particle="never" page-range-format="minimal-two" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Modern Language Association 9th edition (template)</title>
+    <title>Modern Language Association 9th edition</title>
     <title-short>MLA Handbook</title-short>
     <id>http://www.zotero.org/styles/modern-language-association</id>
     <link href="http://www.zotero.org/styles/modern-language-association" rel="self"/>
