--- templates/new-harts-rules-template.csl
+++ development/new-harts-rules-numbered.csl
@@ -3,18 +3,19 @@
   type="application/relax-ng-compact-syntax" schematypens="http://relaxng.org/ns/compatibility/annotations/1.0"?>
 <?xml-model href="https://raw.githubusercontent.com/citation-style-language/schema/refs/heads/master/schemas/styles/csl-repository.rnc"
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-<style and="text" class="note" default-locale="en-GB" demote-non-dropping-particle="display-and-sort" et-al-min="4" et-al-use-first="1" page-range-format="minimal" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
+<style and="text" class="in-text" default-locale="en-GB" demote-non-dropping-particle="display-and-sort" et-al-min="4" et-al-use-first="1" initialize-with=". " page-range-format="minimal" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>New Hart's Rules: The Oxford Style Guide 2nd edition (template)</title>
-    <title-short>NHR (template)</title-short>
-    <id>http://www.zotero.org/styles/new-harts-rules-notes</id>
-    <link href="http://www.zotero.org/styles/new-harts-rules-notes" rel="self"/>
+    <title>New Hart's Rules: The Oxford Style Guide 2nd edition (numbered)</title>
+    <title-short>NHR (author-number/Vancouver [17.4])</title-short>
+    <id>http://www.zotero.org/styles/new-harts-rules-numbered</id>
+    <link href="http://www.zotero.org/styles/new-harts-rules-numbered" rel="self"/>
+    <link href="http://www.zotero.org/styles/new-harts-rules-notes" rel="template"/>
     <link href="https://en.wikipedia.org/wiki/Hart's_Rules" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
       <uri>https://orcid.org/0000-0003-0464-5036</uri>
     </author>
-    <category citation-format="note"/>
+    <category citation-format="numeric"/>
     <category field="anthropology"/>
     <category field="communications"/>
     <category field="generic-base"/>
@@ -30,7 +31,7 @@
     <category field="social_science"/>
     <category field="sociology"/>
     <category field="theology"/>
-    <summary>Oxford source citations, template</summary>
+    <summary>Oxford source citations, numbered system</summary>
     <updated>2025-11-05T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -3897,9 +3898,12 @@
       </choose>
     </group>
   </macro>
-  <citation disambiguate-add-givenname="true" disambiguate-add-names="true">
-    <layout delimiter="; " suffix=".">
-      <text macro="citation-notes-full-subsequent-author-title"/>
+  <citation collapse="citation-number">
+    <sort>
+      <key variable="citation-number"/>
+    </sort>
+    <layout delimiter="," vertical-align="sup">
+      <text variable="citation-number"/>
     </layout>
   </citation>
   <!-- Bibliography -->
@@ -4009,7 +4013,7 @@
       </else>
     </choose>
   </macro>
-  <bibliography hanging-indent="true">
+  <bibliography hanging-indent="true" second-field-align="flush">
     <sort>
       <key macro="author-bib"/>
       <key macro="title-and-descriptions"/>
@@ -4022,7 +4026,7 @@
       <key macro="source-archive"/>
     </sort>
     <layout suffix=".">
-      <text macro="bibliography-filtered-notes"/>
+      <text macro="bibliography-numbered"/>
     </layout>
   </bibliography>
 </style>
