--- templates/new-harts-rules-template.csl
+++ development/new-harts-rules-notes-initials-publisher-no-url.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style and="text" class="note" default-locale="en-GB" demote-non-dropping-particle="display-and-sort" et-al-min="4" et-al-use-first="1" initialize-with=". " page-range-format="minimal" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>New Hart's Rules: The Oxford Style Guide 2nd edition (template)</title>
-    <title-short>NHR (template)</title-short>
-    <id>http://www.zotero.org/styles/new-harts-rules-notes</id>
-    <link href="http://www.zotero.org/styles/new-harts-rules-notes" rel="self"/>
+    <title>New Hart's Rules: The Oxford Style Guide 2nd edition (notes, author initials, publisher, without URLs)</title>
+    <title-short>NHR (notes [17.2], author initials [18.2.2], publisher [18.2.10], no URLs))</title-short>
+    <id>http://www.zotero.org/styles/new-harts-rules-notes-initials-publisher-no-url</id>
+    <link href="http://www.zotero.org/styles/new-harts-rules-notes-initials-publisher-no-url" rel="self"/>
+    <link href="http://www.zotero.org/styles/new-harts-rules-notes" rel="template"/>
     <link href="https://en.wikipedia.org/wiki/Hart's_Rules" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -30,7 +31,7 @@
     <category field="social_science"/>
     <category field="sociology"/>
     <category field="theology"/>
-    <summary>Oxford source citations, template</summary>
+    <summary>Oxford source citations, notes system, author forenames reduced to initials (NHR 18.2.2), publisher name (NHR 18.2.10), without URLs</summary>
     <updated>2026-02-05T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -381,9 +382,9 @@
   <macro name="author-bib">
     <group delimiter=", ">
       <names variable="composer">
-        <name delimiter-precedes-et-al="after-inverted-name" delimiter-precedes-last="always" initialize="false" name-as-sort-order="first"/>
+        <name delimiter-precedes-et-al="after-inverted-name" delimiter-precedes-last="always" initialize="true" name-as-sort-order="first"/>
         <!-- for OGS: -->
-        <!-- <name delimiter-precedes-et-al="after-inverted-name" delimiter-precedes-last="always" font-variant="small-caps" initialize="false" name-as-sort-order="first"/>
+        <!-- <name delimiter-precedes-et-al="after-inverted-name" delimiter-precedes-last="always" font-variant="small-caps" initialize="true" name-as-sort-order="first"/>
         <et-al font-variant="normal"/> -->
         <label form="short" prefix=", "/>
         <!-- for OGS or variants with bracketed roles: -->
@@ -528,7 +529,7 @@
   <macro name="author-note">
     <group delimiter=", ">
       <names variable="composer">
-        <name initialize="false"/>
+        <name initialize="true"/>
         <label form="short" prefix=", "/>
         <!-- for OGS or variants with bracketed roles: -->
         <!-- <label form="short" prefix=" (" suffix=")"/> -->
@@ -1213,7 +1214,7 @@
         <text macro="source-medium" prefix="[" suffix="]"/>
       </group>
       <text macro="source-archive"/>
-      <text macro="source-date-accessed-DOI-URL"/>
+      <text macro="source-date-accessed"/>
     </group>
   </macro>
   <!-- 3.1. Title -->
@@ -1741,7 +1742,7 @@
           <!-- `genre` printed with `number` -->
           <names variable="interviewer">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
         </if>
         <else-if variable="genre">
@@ -1750,7 +1751,7 @@
             <group delimiter=" ">
               <text form="verb" term="container-author"/>
               <names variable="interviewer">
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
             </group>
           </group>
@@ -1758,7 +1759,7 @@
         <else-if variable="interviewer">
           <names variable="interviewer">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
         </else-if>
         <else>
@@ -1775,7 +1776,7 @@
           <!-- `genre` printed with `number` -->
           <names variable="interviewer">
             <label form="verb" suffix=" " text-case="capitalize-first"/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
         </if>
         <else-if variable="genre">
@@ -1784,7 +1785,7 @@
             <group delimiter=" ">
               <text form="verb" term="container-author"/>
               <names variable="interviewer">
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
             </group>
           </group>
@@ -1792,7 +1793,7 @@
         <else-if variable="interviewer">
           <names variable="interviewer">
             <label form="verb" suffix=" " text-case="capitalize-first"/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
         </else-if>
         <else>
@@ -1839,7 +1840,7 @@
               <!-- `genre` appears with `number` -->
               <names variable="recipient">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
             </if>
             <else>
@@ -1847,7 +1848,7 @@
                 <text macro="description-format"/>
                 <names variable="recipient">
                   <label form="verb" suffix=" "/>
-                  <name initialize="false"/>
+                  <name initialize="true"/>
                 </names>
               </group>
             </else>
@@ -1870,7 +1871,7 @@
               <!-- `genre` appears with `number` -->
               <names variable="recipient">
                 <label form="verb" suffix=" " text-case="capitalize-first"/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
             </if>
             <else>
@@ -1878,7 +1879,7 @@
                 <text macro="description-format-capitalized"/>
                 <names variable="recipient">
                   <label form="verb" suffix=" "/>
-                  <name initialize="false"/>
+                  <name initialize="true"/>
                 </names>
               </group>
             </else>
@@ -1964,7 +1965,7 @@
         </choose>
         <group delimiter=", ">
           <names variable="reviewed-author">
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <text macro="description-review-title"/>
           <text variable="reviewed-genre"/>
@@ -1988,7 +1989,7 @@
         </choose>
         <group delimiter=", ">
           <names variable="reviewed-author">
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <text macro="description-review-title"/>
           <text variable="reviewed-genre"/>
@@ -2074,7 +2075,7 @@
         </if>
       </choose>
       <names variable="author">
-        <name delimiter-precedes-last="always" initialize="false"/>
+        <name delimiter-precedes-last="always" initialize="true"/>
         <substitute>
           <names variable="performer"/>
         </substitute>
@@ -2093,7 +2094,7 @@
         </if>
       </choose>
       <names variable="author">
-        <name delimiter-precedes-last="always" initialize="false"/>
+        <name delimiter-precedes-last="always" initialize="true"/>
         <substitute>
           <names variable="performer"/>
         </substitute>
@@ -2164,15 +2165,15 @@
         <if match="any" type="post webpage">
           <names variable="container-author">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names variable="editor-translator">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names delimiter=", " variable="editor translator">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <choose>
             <if variable="collection-editor editor"/>
@@ -2180,67 +2181,67 @@
             <else-if match="any" variable="number-of-volumes part-number part-title volume volume-title">
               <names variable="collection-editor">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
             </else-if>
           </choose>
           <names variable="editorial-director">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names variable="guest">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names variable="host">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names variable="illustrator">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names variable="narrator">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names variable="compiler">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names delimiter=", " variable="chair organizer curator">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names delimiter=", " variable="series-creator executive-producer">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names variable="producer">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names variable="director">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <choose>
             <if match="any" type="broadcast performance">
               <names variable="script-writer">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
             </if>
           </choose>
           <names variable="performer">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <choose>
             <if match="none" type="thesis">
               <names variable="contributor">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
             </if>
           </choose>
@@ -2251,15 +2252,15 @@
             <if match="none" variable="container-title">
               <names variable="container-author">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <names variable="editor-translator">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <names delimiter=", " variable="editor translator">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <choose>
                 <if variable="collection-editor editor"/>
@@ -2267,71 +2268,71 @@
                 <else-if match="any" variable="number-of-volumes part-number part-title volume volume-title">
                   <names variable="collection-editor">
                     <label form="verb-short" suffix=" "/>
-                    <name initialize="false"/>
+                    <name initialize="true"/>
                   </names>
                 </else-if>
               </choose>
               <names variable="editorial-director">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <names variable="guest">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <names variable="host">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
             </if>
           </choose>
           <names variable="illustrator">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names variable="narrator">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <choose>
             <if match="none" variable="container-title">
               <names variable="compiler">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <names delimiter=", " variable="chair organizer curator">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <names delimiter=", " variable="series-creator executive-producer">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <names variable="producer">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <names variable="director">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <choose>
                 <if match="any" type="broadcast performance">
                   <names variable="script-writer">
                     <label form="verb" suffix=" "/>
-                    <name initialize="false"/>
+                    <name initialize="true"/>
                   </names>
                 </if>
               </choose>
               <names variable="performer">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <choose>
                 <if match="none" type="song thesis">
                   <names variable="contributor">
                     <label form="verb" suffix=" "/>
-                    <name initialize="false"/>
+                    <name initialize="true"/>
                   </names>
                 </if>
               </choose>
@@ -2342,7 +2343,7 @@
               <!-- Song contributors attached to album (cf. CMOS18 14.163) -->
               <names variable="contributor">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
             </if>
           </choose>
@@ -2356,42 +2357,42 @@
         <if type="periodical">
           <names variable="editor">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
         </if>
       </choose>
       <names variable="translator">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="narrator">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="compiler">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names delimiter=", " variable="chair organizer curator">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names delimiter=", " variable="series-creator executive-producer">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="producer">
         <!-- not abbreviated (cf. CMOS18 14.168) -->
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="director">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="performer">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
     </group>
   </macro>
@@ -2842,7 +2843,7 @@
   <macro name="source-monographic-author">
     <!-- based on `author-note` -->
     <names variable="container-author">
-      <name initialize="false"/>
+      <name initialize="true"/>
       <label form="short" prefix=", "/>
       <!-- for OGS or variants with bracketed roles: -->
       <!-- <label form="short" prefix=" (" suffix=")"/> -->
@@ -2939,11 +2940,11 @@
     <group delimiter=", ">
       <names variable="editor-translator">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names delimiter=", " variable="editor translator">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <choose>
         <if variable="collection-editor editor"/>
@@ -2951,72 +2952,72 @@
         <else-if match="any" variable="number-of-volumes part-number part-title volume volume-title">
           <names variable="collection-editor">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
         </else-if>
       </choose>
       <names variable="guest">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="host">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names delimiter=", " variable="chair organizer">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="illustrator">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="narrator">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="compiler">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="curator">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names delimiter=", " variable="series-creator executive-producer">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="producer">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="editorial-director">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <choose>
         <if match="any" type="broadcast performance">
           <names variable="script-writer">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
         </if>
       </choose>
       <names variable="director">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="performer">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <choose>
         <if match="none" type="song thesis">
           <!-- Song contributors attached to album (cf. CMOS18 14.163) -->
           <names variable="contributor">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
         </if>
       </choose>
@@ -3476,7 +3477,7 @@
           </choose>
           <group delimiter=", ">
             <names variable="original-author">
-              <name initialize="false"/>
+              <name initialize="true"/>
             </names>
             <text font-style="italic" text-case="title" variable="original-title"/>
           </group>
@@ -3505,7 +3506,7 @@
                 <text form="short" term="no-place"/>
               </else>
             </choose>
-            <!-- <text variable="publisher"/> -->
+            <text variable="publisher"/>
           </group>
         </if>
         <else>
@@ -3525,7 +3526,7 @@
       <if match="any" type="book chapter classic entry entry-dictionary entry-encyclopedia interview musical_score pamphlet paper-conference report">
         <group delimiter=": ">
           <text variable="original-publisher-place"/>
-          <!-- <text variable="original-publisher"/> -->
+          <text variable="original-publisher"/>
         </group>
       </if>
       <else>
