--- templates/new-harts-rules-template.csl
+++ development/new-harts-rules-author-date.csl
@@ -3,18 +3,19 @@
   type="application/relax-ng-compact-syntax" schematypens="http://relaxng.org/ns/compatibility/annotations/1.0"?>
 <?xml-model href="https://raw.githubusercontent.com/citation-style-language/schema/refs/heads/master/schemas/styles/csl-repository.rnc"
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-<style and="text" class="note" default-locale="en-GB" demote-non-dropping-particle="display-and-sort" et-al-min="4" et-al-use-first="1" page-range-format="minimal" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
+<style and="text" class="in-text" default-locale="en-GB" demote-non-dropping-particle="display-and-sort" et-al-min="4" et-al-use-first="1" page-range-format="minimal" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>New Hart's Rules: The Oxford Style Guide 2nd edition (template)</title>
-    <title-short>NHR (template)</title-short>
-    <id>http://www.zotero.org/styles/new-harts-rules-notes</id>
-    <link href="http://www.zotero.org/styles/new-harts-rules-notes" rel="self"/>
+    <title>New Hart's Rules: The Oxford Style Guide 2nd edition (author-date, bracketed date with comma)</title>
+    <title-short>NHR (author-date/Harvard [17.3], date with comma)</title-short>
+    <id>http://www.zotero.org/styles/new-harts-rules-author-date</id>
+    <link href="http://www.zotero.org/styles/new-harts-rules-author-date" rel="self"/>
+    <link href="http://www.zotero.org/styles/new-harts-rules-notes" rel="template"/>
     <link href="https://en.wikipedia.org/wiki/Hart's_Rules" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
       <uri>https://orcid.org/0000-0003-0464-5036</uri>
     </author>
-    <category citation-format="note"/>
+    <category citation-format="author-date"/>
     <category field="anthropology"/>
     <category field="communications"/>
     <category field="generic-base"/>
@@ -30,7 +31,7 @@
     <category field="social_science"/>
     <category field="sociology"/>
     <category field="theology"/>
-    <summary>Oxford source citations, template</summary>
+    <summary>Oxford source citations, author-date system</summary>
     <updated>2025-11-05T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -2283,29 +2284,25 @@
         <if type="article-newspaper">
           <group delimiter=" ">
             <text macro="source-serial-title"/>
-            <text macro="source-serial-identifier"/>
+            <text macro="source-serial-identifier-author-date"/>
           </group>
         </if>
         <else-if match="any" variable="collection-title issue supplement-number volume">
           <text macro="source-serial-title"/>
-          <text macro="source-serial-identifier"/>
+          <text macro="source-serial-identifier-author-date"/>
         </else-if>
         <else-if type="interview" variable="publisher">
           <!-- publisher possible with `interview` (NHR 18.7.3) -->
           <text macro="source-serial-title"/>
           <group delimiter=" ">
             <text variable="publisher"/>
-            <text macro="source-serial-identifier"/>
-            <!-- for author-date: -->
-            <!-- <text macro="source-serial-identifier-author-date"/> -->
+            <text macro="source-serial-identifier-author-date"/>
           </group>
         </else-if>
         <else>
           <group delimiter=" ">
             <text macro="source-serial-title"/>
-            <text macro="source-serial-identifier"/>
-            <!-- for author-date: -->
-            <!-- <text macro="source-serial-identifier-author-date"/> -->
+            <text macro="source-serial-identifier-author-date"/>
           </group>
         </else>
       </choose>
@@ -3260,9 +3257,7 @@
   </macro>
   <!-- 4.6. Date -->
   <macro name="source-date">
-    <text macro="source-date-issued-or-status"/>
-    <!-- for author-date: -->
-    <!-- <text macro="source-date-issued-day-month"/> -->
+    <text macro="source-date-issued-day-month"/>
   </macro>
   <!-- Date elements -->
   <macro name="source-date-issued-full">
@@ -3394,9 +3389,7 @@
     </choose>
   </macro>
   <macro name="source-date-original">
-    <text macro="source-date-original-full"/>
-    <!-- for author-date: -->
-    <!-- <text macro="source-date-original-day-month"/> -->
+    <text macro="source-date-original-day-month"/>
   </macro>
   <macro name="source-date-original-full">
     <!-- Give full date for more ephemeral types; reflected in `date-year-sort` -->
@@ -3919,9 +3912,9 @@
       </choose>
     </group>
   </macro>
-  <citation disambiguate-add-givenname="true" disambiguate-add-names="true">
-    <layout delimiter="; " suffix=".">
-      <text macro="citation-notes-full-subsequent-author-title"/>
+  <citation after-collapse-delimiter="; " collapse="year" disambiguate-add-givenname="true" disambiguate-add-names="true" disambiguate-add-year-suffix="true">
+    <layout delimiter="; " prefix="(" suffix=")">
+      <text macro="citation-author-date"/>
     </layout>
   </citation>
   <!-- Bibliography -->
@@ -4053,17 +4046,16 @@
   <bibliography hanging-indent="true">
     <sort>
       <key macro="author-bib"/>
-      <key macro="title-and-descriptions"/>
-      <key macro="source"/>
+      <key macro="date-sort-group"/>
+      <key macro="date-sort-year"/>
+      <key variable="status"/>
+      <key macro="title"/>
       <key variable="volume"/>
       <key variable="part-number"/>
-      <key variable="event-date"/>
-      <key variable="original-date"/>
-      <key macro="source-date-issued-or-status"/>
       <key macro="source-archive"/>
     </sort>
     <layout suffix=".">
-      <text macro="bibliography-filtered-notes"/>
+      <text macro="bibliography-filtered-author-date"/>
     </layout>
   </bibliography>
 </style>
