--- templates/new-harts-rules-template.csl
+++ development/new-harts-rules-notes-initials.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style and="text" class="note" default-locale="en-GB" demote-non-dropping-particle="display-and-sort" et-al-min="4" et-al-use-first="1" initialize-with=". " page-range-format="minimal" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>New Hart's Rules: The Oxford Style Guide 2nd edition (template)</title>
-    <title-short>NHR (template)</title-short>
-    <id>http://www.zotero.org/styles/new-harts-rules-notes</id>
-    <link href="http://www.zotero.org/styles/new-harts-rules-notes" rel="self"/>
+    <title>New Hart's Rules: The Oxford Style Guide 2nd edition (notes, author initials)</title>
+    <title-short>NHR (notes [17.2], author initials [18.2.2])</title-short>
+    <id>http://www.zotero.org/styles/new-harts-rules-notes-initials</id>
+    <link href="http://www.zotero.org/styles/new-harts-rules-notes-initials" rel="self"/>
+    <link href="http://www.zotero.org/styles/new-harts-rules-notes" rel="template"/>
     <link href="https://en.wikipedia.org/wiki/Hart's_Rules" rel="documentation"/>
     <link href="http://zotero.org/groups/2205533/collections/499SCKUW" rel="documentation"/>
     <author>
@@ -31,7 +32,7 @@
     <category field="social_science"/>
     <category field="sociology"/>
     <category field="theology"/>
-    <summary>Oxford source citations, template</summary>
+    <summary>Oxford source citations, notes system, author forenames reduced to initials (NHR 18.2.2)</summary>
     <updated>2026-02-05T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -382,9 +383,9 @@
   <macro name="author-bib">
     <group delimiter=", ">
       <names variable="composer">
-        <name delimiter-precedes-et-al="after-inverted-name" delimiter-precedes-last="always" initialize="false" name-as-sort-order="first"/>
+        <name delimiter-precedes-et-al="after-inverted-name" delimiter-precedes-last="always" initialize="true" name-as-sort-order="first"/>
         <!-- for OGS: -->
-        <!-- <name delimiter-precedes-et-al="after-inverted-name" delimiter-precedes-last="always" font-variant="small-caps" initialize="false" name-as-sort-order="first"/>
+        <!-- <name delimiter-precedes-et-al="after-inverted-name" delimiter-precedes-last="always" font-variant="small-caps" initialize="true" name-as-sort-order="first"/>
         <et-al font-variant="normal"/> -->
         <label form="short" prefix=", "/>
         <!-- for OGS or variants with bracketed roles: -->
@@ -529,7 +530,7 @@
   <macro name="author-note">
     <group delimiter=", ">
       <names variable="composer">
-        <name initialize="false"/>
+        <name initialize="true"/>
         <label form="short" prefix=", "/>
         <!-- for OGS or variants with bracketed roles: -->
         <!-- <label form="short" prefix=" (" suffix=")"/> -->
@@ -1742,7 +1743,7 @@
           <!-- `genre` printed with `number` -->
           <names variable="interviewer">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
         </if>
         <else-if variable="genre">
@@ -1751,7 +1752,7 @@
             <group delimiter=" ">
               <text form="verb" term="container-author"/>
               <names variable="interviewer">
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
             </group>
           </group>
@@ -1759,7 +1760,7 @@
         <else-if variable="interviewer">
           <names variable="interviewer">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
         </else-if>
         <else>
@@ -1776,7 +1777,7 @@
           <!-- `genre` printed with `number` -->
           <names variable="interviewer">
             <label form="verb" suffix=" " text-case="capitalize-first"/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
         </if>
         <else-if variable="genre">
@@ -1785,7 +1786,7 @@
             <group delimiter=" ">
               <text form="verb" term="container-author"/>
               <names variable="interviewer">
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
             </group>
           </group>
@@ -1793,7 +1794,7 @@
         <else-if variable="interviewer">
           <names variable="interviewer">
             <label form="verb" suffix=" " text-case="capitalize-first"/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
         </else-if>
         <else>
@@ -1840,7 +1841,7 @@
               <!-- `genre` appears with `number` -->
               <names variable="recipient">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
             </if>
             <else>
@@ -1848,7 +1849,7 @@
                 <text macro="description-format"/>
                 <names variable="recipient">
                   <label form="verb" suffix=" "/>
-                  <name initialize="false"/>
+                  <name initialize="true"/>
                 </names>
               </group>
             </else>
@@ -1871,7 +1872,7 @@
               <!-- `genre` appears with `number` -->
               <names variable="recipient">
                 <label form="verb" suffix=" " text-case="capitalize-first"/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
             </if>
             <else>
@@ -1879,7 +1880,7 @@
                 <text macro="description-format-capitalized"/>
                 <names variable="recipient">
                   <label form="verb" suffix=" "/>
-                  <name initialize="false"/>
+                  <name initialize="true"/>
                 </names>
               </group>
             </else>
@@ -1965,7 +1966,7 @@
         </choose>
         <group delimiter=", ">
           <names variable="reviewed-author">
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <text macro="description-review-title"/>
           <text variable="reviewed-genre"/>
@@ -1989,7 +1990,7 @@
         </choose>
         <group delimiter=", ">
           <names variable="reviewed-author">
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <text macro="description-review-title"/>
           <text variable="reviewed-genre"/>
@@ -2075,7 +2076,7 @@
         </if>
       </choose>
       <names variable="author">
-        <name delimiter-precedes-last="always" initialize="false"/>
+        <name delimiter-precedes-last="always" initialize="true"/>
         <substitute>
           <names variable="performer"/>
         </substitute>
@@ -2094,7 +2095,7 @@
         </if>
       </choose>
       <names variable="author">
-        <name delimiter-precedes-last="always" initialize="false"/>
+        <name delimiter-precedes-last="always" initialize="true"/>
         <substitute>
           <names variable="performer"/>
         </substitute>
@@ -2165,15 +2166,15 @@
         <if match="any" type="post webpage">
           <names variable="container-author">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names variable="editor-translator">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names delimiter=", " variable="editor translator">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <choose>
             <if variable="collection-editor editor"/>
@@ -2181,67 +2182,67 @@
             <else-if match="any" variable="number-of-volumes part-number part-title volume volume-title">
               <names variable="collection-editor">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
             </else-if>
           </choose>
           <names variable="editorial-director">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names variable="guest">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names variable="host">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names variable="illustrator">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names variable="narrator">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names variable="compiler">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names delimiter=", " variable="chair organizer curator">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names delimiter=", " variable="series-creator executive-producer">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names variable="producer">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names variable="director">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <choose>
             <if match="any" type="broadcast performance">
               <names variable="script-writer">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
             </if>
           </choose>
           <names variable="performer">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <choose>
             <if match="none" type="thesis">
               <names variable="contributor">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
             </if>
           </choose>
@@ -2252,15 +2253,15 @@
             <if match="none" variable="container-title">
               <names variable="container-author">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <names variable="editor-translator">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <names delimiter=", " variable="editor translator">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <choose>
                 <if variable="collection-editor editor"/>
@@ -2268,71 +2269,71 @@
                 <else-if match="any" variable="number-of-volumes part-number part-title volume volume-title">
                   <names variable="collection-editor">
                     <label form="verb-short" suffix=" "/>
-                    <name initialize="false"/>
+                    <name initialize="true"/>
                   </names>
                 </else-if>
               </choose>
               <names variable="editorial-director">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <names variable="guest">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <names variable="host">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
             </if>
           </choose>
           <names variable="illustrator">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <names variable="narrator">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
           <choose>
             <if match="none" variable="container-title">
               <names variable="compiler">
                 <label form="verb-short" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <names delimiter=", " variable="chair organizer curator">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <names delimiter=", " variable="series-creator executive-producer">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <names variable="producer">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <names variable="director">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <choose>
                 <if match="any" type="broadcast performance">
                   <names variable="script-writer">
                     <label form="verb" suffix=" "/>
-                    <name initialize="false"/>
+                    <name initialize="true"/>
                   </names>
                 </if>
               </choose>
               <names variable="performer">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
               <choose>
                 <if match="none" type="song thesis">
                   <names variable="contributor">
                     <label form="verb" suffix=" "/>
-                    <name initialize="false"/>
+                    <name initialize="true"/>
                   </names>
                 </if>
               </choose>
@@ -2343,7 +2344,7 @@
               <!-- Song contributors attached to album (cf. CMOS18 14.163) -->
               <names variable="contributor">
                 <label form="verb" suffix=" "/>
-                <name initialize="false"/>
+                <name initialize="true"/>
               </names>
             </if>
           </choose>
@@ -2357,42 +2358,42 @@
         <if type="periodical">
           <names variable="editor">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
         </if>
       </choose>
       <names variable="translator">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="narrator">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="compiler">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names delimiter=", " variable="chair organizer curator">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names delimiter=", " variable="series-creator executive-producer">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="producer">
         <!-- not abbreviated (cf. CMOS18 14.168) -->
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="director">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="performer">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
     </group>
   </macro>
@@ -2843,7 +2844,7 @@
   <macro name="source-monographic-author">
     <!-- based on `author-note` -->
     <names variable="container-author">
-      <name initialize="false"/>
+      <name initialize="true"/>
       <label form="short" prefix=", "/>
       <!-- for OGS or variants with bracketed roles: -->
       <!-- <label form="short" prefix=" (" suffix=")"/> -->
@@ -2940,11 +2941,11 @@
     <group delimiter=", ">
       <names variable="editor-translator">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names delimiter=", " variable="editor translator">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <choose>
         <if variable="collection-editor editor"/>
@@ -2952,72 +2953,72 @@
         <else-if match="any" variable="number-of-volumes part-number part-title volume volume-title">
           <names variable="collection-editor">
             <label form="verb-short" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
         </else-if>
       </choose>
       <names variable="guest">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="host">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names delimiter=", " variable="chair organizer">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="illustrator">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="narrator">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="compiler">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="curator">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names delimiter=", " variable="series-creator executive-producer">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="producer">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="editorial-director">
         <label form="verb-short" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <choose>
         <if match="any" type="broadcast performance">
           <names variable="script-writer">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
         </if>
       </choose>
       <names variable="director">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <names variable="performer">
         <label form="verb" suffix=" "/>
-        <name initialize="false"/>
+        <name initialize="true"/>
       </names>
       <choose>
         <if match="none" type="song thesis">
           <!-- Song contributors attached to album (cf. CMOS18 14.163) -->
           <names variable="contributor">
             <label form="verb" suffix=" "/>
-            <name initialize="false"/>
+            <name initialize="true"/>
           </names>
         </if>
       </choose>
@@ -3477,7 +3478,7 @@
           </choose>
           <group delimiter=", ">
             <names variable="original-author">
-              <name initialize="false"/>
+              <name initialize="true"/>
             </names>
             <text font-style="italic" text-case="title" variable="original-title"/>
           </group>
