--- templates/nlm-template.csl
+++ development/nlm-citation-sequence.csl
@@ -5,7 +5,7 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style class="in-text" delimiter-precedes-last="always" demote-non-dropping-particle="sort-only" initialize-with="" initialize-with-hyphen="false" name-as-sort-order="all" name-delimiter=", " names-delimiter=", " page-range-format="minimal" sort-separator=" " version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>NLM Style Guide (Vancouver): Citing Medicine 2nd edition (template)</title>
+    <title>NLM Style Guide (Vancouver): Citing Medicine 2nd edition (citation-sequence)</title>
     <title-short>National Library of Medicine, ANSI/NISO Z39.29-2005 (R2010), ICMJE Recommendations/URMs (C-S)</title-short>
     <id>http://www.zotero.org/styles/nlm-citation-sequence</id>
     <link href="http://www.zotero.org/styles/nlm-citation-sequence" rel="self"/>
