--- templates/nlm-template.csl
+++ development/nlm-name-year.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style class="in-text" demote-non-dropping-particle="sort-only" initialize-with-hyphen="false" page-range-format="minimal" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>NLM Style Guide (Vancouver): Citing Medicine 2nd edition (template)</title>
-    <title-short>National Library of Medicine, ANSI/NISO Z39.29-2005 (R2010), ICMJE Recommendations/URMs (C-S)</title-short>
-    <id>http://www.zotero.org/styles/nlm-citation-sequence</id>
-    <link href="http://www.zotero.org/styles/nlm-citation-sequence" rel="self"/>
+    <title>NLM Style Guide (Vancouver): Citing Medicine 2nd edition (name-year)</title>
+    <title-short>National Library of Medicine, ANSI/NISO Z39.29-2005 (R2010), ICMJE Recommendations/URMs (author-date)</title-short>
+    <id>http://www.zotero.org/styles/nlm-name-year</id>
+    <link href="http://www.zotero.org/styles/nlm-name-year" rel="self"/>
+    <link href="http://www.zotero.org/styles/nlm-citation-sequence" rel="template"/>
     <link href="https://www.nlm.nih.gov/citingmedicine" rel="documentation"/>
     <link href="https://www.nlm.nih.gov/bsd/uniform_requirements.html" rel="documentation"/>
     <link href="https://www.icmje.org/recommendations/" rel="documentation"/>
@@ -39,11 +40,11 @@
       <name>Sean Takats</name>
       <uri>https://orcid.org/0000-0002-7851-5069</uri>
     </contributor>
-    <category citation-format="numeric"/>
+    <category citation-format="author-date"/>
     <category field="generic-base"/>
     <category field="medicine"/>
     <category field="science"/>
-    <summary>Citing Medicine: The NLM Style Guide for Authors, Editors, and Publishers, 2nd edition (2015), based on ANSI/NISO Z39.29-2005 (R2010); citation-sequence system.</summary>
+    <summary>Citing Medicine: The NLM Style Guide for Authors, Editors, and Publishers, 2nd edition (2015), based on ANSI/NISO Z39.29-2005 (R2010); name-year system.</summary>
     <updated>2026-01-07T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -386,17 +387,28 @@
       </if>
     </choose>
   </macro>
-  <citation collapse="citation-number">
+  <citation collapse="year" disambiguate-add-year-suffix="true" et-al-min="3" et-al-use-first="1">
     <sort>
-      <key variable="citation-number"/>
+      <key macro="date-short"/>
+      <key macro="author-short"/>
     </sort>
-    <layout delimiter="," prefix="(" suffix=")">
-      <text variable="citation-number"/>
+    <layout delimiter="; " prefix="(" suffix=")">
+      <group delimiter=", ">
+        <group delimiter=" ">
+          <text macro="author-short"/>
+          <text macro="date-short"/>
+        </group>
+        <text macro="label-locator"/>
+      </group>
     </layout>
   </citation>
-  <bibliography et-al-min="7" et-al-use-first="6" second-field-align="flush">
+  <bibliography et-al-min="7" et-al-use-first="6" hanging-indent="true">
+    <sort>
+      <key macro="author"/>
+      <key macro="date-short"/>
+      <key variable="title"/>
+    </sort>
     <layout>
-      <text suffix="." variable="citation-number"/>
       <group delimiter=". " suffix=". ">
         <text macro="author"/>
         <text macro="title"/>
