--- templates/nlm-template.csl
+++ development/nlm-citation-sequence-brackets-no-et-al.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style class="in-text" delimiter-precedes-last="always" demote-non-dropping-particle="sort-only" initialize-with="" initialize-with-hyphen="false" name-as-sort-order="all" name-delimiter=", " names-delimiter=", " page-range-format="minimal" sort-separator=" " version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>NLM Style Guide (Vancouver): Citing Medicine 2nd edition (template)</title>
-    <title-short>National Library of Medicine, ANSI/NISO Z39.29-2005 (R2010), ICMJE Recommendations/URMs (C-S)</title-short>
-    <id>http://www.zotero.org/styles/nlm-citation-sequence</id>
-    <link href="http://www.zotero.org/styles/nlm-citation-sequence" rel="self"/>
+    <title>NLM Style Guide (Vancouver): Citing Medicine 2nd edition (citation-sequence, brackets, no "et al.")</title>
+    <title-short>National Library of Medicine, ANSI/NISO Z39.29-2005 (R2010), ICMJE Recommendations/URMs (C-S, brackets, no "et al.")</title-short>
+    <id>http://www.zotero.org/styles/nlm-citation-sequence-brackets-no-et-al</id>
+    <link href="http://www.zotero.org/styles/nlm-citation-sequence-brackets-no-et-al" rel="self"/>
+    <link href="http://www.zotero.org/styles/nlm-citation-sequence-brackets" rel="template"/>
     <link href="https://www.nlm.nih.gov/citingmedicine" rel="documentation"/>
     <link href="https://www.nlm.nih.gov/bsd/uniform_requirements.html" rel="documentation"/>
     <link href="https://www.icmje.org/recommendations/" rel="documentation"/>
@@ -43,7 +44,7 @@
     <category field="generic-base"/>
     <category field="medicine"/>
     <category field="science"/>
-    <summary>Citing Medicine: The NLM Style Guide for Authors, Editors, and Publishers, 2nd edition (2015), based on ANSI/NISO Z39.29-2005 (R2010); citation-sequence system.</summary>
+    <summary>Citing Medicine: The NLM Style Guide for Authors, Editors, and Publishers, 2nd edition (2015), based on ANSI/NISO Z39.29-2005 (R2010); citation-sequence system, with in-text indicators in brackets, no "et al."</summary>
     <updated>2026-01-07T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -482,7 +483,7 @@
     <sort>
       <key variable="citation-number"/>
     </sort>
-    <layout delimiter="," prefix="(" suffix=")">
+    <layout delimiter="," prefix="[" suffix="]">
       <text variable="citation-number"/>
     </layout>
   </citation>
@@ -514,7 +515,7 @@
       <text macro="notes"/>
     </group>
   </macro>
-  <bibliography et-al-min="7" et-al-use-first="6" second-field-align="flush">
+  <bibliography second-field-align="flush">
     <layout>
       <text suffix="." variable="citation-number"/>
       <text macro="bibliography"/>
