--- templates/norsk-henvisningsstandard-for-rettsvitenskapelige-tekster-template.csl
+++ development/norsk-henvisningsstandard-for-rettsvitenskapelige-tekster-full-firstnote.csl
@@ -2,8 +2,8 @@
 <style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded">
   <!-- Polyglot; this citation style supports both Norwegian dialects, i.e. bokmål (nb-NO) and nynorsk (nn-NO) -->
   <info>
-    <title>Norsk henvisningsstandard for rettsvitenskapelige tekster (Norsk - Bokmål | Norsk - Nynorsk) [template]</title>
-    <id>http://www.zotero.org/styles/norsk-henvisningsstandard-for-rettsvitenskapelige-tekster</id>
+    <title>Norsk henvisningsstandard for rettsvitenskapelige tekster – fullstendig førstenote (Norsk - Bokmål | Norsk - Nynorsk)</title>
+    <id>http://www.zotero.org/styles/norsk-henvisningsstandard-for-rettsvitenskapelige-tekster-full-firstnote</id>
     <link href="http://www.zotero.org/styles/norsk-henvisningsstandard-for-rettsvitenskapelige-tekster" rel="self"/>
     <link href="http://www.zotero.org/styles/universitetet-i-oslo-rettsvitenskap" rel="template"/>
     <link href="https://www.uib.no/sites/w3.uib.no/files/attachments/veiledning_for_henvisning_i_juridiske_tekster.pdf" rel="documentation"/>
@@ -18,7 +18,7 @@
     </contributor>
     <category citation-format="note"/>
     <category field="law"/>
-    <summary>Norwegian legal citation style based on "Veiledning for henvisninger i juridiske tekster" (February 2019).</summary>
+    <summary>Norwegian legal citation style based on "Veiledning for henvisninger i juridiske tekster" (February 2019). Variant with full note on first citation, subsequent notes in short form (bibliography optional).</summary>
     <updated>2026-03-10T22:12:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -517,10 +517,61 @@
     <layout delimiter="; ">
       <group delimiter=" ">
         <choose>
-          <if type="book thesis chapter entry-encyclopedia paper-conference article-journal article-newspaper article-magazine post-weblog" match="any">
-            <text macro="author-short"/>
-            <text macro="issued"/>
+          <if type="book">
+            <choose>
+              <if position="first">
+                <text macro="default-bib"/>
+              </if>
+              <else>
+                <text macro="author-short"/>
+                <text macro="issued"/>
+              </else>
+            </choose>
           </if>
+          <else-if type="thesis">
+            <choose>
+              <if position="first">
+                <text macro="thesis-bib"/>
+              </if>
+              <else>
+                <text macro="author-short"/>
+                <text macro="issued"/>
+              </else>
+            </choose>
+          </else-if>
+          <else-if type="chapter entry-encyclopedia paper-conference" match="any">
+            <choose>
+              <if position="first">
+                <text macro="chapter-encyclopediaentry-conferencepaper-bib"/>
+              </if>
+              <else>
+                <text macro="author-short"/>
+                <text macro="issued"/>
+              </else>
+            </choose>
+          </else-if>
+          <else-if type="article-journal">
+            <choose>
+              <if position="first">
+                <text macro="article-journal-bib"/>
+              </if>
+              <else>
+                <text macro="author-short"/>
+                <text macro="issued"/>
+              </else>
+            </choose>
+          </else-if>
+          <else-if type="article-newspaper article-magazine post-weblog" match="any">
+            <choose>
+              <if position="first">
+                <text macro="article-newspaper-magazine-weblog-bib"/>
+              </if>
+              <else>
+                <text macro="author-short"/>
+                <text macro="issued"/>
+              </else>
+            </choose>
+          </else-if>
           <else-if type="legislation regulation">
             <choose>
               <if position="first">
