--- templates/oscola-template.csl
+++ development/oscola-no-ibid.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="never" default-locale="en-GB">
   <info>
-    <title>OSCOLA 4th edition (template)</title>
+    <title>OSCOLA 4th edition (no ibid.)</title>
     <title-short>Oxford University Standard for Citation of Legal Authorities</title-short>
-    <id>http://www.zotero.org/styles/oscola</id>
-    <link href="http://www.zotero.org/styles/oscola" rel="self"/>
+    <id>http://www.zotero.org/styles/oscola-no-ibid</id>
+    <link href="http://www.zotero.org/styles/oscola-no-ibid" rel="self"/>
+    <link href="http://www.zotero.org/styles/oscola" rel="template"/>
     <link href="https://www.law.ox.ac.uk/oscola" rel="documentation"/>
     <author>
       <name>Sebastian Karcher</name>
@@ -20,7 +21,7 @@
     </author>
     <category citation-format="note"/>
     <category field="law"/>
-    <summary>Oxford University Standard for Citation of Legal Authorities, notes system. For a Zotero group showing data entry: https://zotero.org/groups/2205533/collections/AC7ETLN4</summary>
+    <summary>Oxford University Standard for Citation of Legal Authorities, notes system, no ibid. For a Zotero group showing data entry: https://zotero.org/groups/2205533/collections/AC7ETLN4</summary>
     <updated>2025-07-08T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -597,16 +598,7 @@
   <citation et-al-min="4" et-al-use-first="1">
     <layout suffix="." delimiter="; ">
       <choose>
-        <if position="ibid-with-locator">
-          <group delimiter=" ">
-            <text term="ibid" strip-periods="true" text-case="lowercase"/>
-            <text variable="locator"/>
-          </group>
-        </if>
-        <else-if position="ibid">
-          <text term="ibid" strip-periods="true" text-case="lowercase"/>
-        </else-if>
-        <else-if position="subsequent">
+        <if position="subsequent">
           <choose>
             <if type="legal_case bill legislation treaty" match="any">
               <!--don't use short form and above note for legal citations -->
@@ -644,7 +636,7 @@
               </group>
             </else>
           </choose>
-        </else-if>
+        </if>
         <else>
           <group delimiter=" ">
             <group delimiter=", ">
