--- templates/new-harts-rules-template.csl
+++ development/oxford-guide-to-style-notes-initials.csl
@@ -3,12 +3,13 @@
   type="application/relax-ng-compact-syntax" schematypens="http://relaxng.org/ns/compatibility/annotations/1.0"?>
 <?xml-model href="https://raw.githubusercontent.com/citation-style-language/schema/refs/heads/master/schemas/styles/csl-repository.rnc"
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-<style and="text" class="note" default-locale="en-GB" demote-non-dropping-particle="display-and-sort" et-al-min="4" et-al-use-first="1" page-range-format="minimal" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
+<style and="text" class="note" default-locale="en-GB" demote-non-dropping-particle="display-and-sort" et-al-min="4" et-al-use-first="1" initialize-with=". " page-range-format="minimal" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>New Hart's Rules: The Oxford Style Guide 2nd edition (template)</title>
-    <title-short>NHR (template)</title-short>
-    <id>http://www.zotero.org/styles/new-harts-rules-notes</id>
-    <link href="http://www.zotero.org/styles/new-harts-rules-notes" rel="self"/>
+    <title>Oxford Guide to Style (notes, author initials)</title>
+    <title-short>OGS (notes, author initials)</title-short>
+    <id>http://www.zotero.org/styles/oxford-guide-to-style-notes-initials</id>
+    <link href="http://www.zotero.org/styles/oxford-guide-to-style-notes-initials" rel="self"/>
+    <link href="http://www.zotero.org/styles/new-harts-rules-notes" rel="template"/>
     <link href="https://en.wikipedia.org/wiki/Hart's_Rules" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -30,7 +31,7 @@
     <category field="social_science"/>
     <category field="sociology"/>
     <category field="theology"/>
-    <summary>Oxford source citations, template</summary>
+    <summary>Oxford source citations, notes system, author forenames reduced to initials</summary>
     <updated>2025-11-05T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -43,26 +44,83 @@
     <terms>
       <!-- Like Chicago, Oxford omits 'by' from `verb-short` forms; it abbreviates only the most common roles -->
       <term name="advance-online-publication">published online</term>
+      <!-- OGS uses points with contractions -->
+      <term form="short" name="book">
+        <single>bk.</single>
+        <multiple>bks.</multiple>
+      </term>
+      <term form="short" name="chapter-number">
+        <single>ch.</single>
+        <multiple>chs.</multiple>
+      </term>
+      <term form="short" name="collection-editor">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
       <term form="verb-short" name="collection-editor">ed.</term>
+      <term form="short" name="column">
+        <single>col.</single>
+        <multiple>cols.</multiple>
+      </term>
       <term form="verb-short" name="compiler">comp.</term>
+      <term form="short" name="director">
+        <single>dir.</single>
+        <multiple>dirs.</multiple>
+      </term>
+      <term form="short" name="edition">
+        <single>edn.</single>
+        <multiple>edns.</multiple>
+      </term>
+      <term form="short" name="editor">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
       <term form="verb-short" name="editor">ed.</term>
       <term form="short" name="editor-translator">
         <single>ed. and trans.</single>
-        <multiple>eds and trans.</multiple>
+        <multiple>eds. and trans.</multiple>
       </term>
       <term form="short" name="editortranslator">
         <single>ed. and trans.</single>
-        <multiple>eds and trans.</multiple>
+        <multiple>eds. and trans.</multiple>
       </term>
       <term form="verb-short" name="editor-translator">ed. and trans.</term>
       <term form="verb-short" name="editortranslator">ed. and trans.</term>
+      <term form="short" name="folio">
+        <single>fo.</single>
+        <multiple>fos.</multiple>
+      </term>
       <term form="verb-short" name="illustrator">illus.</term>
       <term name="manuscript">unpublished manuscript</term>
+      <term form="short" name="number">
+        <single>no.</single>
+        <multiple>nos.</multiple>
+      </term>
+      <term form="short" name="number-of-volumes">
+        <single>vol.</single>
+        <multiple>vols.</multiple>
+      </term>
       <term form="short" name="paper-conference">paper</term>
+      <term form="short" name="paragraph">
+        <single>para.</single>
+        <multiple>paras.</multiple>
+      </term>
+      <term form="short" name="part">
+        <single>pt.</single>
+        <multiple>pts.</multiple>
+      </term>
+      <term form="short" name="section">
+        <single>sect.</single>
+        <multiple>sects.</multiple>
+      </term>
       <term form="verb-short" name="translator">trans.</term>
+      <term form="short" name="volume">
+        <single>vol.</single>
+        <multiple>vols.</multiple>
+      </term>
     </terms>
   </locale>
-  <!-- This file's main source is the New Oxford Style Manual (2016), with reference also to the Oxford Style Manual (2003) <https://archive.org/details/oxfordstylemanua0000unse>, which provides more detailed guidance on many points. -->
+  <!-- This file follows the Oxford Style Manual (2003) <https://archive.org/details/oxfordstylemanua0000unse>. -->
   <!-- Contents:
     
        This file interprets Oxford style using APA's four basic reference elements:
@@ -353,33 +411,22 @@
     </group>
   </macro>
   <macro name="label-volume">
-    <group delimiter=" ">
-      <choose>
-        <if is-numeric="volume">
-          <label form="short" variable="volume"/>
-        </if>
-      </choose>
-      <text variable="volume"/>
-    </group>
-    <!-- in OGS and other styles, label a volume by rendering it in Roman numerals (NHR 18.2.7): -->
-    <!-- <choose>
+    <choose>
       <if is-numeric="volume">
         <number form="roman" variable="volume"/>
       </if>
       <else>
         <text variable="volume"/>
       </else>
-    </choose> -->
+    </choose>
   </macro>
   <!-- 1. Author -->
   <macro name="author-bib">
     <group delimiter=", ">
       <names variable="composer">
-        <name delimiter-precedes-et-al="after-inverted-name" delimiter-precedes-last="always" name-as-sort-order="first"/>
-        <!-- for OGS: -->
-        <!-- <name delimiter-precedes-et-al="after-inverted-name" delimiter-precedes-last="always" font-variant="small-caps" name-as-sort-order="first"/> -->
-        <!-- <et-al font-variant="normal"/> -->
-        <label form="short" prefix=", "/>
+        <name delimiter-precedes-et-al="after-inverted-name" delimiter-precedes-last="always" font-variant="small-caps" name-as-sort-order="first"/>
+        <et-al font-variant="normal"/>
+        <label form="short" prefix=" (" suffix=")"/>
         <substitute>
           <names variable="author"/>
           <names variable="host"/>
@@ -1223,20 +1270,12 @@
               <choose>
                 <if match="any" variable="collection-editor container-author editor editorial-director">
                   <!-- monographic usage -->
-                  <text macro="source-monographic-locator"/>
-                  <!-- for styles using Roman volume numbers: -->
-                  <!-- <text macro="source-monographic-locator-roman-volume"/> -->
-                  <!-- for OGS: -->
-                  <!-- <text macro="source-monographic-locator-range"/> -->
+                  <text macro="source-monographic-locator-range"/>
                 </if>
               </choose>
             </else-if>
             <else>
-              <text macro="source-monographic-locator"/>
-              <!-- for styles using Roman volume numbers: -->
-              <!-- <text macro="source-monographic-locator-roman-volume"/> -->
-              <!-- for OGS: -->
-              <!-- <text macro="source-monographic-locator-range"/> -->
+              <text macro="source-monographic-locator-range"/>
             </else>
           </choose>
         </group>
@@ -1506,10 +1545,6 @@
             <text macro="label-volume"/>
           </if>
           <else-if locator="column page" match="any"/>
-          <else-if variable="locator">
-            <!-- remove variable="locator" condition for OGS, which prints ranges -->
-            <text macro="label-volume"/>
-          </else-if>
           <else-if variable="page"/>
           <else>
             <text macro="label-volume"/>
@@ -2458,9 +2493,7 @@
       <text macro="source-serial-identifier-volume"/>
       <!-- periodical edition always capitalized (cf. CMOS18 14.89) -->
       <text macro="label-edition-capitalized"/>
-      <text macro="source-serial-locator"/>
-      <!-- for OGS: -->
-      <!-- <text macro="source-serial-locator-range"/> -->
+      <text macro="source-serial-locator-range"/>
     </group>
   </macro>
   <macro name="source-serial-identifier-author-date">
@@ -2625,8 +2658,6 @@
           </group>
         </if>
       </choose>
-      <!-- remove `source-series` for OGS: -->
-      <text macro="source-series"/>
       <text macro="source-event"/>
     </group>
   </macro>
@@ -3067,8 +3098,7 @@
           <choose>
             <if match="any" variable="collection-editor container-author editor editorial-director">
               <!-- monographic usage -->
-              <!-- add for OGS: -->
-              <!-- <text macro="source-series"/> -->
+              <text macro="source-series"/>
               <text macro="source-publication-and-date"/>
             </if>
           </choose>
@@ -3076,8 +3106,7 @@
         <!-- `patent` date in identification (cf. CMOS18 14.158) -->
         <else-if type="patent"/>
         <else>
-          <!-- add for OGS: -->
-          <!-- <text macro="source-series"/> -->
+          <text macro="source-series"/>
           <text macro="source-publication-and-date"/>
         </else>
       </choose>
@@ -3621,9 +3650,7 @@
         <choose>
           <if match="any" variable="collection-editor compiler editor editorial-director">
             <!-- monographic usage -->
-            <text macro="source-locator-subsequent-monographic"/>
-            <!-- for OGS and other styles using Roman volume numbers: -->
-            <!-- <text macro="source-locator-subsequent-monographic-roman-volume"/> -->
+            <text macro="source-locator-subsequent-monographic-roman-volume"/>
           </if>
           <else>
             <!-- serial usage -->
@@ -3632,9 +3659,7 @@
         </choose>
       </else-if>
       <else>
-        <text macro="source-locator-subsequent-monographic"/>
-        <!-- for OGS and other styles using Roman volume numbers: -->
-        <!-- <text macro="source-locator-subsequent-monographic-roman-volume"/> -->
+        <text macro="source-locator-subsequent-monographic-roman-volume"/>
       </else>
     </choose>
   </macro>
@@ -3680,8 +3705,7 @@
           <!-- Do not provide `volume` or `part-number` for defined section -->
           <text macro="label-locator"/>
         </if>
-        <!-- for OGS: -->
-        <!-- <else-if locator="column page" match="none">
+        <else-if locator="column page" match="none">
           <text macro="label-volume"/>
           <text macro="label-part-number"/>
           <text macro="label-locator"/>
@@ -3691,7 +3715,7 @@
             <text macro="label-volume"/>
             <text macro="label-locator"/>
           </group>
-        </else-if> -->
+        </else-if>
         <else-if variable="part-number">
           <text macro="label-volume"/>
           <text macro="label-part-number"/>
@@ -3727,9 +3751,7 @@
       <!-- usually `archive`, `archive-place` but sometimes also `archive-place`, `archive` (NHR 18.6.5); the latter is more common elsewhere (OGS 15.11.1 and MHRA4 7.7) -->
       <text variable="archive_collection"/>
       <text variable="archive_location"/>
-      <text macro="source-archive-locator"/>
-      <!-- for OGS: -->
-      <!-- <text macro="source-archive-locator-range"/> -->
+      <text macro="source-archive-locator-range"/>
       <choose>
         <!-- database identifier, if not included elsewhere (cf. CMOS18 14.113) -->
         <if match="any" variable="archive_collection archive_location archive-place"/>
