--- templates/new-harts-rules-template.csl
+++ development/oxford-guide-to-style-notes.csl
@@ -5,10 +5,11 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style and="text" class="note" default-locale="en-GB" demote-non-dropping-particle="display-and-sort" et-al-min="4" et-al-use-first="1" page-range-format="minimal" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>New Hart's Rules: The Oxford Style Guide 2nd edition (template)</title>
-    <title-short>NHR (template)</title-short>
-    <id>http://www.zotero.org/styles/new-harts-rules-notes</id>
-    <link href="http://www.zotero.org/styles/new-harts-rules-notes" rel="self"/>
+    <title>Oxford Guide to Style (notes)</title>
+    <title-short>OGS (notes)</title-short>
+    <id>http://www.zotero.org/styles/oxford-guide-to-style-notes</id>
+    <link href="http://www.zotero.org/styles/oxford-guide-to-style-notes" rel="self"/>
+    <link href="http://www.zotero.org/styles/new-harts-rules-notes" rel="template"/>
     <link href="https://en.wikipedia.org/wiki/Hart's_Rules" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
@@ -30,7 +31,7 @@
     <category field="social_science"/>
     <category field="sociology"/>
     <category field="theology"/>
-    <summary>Oxford source citations, template</summary>
+    <summary>Oxford source citations, notes system</summary>
     <updated>2025-07-15T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -363,11 +364,9 @@
   <macro name="author-bib">
     <group delimiter=", ">
       <names variable="composer">
-        <name delimiter-precedes-et-al="after-inverted-name" delimiter-precedes-last="always" name-as-sort-order="first"/>
-        <!-- for OGS: -->
-        <!-- <name delimiter-precedes-et-al="after-inverted-name" delimiter-precedes-last="always" font-variant="small-caps" name-as-sort-order="first"/> -->
-        <!-- <et-al font-variant="normal"/> -->
-        <label form="short" prefix=", "/>
+        <name delimiter-precedes-et-al="after-inverted-name" delimiter-precedes-last="always" font-variant="small-caps" name-as-sort-order="first"/>
+        <et-al font-variant="normal"/>
+        <label form="short" prefix=" (" suffix=")"/>
         <substitute>
           <names variable="author"/>
           <names variable="host"/>
@@ -1211,16 +1210,12 @@
               <choose>
                 <if match="any" variable="collection-editor container-author editor editorial-director">
                   <!-- monographic usage -->
-                  <text macro="source-monographic-locator"/>
-                  <!-- for OGS: -->
-                  <!-- <text macro="source-monographic-locator-range"/> -->
+                  <text macro="source-monographic-locator-range"/>
                 </if>
               </choose>
             </else-if>
             <else>
-              <text macro="source-monographic-locator"/>
-              <!-- for OGS: -->
-              <!-- <text macro="source-monographic-locator-range"/> -->
+              <text macro="source-monographic-locator-range"/>
             </else>
           </choose>
         </group>
@@ -2337,9 +2332,7 @@
       <text macro="source-serial-identifier-volume"/>
       <!-- periodical edition always capitalized (cf. CMOS18 14.89) -->
       <text macro="label-edition-capitalized"/>
-      <text macro="source-serial-locator"/>
-      <!-- for OGS: -->
-      <!-- <text macro="source-serial-locator-range"/> -->
+      <text macro="source-serial-locator-range"/>
     </group>
   </macro>
   <macro name="source-serial-identifier-volume">
@@ -2459,8 +2452,6 @@
           </group>
         </if>
       </choose>
-      <!-- remove `source-series` for OGS: -->
-      <text macro="source-series"/>
       <text macro="source-event"/>
     </group>
   </macro>
@@ -2829,8 +2820,7 @@
           <choose>
             <if match="any" variable="collection-editor container-author editor editorial-director">
               <!-- monographic usage -->
-              <!-- add for OGS: -->
-              <!-- <text macro="source-series"/> -->
+              <text macro="source-series"/>
               <text macro="source-publication-and-date"/>
             </if>
           </choose>
@@ -2838,8 +2828,7 @@
         <!-- `patent` date in identification (cf. CMOS18 14.158) -->
         <else-if type="patent"/>
         <else>
-          <!-- add for OGS: -->
-          <!-- <text macro="source-series"/> -->
+          <text macro="source-series"/>
           <text macro="source-publication-and-date"/>
         </else>
       </choose>
@@ -3434,9 +3423,7 @@
       <!-- usually `archive`, `archive-place` but sometimes also `archive-place`, `archive` (NHR 18.6.5); the latter is more common elsewhere (OGS 15.11.1 and MHRA4 7.7) -->
       <text variable="archive_collection"/>
       <text variable="archive_location"/>
-      <text macro="source-archive-locator"/>
-      <!-- for OGS: -->
-      <!-- <text macro="source-archive-locator-range"/> -->
+      <text macro="source-archive-locator-range"/>
       <choose>
         <!-- database identifier, if not included elsewhere (cf. CMOS18 14.113) -->
         <if match="any" variable="archive_collection archive_location archive-place"/>
