--- templates/chicago-template.csl
+++ development/taylor-and-francis-chicago-author-date.csl
@@ -3,40 +3,20 @@
   type="application/relax-ng-compact-syntax" schematypens="http://relaxng.org/ns/compatibility/annotations/1.0"?>
 <?xml-model href="https://raw.githubusercontent.com/citation-style-language/schema/refs/heads/master/schemas/styles/csl.rnc"
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-<style and="text" class="note" demote-non-dropping-particle="display-and-sort" page-range-format="chicago" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
+<style and="text" class="in-text" demote-non-dropping-particle="display-and-sort" page-range-format="expanded" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Chicago Manual of Style 18th edition (template)</title>
-    <title-short>CMOS</title-short>
-    <id>http://www.zotero.org/styles/chicago-notes-bibliography</id>
-    <link href="http://www.zotero.org/styles/chicago-notes-bibliography" rel="self"/>
-    <link href="http://www.chicagomanualofstyle.org/tools_citationguide.html" rel="documentation"/>
+    <title>Taylor &amp; Francis Journals Standard Reference Style Guide: Chicago author-date version 2.0</title>
+    <title-short>TF-Standard Chicago AD</title-short>
+    <id>http://www.zotero.org/styles/taylor-and-francis-chicago-author-date</id>
+    <link href="http://www.zotero.org/styles/taylor-and-francis-chicago-author-date" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-author-date-17th-edition" rel="template"/>
+    <link href="https://files.taylorandfrancis.com/tf_ChicagoAD.pdf" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
       <uri>https://orcid.org/0000-0003-0464-5036</uri>
     </author>
-    <category citation-format="note"/>
-    <category field="generic-base"/>
-    <category field="history"/>
-    <category field="humanities"/>
-    <category field="law"/>
-    <category field="literature"/>
-    <category field="philosophy"/>
-    <category field="theology"/>
-    <!-- Categories for author-date:
     <category citation-format="author-date"/>
-    <category field="anthropology"/>
-    <category field="communications"/>
-    <category field="generic-base"/>
-    <category field="geography"/>
-    <category field="history"/>
-    <category field="humanities"/>
-    <category field="linguistics"/>
-    <category field="political_science"/>
-    <category field="science"/>
-    <category field="social_science"/>
-    <category field="sociology"/>
-    -->
-    <summary>Chicago-style source citations (with Bluebook for legal citations), template</summary>
+    <summary>Taylor &amp; Francis Journals adaptation of Chicago-style source citations (with Bluebook for legal citations), author-date system, 17th edition: with expanded page ranges, no substitution of repeated authors with em dashes</summary>
     <updated>2025-05-19T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -3323,10 +3303,6 @@
           <if match="any" variable="collection-editor part-number part-title volume-title">
             <text macro="label-volume"/>
           </if>
-          <else-if type="chapter">
-            <!-- remove condition in styles that provide chapter page numbers -->
-            <text macro="label-volume"/>
-          </else-if>
         </choose>
       </else-if>
       <else-if variable="volume">
@@ -3594,14 +3570,14 @@
     <choose>
       <if match="any" variable="issue supplement-number volume">
         <group delimiter=": ">
-          <text macro="source-serial-identifier-volume-bib"/>
+          <text macro="source-serial-identifier-volume-author-date"/>
           <text macro="source-serial-locator"/>
         </group>
       </if>
       <else>
         <group delimiter=", ">
           <group delimiter=". ">
-            <text macro="source-serial-identifier-volume-bib"/>
+            <text macro="source-serial-identifier-volume-author-date"/>
             <!-- periodical edition always capitalized (CMOS 14.89) -->
             <text macro="label-edition-capitalized"/>
           </group>
@@ -3650,9 +3626,7 @@
                 <!-- nothing to substitute if there is no issue/supplement/volume -->
                 <else-if match="none" variable="issue volume"/>
                 <else>
-                  <text macro="source-date-bib"/>
-                  <!-- for author-date 17th edition: -->
-                  <!-- <text macro="source-date-issued-month-day"/> -->
+                  <text macro="source-date-issued-month-day"/>
                 </else>
               </choose>
             </if>
@@ -3685,9 +3659,7 @@
                       <text macro="label-supplement-number"/>
                     </if>
                     <else-if match="any" variable="author chair collection-editor compiler composer curator director editor editor-translator editorial-director executive-producer guest host illustrator organizer producer series-creator translator">
-                      <text macro="source-date-bib"/>
-                      <!-- for author-date 17th edition: -->
-                      <!-- <text macro="source-date-issued-month-day"/> -->
+                      <text macro="source-date-issued-month-day"/>
                     </else-if>
                   </choose>
                 </group>
@@ -3700,9 +3672,7 @@
                   </group>
                   <choose>
                     <if match="any" variable="author chair collection-editor compiler composer curator director editor editor-translator editorial-director executive-producer guest host illustrator organizer producer series-creator translator">
-                      <text macro="source-date-bib" prefix="(" suffix=")"/>
-                      <!-- for author-date 17th edition: -->
-                      <!-- <text macro="source-date-issued-month-day" prefix="(" suffix=")"/> -->
+                      <text macro="source-date-issued-month-day" prefix="(" suffix=")"/>
                     </if>
                   </choose>
                 </group>
@@ -3735,9 +3705,7 @@
           <choose>
             <if match="none" variable="author chair collection-editor compiler composer curator director editor editor-translator editorial-director executive-producer guest host illustrator organizer producer series-creator translator">
               <!-- date appears first if the serial title has been substituted for a missing author (CMOS 14.102) -->
-              <text macro="source-date-bib"/>
-              <!-- for author-date 17th edition: -->
-              <!-- <text macro="source-date-issued-month-day"/> -->
+              <text macro="source-date-issued-month-day"/>
             </if>
           </choose>
           <!-- `collection-title` is for any serial with multiple series (e.g. 'second series') -->
@@ -3791,9 +3759,7 @@
             <else-if match="any" variable="reviewed-genre reviewed-title title"/>
             <else-if match="none" variable="genre section"/>
             <else>
-              <text macro="source-date-note"/>
-              <!-- for author-date 17th edition: -->
-              <!-- <text macro="source-date-issued-month-day"/> -->
+              <text macro="source-date-issued-month-day"/>
             </else>
           </choose>
           <!-- `collection-title` is for any serial with multiple series (e.g. 'second series') -->
@@ -3828,19 +3794,13 @@
             <choose>
               <if match="any" variable="author chair collection-editor compiler composer curator director editor editor-translator editorial-director executive-producer guest host illustrator organizer producer series-creator translator">
                 <!-- date appears first if the serial title has been substituted for a missing author (CMOS 14.89) -->
-                <text macro="source-date-note" prefix="(" suffix=")"/>
-                <!-- for author-date 17th edition: -->
-                <!-- <text macro="source-date-issued-month-day" prefix="(" suffix=")"/> -->
+                <text macro="source-date-issued-month-day" prefix="(" suffix=")"/>
               </if>
               <else-if match="any" variable="reviewed-genre reviewed-title title">
-                <text macro="source-date-note" prefix="(" suffix=")"/>
-                <!-- for author-date 17th edition: -->
                 <!-- <text macro="source-date-issued-month-day" prefix="(" suffix=")"/> -->
               </else-if>
               <else-if match="none" variable="genre section">
-                <text macro="source-date-note" prefix="(" suffix=")"/>
-                <!-- for author-date 17th edition: -->
-                <!-- <text macro="source-date-issued-month-day" prefix="(" suffix=")"/> -->
+                <text macro="source-date-issued-month-day" prefix="(" suffix=")"/>
               </else-if>
             </choose>
           </group>
@@ -3975,11 +3935,9 @@
           </if>
         </choose>
       </else-if>
-      <!-- if printing chapter page numbers:
       <else-if variable="chapter-number page">
         <text term="in" text-case="capitalize-first"/>
       </else-if>
-      -->
       <else-if variable="chapter-number">
         <group delimiter=" ">
           <text macro="label-chapter-number-capitalized"/>
@@ -4011,11 +3969,9 @@
           </if>
         </choose>
       </else-if>
-      <!-- if printing chapter page numbers:
       <else-if variable="chapter-number page">
         <text term="in"/>
       </else-if>
-      -->
       <else-if variable="chapter-number">
         <group delimiter=" ">
           <text macro="label-chapter-number"/>
@@ -4396,10 +4352,6 @@
             <if is-numeric="volume" match="none">
               <text macro="label-volume"/>
             </if>
-            <else-if type="chapter">
-              <!-- remove condition in styles that print chapter page numbers -->
-              <text macro="label-volume"/>
-            </else-if>
             <else-if is-numeric="volume" variable="page">
               <choose>
                 <!-- check for variables that might come between the volume and page number -->
@@ -4436,8 +4388,6 @@
       <else-if variable="locator">
         <text macro="label-locator"/>
       </else-if>
-      <!-- Remove `type="chapter"` condition in styles that provide chapter page numbers -->
-      <else-if type="chapter"/>
       <else-if is-numeric="volume" variable="page">
         <!-- collapse the volume and page number if adjacent -->
         <group delimiter=":">
@@ -5152,7 +5102,7 @@
         </if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text text-case="capitalize-first" variable="publisher-place"/> -->
+            <text text-case="capitalize-first" variable="publisher-place"/>
             <text text-case="capitalize-first" variable="publisher"/>
           </group>
         </else-if>
@@ -5178,7 +5128,7 @@
         </if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text variable="publisher-place"/> -->
+            <text variable="publisher-place"/>
             <text variable="publisher"/>
           </group>
         </else-if>
@@ -5200,7 +5150,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text text-case="capitalize-first" variable="original-publisher-place"/> -->
+          <text text-case="capitalize-first" variable="original-publisher-place"/>
           <text text-case="capitalize-first" variable="original-publisher"/>
         </group>
       </if>
@@ -5213,7 +5163,7 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text variable="original-publisher-place"/> -->
+          <text variable="original-publisher-place"/>
           <text variable="original-publisher"/>
         </group>
       </if>
@@ -5234,8 +5184,7 @@
       </else-if>
       <else>
         <text macro="source-date-issued-or-status"/>
-        <!-- for author-date 17th edition: -->
-        <!-- <text macro="source-date-issued-full"/> -->
+        <text macro="source-date-issued-full"/>
         <!-- for author-date 18th edition: -->
         <!-- <text macro="source-date-issued-month-day"/> -->
       </else>
@@ -5301,14 +5250,10 @@
         <text macro="date-issued-full"/>
       </if>
       <else-if variable="issue volume">
-        <date date-parts="year" form="text" variable="issued"/>
-        <!-- 17th edition: -->
-        <!-- <date date-parts="year-month" form="text" variable="issued"/> -->
+        <date date-parts="year-month" form="text" variable="issued"/>
       </else-if>
       <else-if variable="supplement-number">
-        <date date-parts="year" form="text" variable="issued"/>
-        <!-- 17th edition: -->
-        <!-- <date date-parts="year-month" form="text" variable="issued"/> -->
+        <date date-parts="year-month" form="text" variable="issued"/>
       </else-if>
       <else-if match="any" variable="issue volume">
         <date date-parts="year-month" form="text" variable="issued"/>
@@ -5461,14 +5406,10 @@
   <macro name="source-date-original-full-serial">
     <choose>
       <if variable="issue volume">
-        <date date-parts="year" form="text" variable="original-date"/>
-        <!-- 17th edition: -->
-        <!-- <date date-parts="year-month" form="text" variable="original-date"/> -->
+        <date date-parts="year-month" form="text" variable="original-date"/>
       </if>
       <else-if variable="supplement-number">
-        <date date-parts="year" form="text" variable="original-date"/>
-        <!-- 17th edition: -->
-        <!-- <date date-parts="year-month" form="text" variable="original-date"/> -->
+        <date date-parts="year-month" form="text" variable="original-date"/>
       </else-if>
       <else-if match="any" variable="issue volume">
         <date date-parts="year-month" form="text" variable="original-date"/>
@@ -6445,9 +6386,9 @@
       </else>
     </choose>
   </macro>
-  <citation disambiguate-add-givenname="true" disambiguate-add-names="true" et-al-min="3" et-al-use-first="1">
-    <layout delimiter="; " suffix=".">
-      <text macro="citation-notes-full-subsequent-author-title"/>
+  <citation after-collapse-delimiter="; " collapse="year" disambiguate-add-givenname="true" disambiguate-add-names="true" disambiguate-add-year-suffix="true" et-al-min="4" et-al-use-first="1" givenname-disambiguation-rule="primary-name-with-initials">
+    <layout delimiter="; " prefix="(" suffix=")">
+      <text macro="citation-author-date"/>
     </layout>
   </citation>
   <!-- Bibliography -->
@@ -6605,18 +6546,18 @@
       </else>
     </choose>
   </macro>
-  <bibliography et-al-min="7" et-al-use-first="3" hanging-indent="true">
+  <bibliography et-al-min="11" et-al-use-first="7" hanging-indent="true">
     <sort>
       <key macro="author-sort"/>
+      <key macro="date-sort-group"/>
+      <key macro="date-sort"/>
+      <key variable="status"/>
       <key macro="title-bib"/>
       <key variable="part-number"/>
       <key variable="volume"/>
-      <key macro="date-sort"/>
-      <key variable="status"/>
-      <key macro="source-archive-bib"/>
     </sort>
     <layout suffix=".">
-      <text macro="bibliography-filtered-notes"/>
+      <text macro="bibliography-filtered-author-date"/>
     </layout>
   </bibliography>
 </style>
