--- templates/new-harts-rules-template.csl
+++ development/the-journal-of-ecclesiastical-history.csl
@@ -5,33 +5,24 @@
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <style and="text" class="note" default-locale="en-GB" demote-non-dropping-particle="display-and-sort" et-al-min="4" et-al-use-first="1" page-range-format="minimal" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>New Hart's Rules: The Oxford Style Guide 2nd edition (template)</title>
-    <title-short>NHR (template)</title-short>
-    <id>http://www.zotero.org/styles/new-harts-rules-notes</id>
-    <link href="http://www.zotero.org/styles/new-harts-rules-notes" rel="self"/>
-    <link href="https://en.wikipedia.org/wiki/Hart's_Rules" rel="documentation"/>
+    <title>The Journal of Ecclesiastical History</title>
+    <title-short>JEH</title-short>
+    <id>http://www.zotero.org/styles/the-journal-of-ecclesiastical-history</id>
+    <link href="http://www.zotero.org/styles/the-journal-of-ecclesiastical-history" rel="self"/>
+    <link href="http://www.zotero.org/styles/new-harts-rules-notes" rel="template"/>
+    <link href="https://www.cambridge.org/core/journals/journal-of-ecclesiastical-history/information/author-instructions/preparing-your-materials" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
       <uri>https://orcid.org/0000-0003-0464-5036</uri>
     </author>
     <category citation-format="note"/>
-    <category field="anthropology"/>
-    <category field="communications"/>
-    <category field="generic-base"/>
-    <category field="geography"/>
     <category field="history"/>
     <category field="humanities"/>
-    <category field="linguistics"/>
-    <category field="literature"/>
-    <category field="philosophy"/>
-    <category field="political_science"/>
-    <category field="psychology"/>
-    <category field="science"/>
-    <category field="social_science"/>
-    <category field="sociology"/>
     <category field="theology"/>
-    <summary>Oxford source citations, template</summary>
-    <updated>2025-11-05T00:00:00+00:00</updated>
+    <issn>0022-0469</issn>
+    <eissn>1469-7637</eissn>
+    <summary>Oxford Guide to Style derivative, with all titles in sentence case, bracketed name type labels, unbracketed publication details except with series, space after shortened citation of `classic`.</summary>
+    <updated>2025-11-11T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="en">
@@ -56,6 +47,11 @@
       </term>
       <term form="verb-short" name="editor-translator">ed. and trans.</term>
       <term form="verb-short" name="editortranslator">ed. and trans.</term>
+      <term name="et-al">and others</term>
+      <term form="short" name="folio">
+        <single>fo.</single>
+        <multiple>fos</multiple>
+      </term>
       <term form="verb-short" name="illustrator">illus.</term>
       <term name="manuscript">unpublished manuscript</term>
       <term form="short" name="paper-conference">paper</term>
@@ -292,7 +288,7 @@
           <label variable="section"/>
         </if>
       </choose>
-      <text text-case="title" variable="section"/>
+      <text text-case="capitalize-first" variable="section"/>
     </group>
   </macro>
   <macro name="label-section-capitalized">
@@ -302,7 +298,7 @@
           <label text-case="capitalize-first" variable="section"/>
         </if>
       </choose>
-      <text text-case="title" variable="section"/>
+      <text text-case="capitalize-first" variable="section"/>
     </group>
   </macro>
   <macro name="label-section-symbol">
@@ -346,23 +342,14 @@
     </group>
   </macro>
   <macro name="label-volume">
-    <group delimiter=" ">
-      <choose>
-        <if is-numeric="volume">
-          <label form="short" variable="volume"/>
-        </if>
-      </choose>
-      <text variable="volume"/>
-    </group>
-    <!-- in OGS and other styles, label a volume by rendering it in Roman numerals (NHR 18.2.7): -->
-    <!-- <choose>
+    <choose>
       <if is-numeric="volume">
         <number form="roman" variable="volume"/>
       </if>
       <else>
         <text variable="volume"/>
       </else>
-    </choose> -->
+    </choose>
   </macro>
   <!-- 1. Author -->
   <macro name="author-bib">
@@ -372,7 +359,7 @@
         <!-- for OGS: -->
         <!-- <name delimiter-precedes-et-al="after-inverted-name" delimiter-precedes-last="always" font-variant="small-caps" name-as-sort-order="first"/> -->
         <!-- <et-al font-variant="normal"/> -->
-        <label form="short" prefix=", "/>
+        <label form="short" prefix=" (" suffix=")"/>
         <substitute>
           <names variable="author"/>
           <names variable="host"/>
@@ -508,7 +495,7 @@
     <group delimiter=", ">
       <names variable="composer">
         <name delimiter-precedes-last="contextual"/>
-        <label form="short" prefix=", "/>
+        <label form="short" prefix=" (" suffix=")"/>
         <substitute>
           <names variable="author"/>
           <names variable="guest"/>
@@ -781,11 +768,11 @@
     <choose>
       <if type="webpage">
         <!-- Website title substituted in bibliography only (cf. CMOS18 14.104) -->
-        <text text-case="title" variable="container-title"/>
+        <text text-case="capitalize-first" variable="container-title"/>
       </if>
       <else-if match="any" type="entry entry-dictionary entry-encyclopedia">
         <!-- Anonymous entries in reference works (cf. CMOS18 14.130) -->
-        <text font-style="italic" text-case="title" variable="container-title"/>
+        <text font-style="italic" text-case="capitalize-first" variable="container-title"/>
       </else-if>
     </choose>
   </macro>
@@ -793,11 +780,11 @@
     <choose>
       <if type="webpage">
         <!-- Website title substituted in bibliography only (cf. CMOS18 14.104) -->
-        <text form="short" text-case="title" variable="container-title"/>
+        <text form="short" text-case="capitalize-first" variable="container-title"/>
       </if>
       <else-if match="any" type="entry entry-dictionary entry-encyclopedia">
         <!-- Anonymous entries in reference works (cf. CMOS18 14.130) -->
-        <text font-style="italic" form="short" text-case="title" variable="container-title"/>
+        <text font-style="italic" form="short" text-case="capitalize-first" variable="container-title"/>
       </else-if>
     </choose>
   </macro>
@@ -1205,20 +1192,12 @@
             <if match="any" type="article-journal article-magazine article-newspaper periodical post-weblog review review-book"/>
             <else-if match="any" variable="collection-editor compiler editor editorial-director">
               <!-- monographic types -->
-              <text macro="source-monographic-locator"/>
-              <!-- for styles using Roman volume numbers: -->
-              <!-- <text macro="source-monographic-locator-roman-volume"/> -->
-              <!-- for OGS: -->
-              <!-- <text macro="source-monographic-locator-range"/> -->
+              <text macro="source-monographic-locator-range"/>
             </else-if>
             <!-- omit serial types -->
             <else-if match="any" type="interview paper-conference"/>
             <else>
-              <text macro="source-monographic-locator"/>
-              <!-- for styles using Roman volume numbers: -->
-              <!-- <text macro="source-monographic-locator-roman-volume"/> -->
-              <!-- for OGS: -->
-              <!-- <text macro="source-monographic-locator-range"/> -->
+              <text macro="source-monographic-locator-range"/>
             </else>
           </choose>
         </group>
@@ -1259,12 +1238,12 @@
         <!-- If a review has no `reviewed-title`, assume that `title` contains the title of the reviewed work; the description provides it. -->
         <choose>
           <if variable="reviewed-genre title">
-            <!-- Quotes, title case -->
-            <text form="short" quotes="true" text-case="title" variable="title"/>
+            <!-- Quotes, sentence case -->
+            <text form="short" quotes="true" text-case="capitalize-first" variable="title"/>
           </if>
           <else-if variable="reviewed-genre reviewed-title title">
-            <!-- Quotes, title case -->
-            <text form="short" quotes="true" text-case="title" variable="title"/>
+            <!-- Quotes, sentence case -->
+            <text form="short" quotes="true" text-case="capitalize-first" variable="title"/>
           </else-if>
           <else>
             <text macro="description-short"/>
@@ -1323,16 +1302,16 @@
         <text form="short" text-case="capitalize-first" variable="part-title"/>
       </if>
       <else-if match="any" type="bill collection legislation regulation treaty">
-        <!-- No italics or quotes, title case -->
-        <text text-case="title" variable="part-title"/>
+        <!-- No italics or quotes, sentence case -->
+        <text text-case="capitalize-first" variable="part-title"/>
       </else-if>
       <else-if match="any" type="book classic graphic hearing map periodical">
-        <!-- Italicized, title case (regardless of `container-title`) -->
-        <text font-style="italic" text-case="title" variable="part-title"/>
+        <!-- Italicized, sentence case (regardless of `container-title`) -->
+        <text font-style="italic" text-case="capitalize-first" variable="part-title"/>
       </else-if>
       <else-if type="entry-encyclopedia" variable="author container-title">
-        <!-- Signed encyclopedia entry in quotes, title case (cf. CMOS18 14.132) -->
-        <text quotes="true" text-case="title" variable="part-title"/>
+        <!-- Signed encyclopedia entry in quotes, sentence case (cf. CMOS18 14.132) -->
+        <text quotes="true" text-case="capitalize-first" variable="part-title"/>
       </else-if>
       <else-if type="entry-dictionary" variable="container-title">
         <!-- Quotes, sentence case -->
@@ -1351,34 +1330,34 @@
         <choose>
           <if match="any" type="article-journal article-magazine article-newspaper periodical post-weblog review review-book">
             <!-- serial types -->
-            <!-- Quotes, title case -->
-            <text quotes="true" text-case="title" variable="part-title"/>
+            <!-- Quotes, sentence case -->
+            <text quotes="true" text-case="capitalize-first" variable="part-title"/>
           </if>
           <else-if match="any" variable="collection-editor compiler editor editorial-director">
             <!-- monographic types -->
-            <!-- Italics, title case -->
-            <text font-style="italic" text-case="title" variable="part-title"/>
+            <!-- Italics, sentence case -->
+            <text font-style="italic" text-case="capitalize-first" variable="part-title"/>
           </else-if>
           <else-if match="any" type="interview paper-conference">
             <!-- serial types -->
-            <!-- Quotes, title case -->
-            <text quotes="true" text-case="title" variable="part-title"/>
+            <!-- Quotes, sentence case -->
+            <text quotes="true" text-case="capitalize-first" variable="part-title"/>
           </else-if>
           <else>
             <!-- monographic types -->
-            <!-- Italics, title case -->
-            <text font-style="italic" text-case="title" variable="part-title"/>
+            <!-- Italics, sentence case -->
+            <text font-style="italic" text-case="capitalize-first" variable="part-title"/>
           </else>
         </choose>
       </else-if>
       <else-if match="any" type="article dataset document interview manuscript paper-conference personal_communication speech thesis webpage">
         <!-- Container-like but not necessarily with `container-title` -->
-        <!-- Quotes, title case -->
-        <text quotes="true" text-case="title" variable="part-title"/>
-      </else-if>
-      <else>
-        <!-- Italicized, title case (default) -->
-        <text font-style="italic" text-case="title" variable="part-title"/>
+        <!-- Quotes, sentence case -->
+        <text quotes="true" text-case="capitalize-first" variable="part-title"/>
+      </else-if>
+      <else>
+        <!-- Italicized, sentence case (default) -->
+        <text font-style="italic" text-case="capitalize-first" variable="part-title"/>
       </else>
     </choose>
   </macro>
@@ -1389,16 +1368,16 @@
         <text form="short" text-case="capitalize-first" variable="title"/>
       </if>
       <else-if match="any" type="bill collection legislation regulation treaty">
-        <!-- No italics or quotes, title case -->
-        <text text-case="title" variable="title"/>
+        <!-- No italics or quotes, sentence case -->
+        <text text-case="capitalize-first" variable="title"/>
       </else-if>
       <else-if match="any" type="book classic graphic hearing map periodical">
-        <!-- Italicized, title case (regardless of `container-title`) -->
-        <text font-style="italic" text-case="title" variable="title"/>
+        <!-- Italicized, sentence case (regardless of `container-title`) -->
+        <text font-style="italic" text-case="capitalize-first" variable="title"/>
       </else-if>
       <else-if type="entry-encyclopedia" variable="author container-title">
-        <!-- Signed encyclopedia entry in quotes, title case (cf. CMOS18 14.132) -->
-        <text quotes="true" text-case="title" variable="title"/>
+        <!-- Signed encyclopedia entry in quotes, sentence case (cf. CMOS18 14.132) -->
+        <text quotes="true" text-case="capitalize-first" variable="title"/>
       </else-if>
       <else-if type="entry-dictionary" variable="container-title">
         <!-- Quotes, sentence case -->
@@ -1414,17 +1393,17 @@
       </else-if>
       <!-- Other types are formatted based on presence of `container-title` -->
       <else-if variable="container-title">
-        <!-- Quotes, title case -->
-        <text quotes="true" text-case="title" variable="title"/>
+        <!-- Quotes, sentence case -->
+        <text quotes="true" text-case="capitalize-first" variable="title"/>
       </else-if>
       <else-if match="any" type="article dataset document interview manuscript paper-conference personal_communication speech thesis webpage">
         <!-- Container-like but not necessarily with `container-title` -->
-        <!-- Quotes, title case -->
-        <text quotes="true" text-case="title" variable="title"/>
-      </else-if>
-      <else>
-        <!-- Italicized, title case (default) -->
-        <text font-style="italic" text-case="title" variable="title"/>
+        <!-- Quotes, sentence case -->
+        <text quotes="true" text-case="capitalize-first" variable="title"/>
+      </else-if>
+      <else>
+        <!-- Italicized, sentence case (default) -->
+        <text font-style="italic" text-case="capitalize-first" variable="title"/>
       </else>
     </choose>
   </macro>
@@ -1435,16 +1414,16 @@
         <text form="short" text-case="capitalize-first" variable="title"/>
       </if>
       <else-if match="any" type="bill collection legislation regulation treaty">
-        <!-- No italics or quotes, title case -->
-        <text form="short" text-case="title" variable="title"/>
+        <!-- No italics or quotes, sentence case -->
+        <text form="short" text-case="capitalize-first" variable="title"/>
       </else-if>
       <else-if match="any" type="book classic graphic hearing map periodical">
-        <!-- Italicized, title case (regardless of `container-title`) -->
-        <text font-style="italic" form="short" text-case="title" variable="title"/>
+        <!-- Italicized, sentence case (regardless of `container-title`) -->
+        <text font-style="italic" form="short" text-case="capitalize-first" variable="title"/>
       </else-if>
       <else-if type="entry-encyclopedia" variable="author container-title">
-        <!-- Signed encyclopedia entry in quotes, title case (cf. CMOS18 14.132) -->
-        <text form="short" quotes="true" text-case="title" variable="title"/>
+        <!-- Signed encyclopedia entry in quotes, sentence case (cf. CMOS18 14.132) -->
+        <text form="short" quotes="true" text-case="capitalize-first" variable="title"/>
       </else-if>
       <else-if type="entry-dictionary" variable="container-title">
         <!-- Quotes, sentence case -->
@@ -1460,17 +1439,17 @@
       </else-if>
       <!-- Other types are formatted based on presence of `container-title` -->
       <else-if variable="container-title">
-        <!-- Quotes, title case -->
-        <text form="short" quotes="true" text-case="title" variable="title"/>
+        <!-- Quotes, sentence case -->
+        <text form="short" quotes="true" text-case="capitalize-first" variable="title"/>
       </else-if>
       <else-if match="any" type="article dataset document interview manuscript paper-conference personal_communication speech thesis webpage">
         <!-- Container-like but not necessarily with `container-title` -->
-        <!-- Quotes, title case -->
-        <text form="short" quotes="true" text-case="title" variable="title"/>
-      </else-if>
-      <else>
-        <!-- Italicized, title case (default) -->
-        <text font-style="italic" form="short" text-case="title" variable="title"/>
+        <!-- Quotes, sentence case -->
+        <text form="short" quotes="true" text-case="capitalize-first" variable="title"/>
+      </else-if>
+      <else>
+        <!-- Italicized, sentence case (default) -->
+        <text font-style="italic" form="short" text-case="capitalize-first" variable="title"/>
       </else>
     </choose>
   </macro>
@@ -1483,16 +1462,12 @@
             <text macro="label-volume"/>
           </if>
           <else-if locator="column page" match="any"/>
-          <else-if variable="locator">
-            <!-- remove variable="locator" condition for OGS, which prints ranges -->
-            <text macro="label-volume"/>
-          </else-if>
           <else-if variable="page"/>
           <else>
             <text macro="label-volume"/>
           </else>
         </choose>
-        <text font-style="italic" text-case="title" variable="volume-title"/>
+        <text font-style="italic" text-case="capitalize-first" variable="volume-title"/>
       </group>
       <!-- `part-number` and `part-title` of monographic items refer to the source book/volume -->
       <group delimiter=": ">
@@ -1628,7 +1603,7 @@
           <text form="short" variable="genre"/>
           <!-- untitled introductions, based on review model (cf. CMOS18 14.101, MHRA4 7.12) -->
           <text macro="source-monographic-preposition"/>
-          <text font-style="italic" form="short" text-case="title" variable="container-title"/>
+          <text font-style="italic" form="short" text-case="capitalize-first" variable="container-title"/>
         </if>
         <else-if type="article-journal" variable="container-title genre volume-title">
           <text form="short" variable="genre"/>
@@ -1661,7 +1636,7 @@
         <else-if variable="chapter-number container-title">
           <text macro="label-chapter-number"/>
           <text term="in"/>
-          <text font-style="italic" form="short" text-case="title" variable="container-title"/>
+          <text font-style="italic" form="short" text-case="capitalize-first" variable="container-title"/>
         </else-if>
         <else-if variable="chapter-number">
           <text macro="label-chapter-number"/>
@@ -1821,11 +1796,11 @@
             <if match="any" variable="reviewed-genre reviewed-title">
               <!-- Not possible to distinguish TV series episode from other reviewed works without a reviewed source title -->
               <!-- Adapt for `reviewed-container-title` or similar if it becomes available -->
-              <text font-style="italic" text-case="title" variable="reviewed-title"/>
+              <text font-style="italic" text-case="capitalize-first" variable="reviewed-title"/>
             </if>
             <else>
               <!-- Assume title is title of reviewed work -->
-              <text font-style="italic" text-case="title" variable="title"/>
+              <text font-style="italic" text-case="capitalize-first" variable="title"/>
             </else>
           </choose>
           <text variable="reviewed-genre"/>
@@ -1842,11 +1817,11 @@
         <if match="any" variable="reviewed-genre reviewed-title">
           <!-- Not possible to distinguish TV series episode from other reviewed works without a reviewed source title -->
           <!-- Adapt for `reviewed-container-title` or similar if it becomes available -->
-          <text font-style="italic" form="short" text-case="title" variable="reviewed-title"/>
+          <text font-style="italic" form="short" text-case="capitalize-first" variable="reviewed-title"/>
         </if>
         <else>
           <!-- Assume title is title of reviewed work -->
-          <text font-style="italic" form="short" text-case="title" variable="title"/>
+          <text font-style="italic" form="short" text-case="capitalize-first" variable="title"/>
         </else>
       </choose>
     </group>
@@ -2305,6 +2280,16 @@
             <text macro="source-serial-identifier"/>
           </group>
         </if>
+        <else-if variable="collection-title volume">
+          <text macro="source-serial-title"/>
+          <text macro="source-serial-identifier"/>
+        </else-if>
+        <else-if variable="volume">
+          <group delimiter=" ">
+            <text macro="source-serial-title"/>
+            <text macro="source-serial-identifier"/>
+          </group>
+        </else-if>
         <else-if match="any" variable="collection-title issue supplement-number volume">
           <text macro="source-serial-title"/>
           <text macro="source-serial-identifier"/>
@@ -2393,7 +2378,7 @@
             <text term="in"/>
             <group delimiter=", ">
               <text macro="source-monographic-author"/>
-              <text font-style="italic" text-case="title" variable="volume-title"/>
+              <text font-style="italic" text-case="capitalize-first" variable="volume-title"/>
             </group>
           </group>
         </group>
@@ -2406,9 +2391,7 @@
       <text macro="source-serial-identifier-volume"/>
       <!-- periodical edition always capitalized (cf. CMOS18 14.89) -->
       <text macro="label-edition-capitalized"/>
-      <text macro="source-serial-locator"/>
-      <!-- for OGS: -->
-      <!-- <text macro="source-serial-locator-range"/> -->
+      <text macro="source-serial-locator-range"/>
     </group>
   </macro>
   <macro name="source-serial-identifier-author-date">
@@ -2449,16 +2432,12 @@
               <choose>
                 <if variable="issue volume">
                   <group delimiter="/">
-                    <text variable="volume"/>
-                    <!-- for original Hart's Rules or other options with Roman volume numbers for serials (as shown in NHR 18.1.3) -->
-                    <!-- <text macro="label-volume"/> -->
+                    <text macro="label-volume"/>
                     <text variable="issue"/>
                   </group>
                 </if>
                 <else>
-                  <text variable="volume"/>
-                  <!-- for serial Roman volume numbers: -->
-                  <!-- <text macro="label-volume"/> -->
+                  <text macro="label-volume"/>
                   <text macro="label-issue"/>
                 </else>
               </choose>
@@ -2566,7 +2545,7 @@
         </if>
       </choose>
       <!-- remove `source-series` for OGS: -->
-      <text macro="source-series"/>
+      <!-- <text macro="source-series"/> -->
       <text macro="source-event"/>
     </group>
   </macro>
@@ -2592,7 +2571,7 @@
     <!-- based on `author-note` -->
     <names variable="container-author">
       <name delimiter-precedes-last="contextual"/>
-      <label form="short" prefix=", "/>
+      <label form="short" prefix=" (" suffix=")"/>
       <substitute>
         <names variable="author"/>
         <names variable="guest"/>
@@ -2640,7 +2619,7 @@
   </macro>
   <macro name="source-monographic-title">
     <group delimiter=", ">
-      <text font-style="italic" text-case="title" variable="container-title"/>
+      <text font-style="italic" text-case="capitalize-first" variable="container-title"/>
       <text macro="title-volume"/>
     </group>
   </macro>
@@ -2884,11 +2863,23 @@
   <!-- 4.3. Series -->
   <macro name="source-series">
     <group delimiter=", ">
-      <text text-case="title" variable="collection-title"/>
-      <text variable="collection-number"/>
-      <!-- for Roman collection numbers (old Hart's Rules): -->
-      <!-- <number form="roman" variable="collection-number"/> -->
-      <text macro="label-issue"/>
+      <choose>
+        <if variable="issue">
+          <text text-case="title" variable="collection-title"/>
+          <number form="roman" variable="collection-number"/>
+          <text macro="label-issue"/>
+        </if>
+        <else-if is-numeric="collection-number" variable="collection-title">
+          <group delimiter=" ">
+            <text text-case="title" variable="collection-title"/>
+            <number form="roman" variable="collection-number"/>
+          </group>
+        </else-if>
+        <else-if variable="collection-number collection-title">
+          <text text-case="title" variable="collection-title"/>
+          <text variable="collection-number"/>
+        </else-if>
+      </choose>
     </group>
   </macro>
   <!-- 4.4. Event -->
@@ -3002,7 +2993,7 @@
   </macro>
   <!-- 4.5. Facts of publication -->
   <macro name="source-monographic-publication">
-    <group delimiter="; ">
+    <group delimiter=", ">
       <choose>
         <!-- `patent` date in identification (cf. CMOS18 14.158) -->
         <if type="patent"/>
@@ -3010,16 +3001,14 @@
         <else-if match="any" type="article-journal article-magazine article-newspaper periodical post-weblog review review-book"/>
         <else-if match="any" variable="collection-editor compiler editor editorial-director">
           <!-- monographic types -->
-          <!-- add for OGS: -->
-          <!-- <text macro="source-series"/> -->
+          <text macro="source-series"/>
           <text macro="source-publication-and-date"/>
         </else-if>
         <!-- omit serial types -->
         <else-if match="any" type="interview paper-conference"/>
         <else>
           <!-- monographic types -->
-          <!-- add for OGS: -->
-          <!-- <text macro="source-series"/> -->
+          <text macro="source-series"/>
           <text macro="source-publication-and-date"/>
         </else>
       </choose>
@@ -3029,45 +3018,15 @@
   <macro name="source-monographic-publication-bracketed">
     <!-- Facts of publication with brackets for print monographic formats -->
     <choose>
-      <if match="any" type="book broadcast chapter classic entry interview motion_picture musical_score pamphlet paper-conference report software song standard">
+      <if variable="collection-title">
         <text macro="source-monographic-publication" prefix="(" suffix=")"/>
       </if>
-      <else-if type="article" variable="genre publisher">
-        <!-- without brackets for a preprint in a repository (cf. CMOS18 14.76), with brackets for a working paper (cf. CMOS18 14.116) -->
-        <text macro="source-monographic-publication" prefix="(" suffix=")"/>
-      </else-if>
-      <!-- dictionary/encyclopedia entries probably are bracketed only with publisher (cf. MHRA4 7.5) -->
-      <else-if type="entry-dictionary" variable="publisher">
-        <text macro="source-monographic-publication" prefix="(" suffix=")"/>
-      </else-if>
-      <else-if type="entry-encyclopedia" variable="publisher">
-        <text macro="source-monographic-publication" prefix="(" suffix=")"/>
-      </else-if>
-      <else-if type="personal_communication">
-        <choose>
-          <if match="any" variable="collection-editor compiler editor editorial-director issue page supplement-number volume">
-            <!-- in an edited collection -->
-            <text macro="source-monographic-publication" prefix="(" suffix=")"/>
-          </if>
-        </choose>
-      </else-if>
     </choose>
   </macro>
   <macro name="source-monographic-publication-unbracketed">
     <!-- Facts of publication without brackets -->
     <choose>
-      <if match="any" type="book broadcast chapter classic entry interview motion_picture musical_score pamphlet paper-conference report software song standard"/>
-      <else-if type="article" variable="genre publisher"/>
-      <else-if type="entry-dictionary" variable="publisher"/>
-      <else-if type="entry-encyclopedia" variable="publisher"/>
-      <else-if type="personal_communication">
-        <choose>
-          <if match="none" variable="collection-editor compiler editor editorial-director issue page supplement-number volume">
-            <!-- not in an edited collection -->
-            <text macro="source-monographic-publication"/>
-          </if>
-        </choose>
-      </else-if>
+      <if variable="collection-title"/>
       <else>
         <text macro="source-monographic-publication"/>
       </else>
@@ -3137,11 +3096,11 @@
     </choose>
   </macro>
   <macro name="source-publication-and-date">
-    <group delimiter=", ">
+    <group delimiter=" ">
       <choose>
         <if match="any" type="post webpage">
           <!-- `container-title` functions like a publisher with social media and website sources (cf. CMOS18 14.104-106) -->
-          <text text-case="title" variable="container-title"/>
+          <text text-case="capitalize-first" variable="container-title"/>
         </if>
       </choose>
       <text macro="source-publication-history"/>
@@ -3161,7 +3120,10 @@
       </else-if>
       <else-if type="thesis">
         <!-- thesis type appears with university name (cf. MHRA4 7.10) -->
-        <text variable="genre"/>
+        <group delimiter=" ">
+          <text value="unpubl."/>
+          <text variable="genre"/>
+        </group>
       </else-if>
       <else-if match="any" variable="original-publisher original-publisher-place">
         <choose>
@@ -3227,7 +3189,7 @@
           </choose>
           <group delimiter=", ">
             <names variable="original-author"/>
-            <text font-style="italic" text-case="title" variable="original-title"/>
+            <text font-style="italic" text-case="capitalize-first" variable="original-title"/>
           </group>
           <group delimiter=", " prefix="[" suffix="]">
             <text macro="source-publication-publisher-original"/>
@@ -3243,9 +3205,10 @@
         <if match="any" type="book chapter classic collection entry entry-dictionary entry-encyclopedia interview manuscript musical_score pamphlet paper-conference performance report">
           <group delimiter=": ">
             <choose>
-              <if variable="publisher-place">
+              <if variable="collection-title collection-number"/>
+              <else-if variable="publisher-place">
                 <text variable="publisher-place"/>
-              </if>
+              </else-if>
               <else-if type="report">
                 <text variable="publisher"/>
               </else-if>
@@ -3545,17 +3508,13 @@
         <text macro="source-locator-subsequent-serial"/>
       </if>
       <else-if match="any" variable="collection-editor compiler editor editorial-director">
-        <text macro="source-locator-subsequent-monographic"/>
-        <!-- for OGS and other styles using Roman volume numbers: -->
-        <!-- <text macro="source-locator-subsequent-monographic-roman-volume"/> -->
+        <text macro="source-locator-subsequent-monographic-roman-volume"/>
       </else-if>
       <else-if match="any" type="interview paper-conference">
         <text macro="source-locator-subsequent-serial"/>
       </else-if>
       <else>
-        <text macro="source-locator-subsequent-monographic"/>
-        <!-- for OGS and other styles using Roman volume numbers: -->
-        <!-- <text macro="source-locator-subsequent-monographic-roman-volume"/> -->
+        <text macro="source-locator-subsequent-monographic-roman-volume"/>
       </else>
     </choose>
   </macro>
@@ -3602,7 +3561,7 @@
           <text macro="label-locator"/>
         </if>
         <!-- for OGS: -->
-        <!-- <else-if locator="column page" match="none">
+        <else-if locator="column page" match="none">
           <text macro="label-volume"/>
           <text macro="label-part-number"/>
           <text macro="label-locator"/>
@@ -3612,7 +3571,7 @@
             <text macro="label-volume"/>
             <text macro="label-locator"/>
           </group>
-        </else-if> -->
+        </else-if>
         <else-if variable="part-number">
           <text macro="label-volume"/>
           <text macro="label-part-number"/>
@@ -3643,14 +3602,12 @@
   <!-- 4.9. Archival location -->
   <macro name="source-archive">
     <group delimiter=", ">
+      <text variable="archive"/>
       <text variable="archive-place"/>
-      <text variable="archive"/>
       <!-- usually `archive`, `archive-place` but sometimes also `archive-place`, `archive` (NHR 18.6.5); the latter is more common elsewhere (OGS 15.11.1 and MHRA4 7.7) -->
       <text variable="archive_collection"/>
       <text variable="archive_location"/>
-      <text macro="source-archive-locator"/>
-      <!-- for OGS: -->
-      <!-- <text macro="source-archive-locator-range"/> -->
+      <text macro="source-archive-locator-range"/>
       <choose>
         <!-- database identifier, if not included elsewhere (cf. CMOS18 14.113) -->
         <if match="any" variable="archive_collection archive_location archive-place"/>
@@ -3770,7 +3727,7 @@
           </group>
         </else-if>
       </choose>
-      <text macro="source-DOI-URL"/>
+      <!-- <text macro="source-DOI-URL"/> -->
     </group>
   </macro>
   <macro name="source-DOI-URL">
@@ -3847,8 +3804,32 @@
     <group delimiter=", ">
       <choose>
         <if position="subsequent">
-          <text macro="citation-notes-shortened-author-title-item"/>
-          <text macro="source-locator-subsequent"/>
+          <choose>
+            <if match="any" variable="part-number volume">
+              <!-- comma always needed with volume/part -->
+              <text macro="citation-notes-shortened-author-title-item"/>
+              <text macro="source-locator-subsequent"/>
+            </if>
+            <else-if match="none" type="classic">
+              <text macro="citation-notes-shortened-author-title-item"/>
+              <text macro="source-locator-subsequent"/>
+            </else-if>
+            <!-- with `classic`, a non-numeric canonical reference or identifying number is separated by a space rather than a comma (CMOS18 14.145) -->
+            <else-if is-numeric="locator">
+              <text macro="citation-notes-shortened-author-title-item"/>
+              <text macro="source-locator-subsequent"/>
+            </else-if>
+            <else-if locator="chapter line verse" match="any">
+              <group delimiter=" ">
+                <text macro="citation-notes-shortened-author-title-item"/>
+                <text macro="source-locator-subsequent"/>
+              </group>
+            </else-if>
+            <else>
+              <text macro="citation-notes-shortened-author-title-item"/>
+              <text macro="source-locator-subsequent"/>
+            </else>
+          </choose>
         </if>
         <else>
           <text macro="author-note"/>
@@ -3900,9 +3881,12 @@
             <text macro="source-medium"/>
           </else-if>
         </choose>
-        <text macro="source-monographic-publication-short-unbracketed"/>
+        <choose>
+          <if disambiguate="true">
+            <text macro="source-date"/>
+          </if>
+        </choose>
       </group>
-      <text macro="source-monographic-publication-short-bracketed"/>
     </group>
   </macro>
   <macro name="citation-notes-shortened-author-title-disambiguate">
