--- templates/modern-humanities-research-association-template.csl
+++ development/transactions-of-the-cambridge-bibliographical-society.csl
@@ -3,26 +3,24 @@
   type="application/relax-ng-compact-syntax" schematypens="http://relaxng.org/ns/compatibility/annotations/1.0"?>
 <?xml-model href="https://raw.githubusercontent.com/citation-style-language/schema/refs/heads/master/schemas/styles/csl-repository.rnc"
   type="application/relax-ng-compact-syntax" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-<style and="text" class="note" default-locale="en-GB" demote-non-dropping-particle="display-and-sort" et-al-min="4" et-al-use-first="1" page-range-format="minimal-two" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
+<style and="text" class="note" default-locale="en-GB" demote-non-dropping-particle="display-and-sort" et-al-min="4" et-al-use-first="1" initialize-with=". " page-range-format="minimal-two" version="1.0" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
-    <title>Modern Humanities Research Association 4th edition (template)</title>
-    <title-short>MHRA Style Guide</title-short>
-    <id>http://www.zotero.org/styles/modern-humanities-research-association-notes</id>
-    <link href="http://www.zotero.org/styles/modern-humanities-research-association-notes" rel="self"/>
-    <link href="https://www.mhra.org.uk/style/" rel="documentation"/>
+    <title>Transactions of the Cambridge Bibliographical Society</title>
+    <title-short>TCBS</title-short>
+    <id>http://www.zotero.org/styles/transactions-of-the-cambridge-bibliographical-society</id>
+    <link href="http://www.zotero.org/styles/transactions-of-the-cambridge-bibliographical-society" rel="self"/>
+    <link href="http://www.zotero.org/styles/modern-humanities-research-association-notes" rel="template"/>
+    <link href="https://www.lib.cam.ac.uk/collections/special-collections/cambridge-bibliographical-society" rel="documentation"/>
     <author>
       <name>Andrew Dunning</name>
       <uri>https://orcid.org/0000-0003-0464-5036</uri>
     </author>
     <category citation-format="note"/>
-    <category field="generic-base"/>
     <category field="history"/>
     <category field="humanities"/>
-    <category field="linguistics"/>
     <category field="literature"/>
-    <category field="philosophy"/>
-    <category field="theology"/>
-    <summary>MHRA source citations, template</summary>
+    <issn>0068-6611</issn>
+    <summary>MHRA source citations, notes system, substituting places of publication for publisher names; without URLs; unlabelled page locators</summary>
     <updated>2025-08-07T00:00:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
@@ -215,9 +213,10 @@
   <macro name="label-locator">
     <group delimiter=" ">
       <choose>
-        <if is-numeric="locator">
+        <if locator="page"/>
+        <else-if is-numeric="locator">
           <label form="short" variable="locator"/>
-        </if>
+        </else-if>
         <!-- a non-numeric canonical reference is described by its formatting and does not need a label, similar to a timestamp (MHRA 7.3c) -->
         <else-if locator="chapter line verse" match="any"/>
         <else>
@@ -277,7 +276,7 @@
   </macro>
   <macro name="label-page">
     <group delimiter=" ">
-      <label form="short" variable="page"/>
+      <!-- <label form="short" variable="page"/> -->
       <text variable="page"/>
     </group>
   </macro>
@@ -1076,6 +1075,8 @@
         <if variable="title">
           <text macro="title"/>
           <text macro="description"/>
+          <!-- for author-date: -->
+          <!-- <text macro="description-capitalized"/> -->
           <text macro="identifier"/>
         </if>
         <else>
@@ -1145,9 +1146,9 @@
         </choose>
         <text macro="source-medium"/>
         <text macro="source-archive"/>
-        <text macro="source-DOI"/>
+        <!-- <text macro="source-DOI"/> -->
       </group>
-      <text macro="source-URL"/>
+      <!-- <text macro="source-URL"/> -->
     </group>
   </macro>
   <!-- 3.1. Title -->
@@ -3214,8 +3215,8 @@
         </else-if>
         <else-if variable="publisher">
           <group delimiter=": ">
-            <!-- <text variable="publisher-place"/> -->
-            <text variable="publisher"/>
+            <text variable="publisher-place"/>
+            <!-- <text variable="publisher"/> -->
           </group>
         </else-if>
         <!-- TODO: remove conditional when Zotero fixes double-mapping of `event-place` -->
@@ -3236,8 +3237,8 @@
     <choose>
       <if variable="original-publisher">
         <group delimiter=": ">
-          <!-- <text variable="original-publisher-place"/> -->
-          <text variable="original-publisher"/>
+          <text variable="original-publisher-place"/>
+          <!-- <text variable="original-publisher"/> -->
         </group>
       </if>
       <else>
